<!DOCTYPE html> <html lang="en" class="" id="rgw50_56ab1fd7ae687"> <head > <meta charset="utf-8"/> <meta http-equiv="content-type" content="text/html; charset=UTF-8"/> <meta name="Rg-Request-Token" id="Rg-Request-Token" content="JepYZceksY+/UOw5jXXF3yzNmlmvflynIo+DsY1Ozb58qjjWuT14CnOiwC6j40VjvyVJ+eJ7P78Plv0MBP03sDb8Pydea3KtExjBg/4miEk0EBtHbEYMNn3/iUyuhUZ9CJMifnV4RbBY2o9PUSe9zDT9fm7Q+5e/D8UMN0w6b/DuDaasJWh7n9c9a9p1ke45Vl9C7HjlET8a7udQVwuqKYJnZqFgB2zKZoMUoUbMsU913MFIuYf89w3kZKgBG1Q+blJXNoHMTPAtrypPSzz3/VUYhIAPIkg0pxCz6C+YnFQ="/> <meta http-equiv="expires" content="0"/> <link rel="apple-touch-icon" sizes="57x57" href="https://www.researchgate.net/apple-touch-icon-57x57.png"> <link rel="apple-touch-icon" sizes="60x60" href="https://www.researchgate.net/apple-touch-icon-60x60.png"> <link rel="apple-touch-icon" sizes="72x72" href="https://www.researchgate.net/apple-touch-icon-72x72.png"> <link rel="apple-touch-icon" sizes="76x76" href="https://www.researchgate.net/apple-touch-icon-76x76.png"> <link rel="apple-touch-icon" sizes="114x114" href="https://www.researchgate.net/apple-touch-icon-114x114.png"> <link rel="apple-touch-icon" sizes="120x120" href="https://www.researchgate.net/apple-touch-icon-120x120.png"> <link rel="apple-touch-icon" sizes="144x144" href="https://www.researchgate.net/apple-touch-icon-144x144.png"> <link rel="apple-touch-icon" sizes="152x152" href="https://www.researchgate.net/apple-touch-icon-152x152.png"> <link rel="apple-touch-icon" sizes="180x180" href="https://www.researchgate.net/apple-touch-icon-180x180.png"> <link rel="icon" type="image/png" href="https://www.researchgate.net/favicon-32x32.png" sizes="32x32"> <link rel="icon" type="image/png" href="https://www.researchgate.net/android-chrome-192x192.png" sizes="192x192"> <link rel="icon" type="image/png" href="https://www.researchgate.net/favicon-96x96.png" sizes="96x96"> <link rel="icon" type="image/png" href="https://www.researchgate.net/favicon-16x16.png" sizes="16x16"> <link rel="shortcut icon" type="image/x-icon" href="https://c5.rgstatic.net/m/2390829798215018/images/favicon.ico"/> <link rel="manifest" href="https://www.researchgate.net/manifest.json"> <meta name="msapplication-TileColor" content="#da532c"> <meta name="msapplication-TileImage" content="https://www.researchgate.net/mstile-144x144.png"> <meta name="theme-color" content="#444444"> <link rel="search" type="application/opensearchdescription+xml" title="ResearchGate search" href="https://www.researchgate.net/application.DownloadOpenSearchPlugin.html"/> <link rel="meta" type="application/rdf+xml" title="ICRA labels" href="https://www.researchgate.net/application.DownloadLabels.html"/> <link rel="http://oexchange.org/spec/0.8/rel/related-target" type="application/xrd+xml" href="https://www.researchgate.net/application.DownloadOExchange.html"/> <base href="https://www.researchgate.net/"/> <script>
    var rgConfig = {
        correlationId: "rgreq-4f8fec31-4745-407b-bfd0-5c2e051f6cde",
        accountId: "",
        module: "publicliterature",
        action: "publicliterature.PublicPublicationDetails",
        page: "publicationDetail",
        product: "publications",
        continent: "Asia",
        stylesHome: "//c5.rgstatic.net/m/",
        staticHost: "c5.rgstatic.net",
        longRunningRequestIdentifier: "LongRunningRequest.publicliterature.PublicPublicationDetails",
        longRunningRequestFp: "e9e5bbe1366c7ee8c8e41d02f793c884ac9be19b"
    };
    window.rootUrl = "https://www.researchgate.net/";
</script> <link rel="canonical" href="https://www.researchgate.net/publication/11216584_Marginalized_kernels_for_biological_sequences" />
<link rel="dns-prefetch" href="//c5.rgstatic.net" />
<link rel="dns-prefetch" href="//i1.rgstatic.net" />
<meta property="twitter:card" content="summary" />
<meta property="twitter:site" content="@ResearchGate" />
<meta property="og:title" content="Marginalized kernels for biological sequences" />
<meta property="og:description" content="MOTIVATION: Kernel methods such as support vector machines require a kernel function between objects to be defined a priori. Several works have been done to derive kernels from probability..." />
<meta property="og:site_name" content="ResearchGate" />
<meta property="og:image" content="https://i1.rgstatic.net/publication/11216584_Marginalized_kernels_for_biological_sequences/links/00b7d52705ee89c23a000000/smallpreview.png" />
<meta property="og:url" content="https://www.researchgate.net/publication/11216584_Marginalized_kernels_for_biological_sequences" />
<meta property="rg:id" content="PB:11216584" />
<meta name="DC.identifier" scheme="DCTERMS.URI" content="http://dx.doi.org/10.1093/bioinformatics/18.suppl_1.S268" />
<meta name="gs_meta_revision" content="1.1" />
<meta name="citation_title" content="Marginalized kernels for biological sequences" />
<meta name="citation_author" content="Koji Tsuda" />
<meta name="citation_author" content="Taishin Kin" />
<meta name="citation_author" content="Kiyoshi Asai" />
<meta name="citation_pmid" content="12169556" />
<meta name="citation_publication_date" content="2002/02/01" />
<meta name="citation_journal_title" content="Bioinformatics" />
<meta name="citation_issn" content="1367-4803" />
<meta name="citation_volume" content="18 Suppl 1" />
<meta name="citation_issue" content="Suppl 1" />
<meta name="citation_firstpage" content="S268" />
<meta name="citation_lastpage" content="75" />
<meta name="citation_doi" content="10.1093/bioinformatics/18.suppl_1.S268" />
<meta name="citation_pdf_url" content="https://www.researchgate.net/profile/Koji_Tsuda/publication/11216584_Marginalized_kernels_for_biological_sequences/links/00b7d52705ee89c23a000000.pdf" />
<meta name="citation_abstract_html_url" content="https://www.researchgate.net/publication/11216584_Marginalized_kernels_for_biological_sequences" />
<meta name="citation_fulltext_html_url" content="https://www.researchgate.net/publication/11216584_Marginalized_kernels_for_biological_sequences" />
<meta name="viewport" content="width=device-width,initial-scale=1" />
<link href="//c5.rgstatic.net/m/22664197317151888/styles/rg.css" type="text/css" rel="stylesheet"/>
<link href="//c5.rgstatic.net/m/21004998181197492/styles/rg2.css" type="text/css" rel="stylesheet"/>
<!--[if lt IE 9]><link href="//c5.rgstatic.net/m/238176252723686/styles/ie.css" type="text/css" rel="stylesheet"/><![endif]-->
<link href="//c5.rgstatic.net/m/217752362214895/styles/modules/publicprofile.css" type="text/css" rel="stylesheet"/>
<link href="//c5.rgstatic.net/m/21993347442549/styles/pow/publicliterature/FollowPublicationPromo.css" type="text/css" rel="stylesheet"/>
<link href="//c5.rgstatic.net/m/282514599719602/styles/pow/application/PdfJsReader.css" type="text/css" rel="stylesheet"/>
<link href="//c5.rgstatic.net/m/23819663151220/styles/pow/publicliterature/PublicationInlineReader.css" type="text/css" rel="stylesheet"/>
<script src="//c5.rgstatic.net/m/2321000301012716/javascript/vendor/webfontloader/webfontloader.js" type="text/javascript"></script>
 <script>(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){ (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o), m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m) })(window,document,"script","//www.google-analytics.com/analytics.js","ga");
 ga("create","UA-58591210-1");ga("set","anonymizeIp",true);ga('set', 'dimension1', 'publication full-text');ga('set', 'dimension2', 'Article');ga('set', 'dimension3', 'Logged out');ga('set', 'dimension4', 'https');ga("send","pageview");</script>
  <script> WebFontConfig = { custom: { families: ['source_sans_pro_italic', 'source_sans_pro_bold', 'source_sans_pro', 'martel'], urls : ['//c5.rgstatic.net/m/231392577336386/styles/fonts.css'] } }; WebFont.load(WebFontConfig); </script><noscript></noscript>

<title>Marginalized kernels for biological sequences (PDF Download Available)</title>
<meta name="description" content="Official Full-Text Publication: Marginalized kernels for biological sequences on ResearchGate, the professional network for scientists."/>
<meta name="keywords" content="scientific network, scientific platform, scientific community, research partner, research collaboration, journal articles, international collaboration, find researcher, lifescience researcher, interdisciplinary research, research collaboration"/>
</head>
<body class="use-svg-icons logged-out">
<div id="page-container">
<script type="text/javascript">var googletag = googletag || {}; googletag.cmd = googletag.cmd || [];
(function() { var gads = document.createElement("script"); gads.async = true; gads.type = "text/javascript"; var useSSL = "https:" == document.location.protocol; gads.src = (useSSL ? "https:" : "http:") + "//www.googletagservices.com/tag/js/gpt.js"; var node =document.getElementsByTagName("script")[0]; node.parentNode.insertBefore(gads, node); })();</script><div id="main" class="logged-out-header-support">
<div id="content" class="">

<noscript>
<div class="c-box-warning full-width-element" style="text-align: center; ">
    <div style="margin: auto; padding:10px;" class="container">
        <b>For full functionality of ResearchGate it is necessary to enable JavaScript.
            Here are the <a href="http://www.enable-javascript.com/" rel="nofollow" target="_blank">
                instructions how to enable JavaScript in your web browser</a>.</b>
    </div>
</div>
</noscript>

<div id="rgw1_56ab1fd7ae687" itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="publication-wrapper publication-wrapper-onecol" id="rgw2_56ab1fd7ae687" itemscope itemtype="http://schema.org/ScholarlyArticle"> <div class="c-col-content"> <div class="c-content"> <div class="clearfix">  <div class="publication-header"> <div id="rgw5_56ab1fd7ae687">  <div class="type-label"> Article   </div> <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rfr_id=info%3Asid%2Fresearchgate.net%3Aresearchgate&rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F18.suppl_1.S268&rft.atitle=Marginalized%20kernels%20for%20biological%20sequences&rft.title=Bioinformatics%20(Oxford%2C%20England)&rft.jtitle=Bioinformatics%20(Oxford%2C%20England)&rft.volume=18%20Suppl%201&rft.issue=Suppl%201&rft.date=2002&rft.pages=S268-75&rft.issn=1367-4803&rft.au=Koji%20Tsuda%2CTaishin%20Kin%2CKiyoshi%20Asai&rft.genre=article"></span> <h1 class="pub-title" itemprop="name">Marginalized kernels for biological sequences</h1> <meta itemprop="headline" content="Marginalized kernels for biological sequences">  <meta itemprop="image" content="https://i1.rgstatic.net/publication/11216584_Marginalized_kernels_for_biological_sequences/links/00b7d52705ee89c23a000000/smallpreview.png">  <div id="rgw8_56ab1fd7ae687" class="publication-detail-author-list"> <div> <ul class="clearfix js-people-list">  <li id="rgw9_56ab1fd7ae687" itemprop="author" itemscope itemtype="http://schema.org/Person"> <a itemprop="sameAs" href="profile/Koji_Tsuda" class="pub-detail-item account-item"> <div class="indent-left"> <div class="people-img"> <img  src="https://i1.rgstatic.net/ii/profile.image/AS%3A272439313694743%401441966002280_m/Koji_Tsuda.png" title="Koji Tsuda" alt="Koji Tsuda" height="20px" width="20px" style="height: 20px;"/> </div> </div> <div class="indent-content"> <div class="item-name"> <span itemprop="name">Koji Tsuda</span> </div> </div> </a>  <div style="display: none"> <div class="js-tooltip-content"> <ul class="pub-detail-item-tooltip"> <li class="people-item js-list-item  remove-action-indent   " id="rgw10_56ab1fd7ae687" data-account-key="Koji_Tsuda">   <div class="indent-left"> <div class="c-img-l people-img ga-top-coauthor-image">  <a href="profile/Koji_Tsuda"> <img class="lazyload" data-src="https://i1.rgstatic.net/ii/profile.image/AS%3A272439313694743%401441966002280_l/Koji_Tsuda.png" title="Koji Tsuda" alt="Koji Tsuda" height="90px" width="90px"/> </a>   </div> </div> <div class="indent-right">     </div> <div class="indent-content"> <h5 class="ga-top-coauthor-name">  <a href="profile/Koji_Tsuda" class="display-name">Koji Tsuda</a>    </h5> <div class="truncate-single-line meta">   <a class="meta ga-top-coauthor-institution" href="institution/The_University_of_Tokyo" title="The University of Tokyo">The University of Tokyo</a>     </div>  <a href="javascript:" class="btn btn-promote btn-large people-item-contact-author js-contact-author">Message author</a>  <div class="hide-button-container hide-button" style="display: none;"> <a href="javascript:" class="details js-hide action-hide">Remove suggestion</a> </div>  </div>    </li> </ul> </div> </div>  </li>  <li id="rgw11_56ab1fd7ae687"> <a href="researcher/39227296_Taishin_Kin" class="pub-detail-item author-item" itemprop="author" itemscope itemtype="http://schema.org/Person"> <div class="indent-left"> <div class="people-img">  <img src="https://c5.rgstatic.net/m/2951093203564/images/template/default/profile/profile_default_s.jpg" title="Taishin Kin" alt="Taishin Kin" height="20px" width="20px" style="height: 20px;"/>  </div> </div> <div class="indent-content"> <div class="item-name"> <span itemprop="name">Taishin Kin</span> </div> </div> </a>  <div style="display: none"> <div class="js-tooltip-content"> <ul class="pub-detail-item-tooltip"> <li class="people-item js-list-item   not-claimed " id="rgw12_56ab1fd7ae687">   <div class="indent-left"> <div class="c-img-l people-img ga-top-coauthor-image">  <a href="researcher/39227296_Taishin_Kin"> <img class="lazyload" data-src="https://c5.rgstatic.net/m/237738464651637/images/template/default/profile/profile_default_l.jpg" title="Taishin Kin" alt="Taishin Kin" height="90px" width="90px"/> </a>   </div> </div> <div class="indent-right">       </div> <div class="indent-content"> <h5 class="ga-top-coauthor-name">  <a href="researcher/39227296_Taishin_Kin" class="display-name">Taishin Kin</a>    </h5> <div class="truncate-single-line meta">   </div>  <div class="hide-button-container hide-button" style="display: none;"> <a href="javascript:" class="details js-hide action-hide">Remove suggestion</a> </div>  </div>    </li> </ul> </div> </div>  </li>  <li id="rgw13_56ab1fd7ae687" itemprop="author" itemscope itemtype="http://schema.org/Person"> <a itemprop="sameAs" href="profile/Kiyoshi_Asai" class="pub-detail-item account-item"> <div class="indent-left"> <div class="people-img"> <img  src="https://i1.rgstatic.net/ii/profile.image/AS%3A279014078992401%401443533549046_m/Kiyoshi_Asai.png" title="Kiyoshi Asai" alt="Kiyoshi Asai" height="20px" width="20px" style="height: 20px;"/> </div> </div> <div class="indent-content"> <div class="item-name"> <span itemprop="name">Kiyoshi Asai</span> </div> </div> </a>  <div style="display: none"> <div class="js-tooltip-content"> <ul class="pub-detail-item-tooltip"> <li class="people-item js-list-item  remove-action-indent   " id="rgw14_56ab1fd7ae687" data-account-key="Kiyoshi_Asai">   <div class="indent-left"> <div class="c-img-l people-img ga-top-coauthor-image">  <a href="profile/Kiyoshi_Asai"> <img class="lazyload" data-src="https://i1.rgstatic.net/ii/profile.image/AS%3A279014078992401%401443533549046_l/Kiyoshi_Asai.png" title="Kiyoshi Asai" alt="Kiyoshi Asai" height="90px" width="90px"/> </a>   </div> </div> <div class="indent-right">     </div> <div class="indent-content"> <h5 class="ga-top-coauthor-name">  <a href="profile/Kiyoshi_Asai" class="display-name">Kiyoshi Asai</a>    </h5> <div class="truncate-single-line meta">   <a class="meta ga-top-coauthor-institution" href="institution/The_University_of_Tokyo" title="The University of Tokyo">The University of Tokyo</a>     </div>  <a href="javascript:" class="btn btn-promote btn-large people-item-contact-author js-contact-author">Message author</a>  <div class="hide-button-container hide-button" style="display: none;"> <a href="javascript:" class="details js-hide action-hide">Remove suggestion</a> </div>  </div>    </li> </ul> </div> </div>  </li>   </ul> <div class="js-loading"></div>  </div> </div> <div class="pub-details js-pub-details">  <div> Computational Biology Research Center, National Institute of Advanced Industrial Science and Technology (AIST), 2-41-6 Aomi Koto-ku, Tokyo 135-0064, Japan.  </div>      <span itemprop="isPartOf" itemscope itemtype="http://schema.org/Periodical"> <a itemprop="sameAs" href="journal/1367-4803_Bioinformatics"><span itemprop="name">Bioinformatics</span></a> </span>    (Impact Factor: 4.98).     <meta itemprop="datePublished" content="2002-02">  02/2002;  18 Suppl 1(Suppl 1):S268-75.    DOI:&nbsp;10.1093/bioinformatics/18.suppl_1.S268           <div class="pub-source"> Source: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12169556" rel="nofollow">PubMed</a> </div>  </div> <div id="rgw15_56ab1fd7ae687" class="pub-abstract">  <div class="clearfix">   <div style="max-height: 54px;" class="js-expander-container js-expander-collapsed">  <p itemprop="description"> <strong>ABSTRACT</strong> <div>MOTIVATION: Kernel methods such as support vector machines require a kernel function between objects to be defined a priori. Several works have been done to derive kernels from probability distributions, e.g., the Fisher kernel. However, a general methodology to design a kernel is not fully developed. RESULTS: We propose a reasonable way of designing a kernel when objects are generated from latent variable models (e.g., HMM). First of all, a joint kernel is designed for complete data which include both visible and hidden variables. Then a marginalized kernel for visible data is obtained by taking the expectation with respect to hidden variables. We will show that the Fisher kernel is a special case of marginalized kernels, which gives another viewpoint to the Fisher kernel theory. Although our approach can be applied to any object, we particularly derive several marginalized kernels useful for biological sequences (e.g., DNA and proteins). The effectiveness of marginalized kernels is illustrated in the task of classifying bacterial gyrase subunit B (gyrB) amino acid sequences.</div> </p>  </div>  </div>   </div>      <div class="action-container"> <div id="rgw16_56ab1fd7ae687" class="follow-publication-promo"> <table> <tr> <td class="follow-publication-promo-text-cell"> <p>Get notified about updates to this publication</p> <a class="btn btn-large btn-promote js-follow-publication ga-follow-publication-new-promo">Follow publication</a> </td> <td> <div class="follow-publication-publication-image"></div> </td> </tr> </table> </div>  <div class="clear"></div> <div class="share-dialog-container" style="display: none;">  </div> </div> </div> <div style="margin-left: -20px; margin-right: -20px;">  </div> </div>  <div class="publication-detail-dfp-container rf"> <div id="rgw30_56ab1fd7ae687">  </div> </div>  </div>  <div class="clearfix"> <div class="pdf-js-container clearfix " id="rgw42_56ab1fd7ae687">  <div class="pdf-js-header js-sticky-header clear">  <a class="blue-link js-download rf btn btn-promote" href="https://www.researchgate.net/profile/Koji_Tsuda/publication/11216584_Marginalized_kernels_for_biological_sequences/links/00b7d52705ee89c23a000000.pdf?inViewer=0&amp;pdfJsDownload=0&amp;origin=publication_detail">Download full-text</a>  <h3>Full-text</h3> <span class="publication-info">  <span class="js-doi-container" style="display: none;"> DOI: <span class="js-doi"></span> &middot; </span> Available from: <a class="js-name" href="profile/Koji_Tsuda">Koji Tsuda</a>   </span>  </div>  <div class="social-share-container"><div id="rgw44_56ab1fd7ae687" class="social-share js-social-share"> <div class="social-share-heading">Share</div>  <a id="rgw45_56ab1fd7ae687" href="javascript:" title="Share on Facebook" class="link-nostyle js-share-item share-icon "  data-url="http://www.facebook.com/share.php?u={{url}}{{#title}}&amp;t={{title}}{{/title}}" data-width="600" data-height="350" data-name="Facebook"> <span class="share-icon ico-share-facebook-round reset-background ga-share-blog-facebook"></span> </a>  <a id="rgw46_56ab1fd7ae687" href="javascript:" title="Share on Twitter" class="link-nostyle js-share-item share-icon "  data-url="http://twitter.com/intent/tweet?text={{#title}}{{title}}: {{/title}}{{url}}&amp;via=researchgate" data-width="600" data-height="350" data-name="Twitter"> <span class="share-icon ico-share-twitter-round reset-background ga-share-blog-twitter"></span> </a>  <a id="rgw47_56ab1fd7ae687" href="javascript:" title="Share on Google+" class="link-nostyle js-share-item share-icon "  data-url="https://plus.google.com/share?url={{url}}" data-width="600" data-height="600" data-name="Google+"> <span class="share-icon ico-share-gplus-round reset-background ga-share-blog-gplus"></span> </a>  <a id="rgw48_56ab1fd7ae687" href="javascript:" title="Share on LinkedIn" class="link-nostyle js-share-item share-icon "  data-url="http://www.linkedin.com/shareArticle?mini=true&amp;url={{url}}{{#title}}&amp;title={{title}}{{/title}}&amp;source=ResearchGate" data-width="520" data-height="570" data-name="LinkedIn"> <span class="share-icon ico-share-linkedin-round reset-background ga-share-blog-linkedin"></span> </a>  <a id="rgw49_56ab1fd7ae687" href="javascript:" title="Share on Reddit" class="link-nostyle js-share-item share-icon "  data-url="https://www.reddit.com/submit?url={{url}}{{#title}}&amp;title={{title}}{{/title}}" data-width="600" data-height="600" data-name="Reddit"> <span class="share-icon ico-share-reddit reset-background ga-share-blog-reddit"></span> </a>  </div></div>    <iframe id="rgw43_56ab1fd7ae687" src="https://www.researchgate.net/c/o1o9o3/javascript/lib/pdfjs/web/viewer.html?file=https%3A%2F%2Fwww.researchgate.net%2Fprofile%2FKoji_Tsuda%2Fpublication%2F11216584_Marginalized_kernels_for_biological_sequences%2Flinks%2F00b7d52705ee89c23a000000.pdf%3FinViewer%3D1%26pdfJsDownload%3D1%26origin%3Dpublication_detail" allowfullscreen frameborder="0" style="width: 100%; height: 100%; box-sizing: border-box;"></iframe>    </div> <noscript> <div id="rgw29_56ab1fd7ae687"  itemprop="articleBody">  <p>Page 1</p> <p>BIOINFORMATICS<br />Vol. 18 Suppl. 1 2002<br />Pages S268–S275<br />Marginalized kernels for biological sequences<br />Koji Tsuda, Taishin Kin and Kiyoshi Asai<br />Computational Biology Research Center, National Institute of Advanced Industrial<br />Science and Technology (AIST), 2-41-6 Aomi Koto-ku, Tokyo, 135-0064, Japan<br />Received on January 24, 2002; revised and accepted on April 1, 2002<br />ABSTRACT<br />Motivation: Kernel methods such as support vector<br />machines require a kernel function between objects to be<br />defined a priori. Several works have been done to derive<br />kernels from probability distributions, e.g., the Fisher<br />kernel. However, a general methodology to design a kernel<br />is not fully developed.<br />Results: We propose a reasonable way of designing a<br />kernel when objects are generated from latent variable<br />models (e.g., HMM). First of all, a joint kernel is designed<br />for complete data which include both visible and hidden<br />variables. Then a marginalized kernel for visible data<br />is obtained by taking the expectation with respect to<br />hidden variables. We will show that the Fisher kernel<br />is a special case of marginalized kernels, which gives<br />another viewpoint to the Fisher kernel theory. Although<br />our approach can be applied to any object, we particularly<br />derive several marginalized kernels useful for biological<br />sequences (e.g., DNA and proteins). The effectiveness of<br />marginalized kernels is illustrated in the task of classifying<br />bacterial gyrase subunit B (gyrB) amino acid sequences.<br />Contact: koji.tsuda@aist.go.jp<br />Keywords: kernel design; marginalized kernels; the<br />Fisher kernel; biological sequence classification; string<br />kernels.<br />INTRODUCTION<br />Inkernelmethodssuchassupportvectormachines(M¨ uller<br />et al., 2001), a kernel function between two objects should<br />be determined a priori. In supervised learning algorithms,<br />the objective function to be optimized is clearly stated<br />(e.g., the expected risk), so one should determine the<br />kernel to optimize this function or its approximations<br />(e.g., the leave-one-out error). However, in unsupervised<br />learning algorithms such as clustering, the choice of<br />kernel is quite subjective. The kernel is determined to<br />reflect the user’s notion of similarity, which cannot be<br />justified nor falsified completely. However, it is not an<br />easy task to describe your notion of similarity as a positive<br />semidefinite kernel.<br />DNA and proteins are symbol sequences which may<br />have different lengths. So they have similar character-<br />istics to other symbol sequences such as texts (Frakes<br />and Baeza-Yates, 1992). In order to measure similarity<br />between such sequences, it is common to extract count<br />features, which represent the number of each symbol<br />contained in a sequence. Then, the similarity is obtained<br />as the weighted dot product between these features,<br />where a smaller weight is assigned for the symbols which<br />appear frequently. Although this approach (i.e., vector<br />space representation (Frakes and Baeza-Yates, 1992))<br />achieved a great success for texts, it is not appropriate for<br />biological sequences. The primal reason is that the context<br />changes frequently in one sequence. For example, a<br />DNA sequence has coding and noncoding regions, whose<br />statistical properties are quite different. The residuals ‘A’<br />in coding and noncoding regions have different meanings.<br />Although it is difficult to determine the boundaries of the<br />regions, they should be counted separately. If a sequence<br />of hidden variables which describe the context are avail-<br />able, it would be easier to design a kernel (Figure 1).<br />However, hidden variables are unknown in general, and<br />have to be estimated.<br />In this paper, we propose a new reasonable way to<br />design a kernel. First, a kernel between sequences is<br />defined depending both on visible and hidden variables.<br />Because this kernel requires both visible and hidden<br />variables for calculation, we call it a joint kernel. Since<br />hidden information is assumed to be available, the kernel<br />can be designed depending on the hidden context of<br />sequences. However, the problem is that such hidden<br />information is actually not available. To cope with this<br />problem, the posterior distribution of hidden variables<br />are estimated by means of a probabilistic model such as<br />HMM. Then, we obtain a marginalized kernel by taking<br />expectation of the joint kernel with respect to hidden<br />variables.<br />We will show that the Fisher kernel (Jaakkola and<br />Haussler, 1999)—which has been successfully applied<br />to many tasks e.g., protein classification (Jaakkola et<br />al., 2000; Karchin et al., 2002) and promoter region<br />detection (Pavlidis et al., 2001)—is a special case of<br />marginalized kernels. This reveals the joint kernel im-<br />plicitly assumed in the Fisher kernel, which helps us to<br />understand the Fisher kernel more in detail. For biological<br />S268<br />c ? Oxford University Press 2002</p>  <p>Page 2</p> <p>Marginalized kernels for biological sequences<br />h:<br />x:<br />2<br />C<br />1<br />A<br />2<br />G<br />1<br />G<br />2<br />T<br />2<br />T<br />1<br />C<br />2<br />A<br />2<br />A<br />Fig. 1. A DNA sequence with hidden context information. Sup-<br />pose the hidden variable (‘h’ in the figure) indicates e.g., cod-<br />ing/noncoding regions. If hidden variables are known, it would be<br />much easier to design a kernel function between sequences.<br />sequences, we propose useful kernels, called marginalized<br />count kernels (MCKs). In order to illustrate the effec-<br />tiveness of our kernels, we will perform experiments to<br />classify bacterial gyrB amino acid sequences (Kasai et<br />al., 2000). As a result, it is shown that MCKs compare<br />favorably to the Fisher kernel.<br />METHODS<br />Marginalized kernels<br />Let us describe a visible variable as x ∈ X, where the<br />domain X is a finite set†. Our task is to define a kernel<br />K(x,x?) between two visible variables x,x?. Suppose we<br />have a hidden variable h ∈ H, where H is a finite set.<br />By utilizing the hidden information in h, the joint kernel<br />Kz(z,z?) is designed between two combined variables<br />z = (x,h),z?= (x?,h?). The marginalized kernel in X<br />is derived by taking the expectation with respect to hidden<br />variables:<br />K(x,x?) =<br />?<br />h∈H<br />?<br />h?∈H<br />p(h|x)p(h?|x?)Kz(z,z?).<br />(1)<br />The posterior distribution p(h|x) is unknown in general,<br />and has to be estimated from the data e.g., by HMMs.<br />The calculation of K can be intractable when the cardi-<br />nality of H is too large. However, for useful stochastic<br />models such as HMMs, there are algorithms which<br />enable efficient computation as shown in later sections.<br />Since the class of positive semidefinite (Mercer) kernels<br />are closed under addition and multiplication (Haussler,<br />1999), the marginalized kernel K is positive semidefinite<br />as long as the joint kernel is positive semidefinite. In<br />convolution kernels (Haussler, 1999), sub-kernels for the<br />parts are aggregated into a kernel for the whole set. This<br />is constrastive to our approach deriving the kernel for the<br />part (i.e., visible variables) from the kernel for the whole<br />set (i.e., visible and hidden variables).<br />Marginalized kernel from Gaussian mixture<br />For intuitive understanding, we provide a simple example<br />of marginalized kernels. Here we derive a marginalized<br />†Here we determine X as a finite set for simplicity, but the results in this<br />paper can be easily extended to continuous domains.<br />kernel from Gaussian mixture with m-components. The<br />visible variable is a point in the d-dimensional space<br />x ∈ ?d, and the hidden variable is an index of component<br />h ∈ {1,...,m}. The probabilistic model is written as<br />p(x|θ) =<br />component is a Gaussian distribution with mean µhand<br />covariance matrix A−1<br />?m<br />h=1p(h)q(x|h,µh, Ah), where the hth<br />h:<br />q(x|h,µh, Ah) =exp(−1<br />2(x − µh)?Ah(x − µh))<br />2π|A−1<br />h|1/2<br />. (2)<br />Let us define the joint kernel as Kz(z,z?) = I(h =<br />h?)(x?Ahx?), where I(t) is the indicator function which<br />is 1 if the condition t is true and 0 otherwise. Note that Kz<br />is a positive semidefinite kernel. The marginalized kernel<br />is obtained as<br />K(x,x?) =<br />m<br />?<br />h=1<br />p(h|x)p(h|x?)x?Ahx?.<br />(3)<br />We illustrate the shape of this kernel in Figure 2. Here, the<br />kernel is converted to the distance in the feature space<br />D(x,x?) =<br />Fixing x?at a point, the contours of distance D(x,x?)<br />are shown. When x?belongs to one cluster, the contour<br />shape is similar to the shape of the cluster. The shape<br />gradually changes when the point x?moves to one cluster<br />to the other. In comparison with the Euclidean distance,<br />this distance emphasizes the cluster structure. This kind<br />of kernel is considered to be useful in visualizing cluster<br />structure in a high dimensional space (Tipping, 1999).<br />?<br />K(x,x) + K(x?,x?) − 2K(x,x?).<br />(4)<br />Marginalized count kernel<br />Next we propose an important example of marginal-<br />ized kernels for biological sequences. Let x<br />(x1,...,xm),<br />xi ∈ {1,...,nx} denote a symbol se-<br />quence of length m. Assume that each sequence can have<br />a different length. One simple kernel for such sequences<br />is the count kernel:<br />=<br />K(x,x?) =<br />nx<br />?<br />k=1<br />ck(x)ck(x?),<br />(5)<br />where ck(x) is the number of symbol k normalized by the<br />length:<br />ck(x) =1<br />m<br />m<br />?<br />i=1<br />I(xi= k).<br />The count kernel is often used in text processing litera-<br />tures (Frakes and Baeza-Yates, 1992), but it is not suit-<br />able for biological sequences because of frequent context<br />S269</p>  <p>Page 3</p> <p>K.Tsuda et al.<br />Fig. 2. Contours of the marginalized kernel-based distance from a<br />specified point. The left figure shows sample points generated from<br />the Gaussian mixture model. Here the crossing point of dotted lines<br />indicates the central point from which the kernel-based distance<br />(4) is measured. The right figure shows the distance contours.<br />The contour shapes are adapted to the shape of cluster which the<br />central point belongs to. Compared with the Euclidean distance, this<br />distance emphasizes the cluster structure.<br />changes. We are going to extend the count kernel to in-<br />clude (hidden) context information.<br />Assume that there is a sequence of hidden variables h =<br />(h1,...,hm), hi ∈ {1,...,nh}. Define the combined<br />sequence as<br />z := (z1,...,zm) = ({x1,h1},...,{xm,hm})<br />where each zi can have nz = nxnhsymbols. The count<br />kernel for z can be defined as<br />Kz(z,z?) =<br />nx<br />?<br />k=1<br />nh<br />?<br />?=1<br />m<br />?<br />i=1<br />ck?(z)ck?(z?),<br />ck?(z) =1<br />m<br />I(xi= k,hi= ?).<br />(6)<br />When h is regarded as the context information, symbols<br />are counted separately in each context (Figure 3). For<br />example, if the sequences are DNA and the hidden<br />contexts are exon/intron, the frequencies of ‘A’, ‘C’, ‘G’,<br />‘T’ are counted and compared separately for exon/intron.<br />Setting (6) as a joint kernel, the marginalized count<br />kernel is defined as<br />K(x,x?) =<br />?<br />h<br />?<br />h?<br />p(h|x)p(h?|x?)Kz(z,z?),<br />(7)<br />where<br />rewritten as<br />?<br />h<br />=<br />?nh<br />h1=1···?nh<br />hm=1. This kernel (7) is<br />K(x,x?) =<br />nx<br />?<br />k=1<br />nh<br />?<br />?=1<br />γkl(x)γkl(x?),<br />(T,1) = 0<br />(T,2) = 2<br />(G,1) = 1<br />(G,2) = 1<br />(C,1) = 1<br />(C,2) = 1<br />(A,1) = 1<br />(A,2) = 2<br />h:<br />x:<br />2<br />C<br />1<br />A<br />2<br />G<br />1<br />G<br />2<br />T<br />2<br />T<br />1<br />C<br />2<br />A<br />2<br />A<br />Fig. 3. Illustration of the marginalized count kernel (MCK1). Each<br />feature is obtained by counting the number of combined symbols,<br />and the joint kernel is defined as the dot product between these<br />features. Finally, MCK1 is obtained by marginalizing the joint<br />kernel.<br />where the marginalized counts γkl(x) are described as<br />γkl(x) =1<br />m<br />?<br />h<br />m<br />?<br />i=1<br />p(h|x)<br />m<br />?<br />i=1<br />I(xi= k,hi= ?)<br />=1<br />m<br />nh<br />?<br />hi=1<br />p(hi|x)I(xi= k,hi= ?).<br />Fortunately, the sum over all hidden variables h can be<br />replaced by the sum over each hi, which reduces the<br />computational cost.<br />When the probability distribution p(x) is represented<br />as HMM, the posterior probability p(hi|x) is computed<br />easily by the forward-backward algorithm (Durbin et al.,<br />1998). An HMM is described as<br />p(x|θ) =<br />?<br />h<br />qh1eh1x1[<br />m<br />?<br />i=2<br />ahi−1hiehixi]dhm,<br />(8)<br />wheretheparametersθ = {a,e,q,d}aretransitionproba-<br />bilities,emissionprobabilities,initialstatedistributionand<br />terminal state distribution, respectively. The forward and<br />backward algorithms provide the following probabilities,<br />respectively:<br />fk(i) = p(x1,...,xi,hi= k),<br />bk(i) = p(xi+1,...,xm|hi= k).<br />Then the posterior probability is described as<br />p(hi= ?|x) =<br />f?(i)b?(i)<br />p(x)<br />which is known as γi(?) in HMM literatures (Rabiner,<br />1989).<br />Second-order marginalized count kernel<br />When adjacent relations between symbols have essential<br />meanings, the count kernel is obviously not sufficient. In<br />S270</p>  <p>Page 4</p> <p>Marginalized kernels for biological sequences<br />such cases, it would be better to count the number of<br />combinations of two adjacent symbols (Figure 4). The dot<br />product of such counts is described as<br />K(x,x?) =<br />nx<br />?<br />k=1<br />nx<br />?<br />k?=1<br />1<br />m − 1<br />ckk?(x)ckk?(x?),<br />(9)<br />ckk?(x) =<br />m−1<br />?<br />i=1<br />I(xi= k)I(xi+1= k?).<br />We call it the second order count kernel. Incorporating<br />hiddenvariablesto(9),onecaneasilyextendthemarginal-<br />ized count kernel to second order: Let us define<br />δi<br />k?k??? := I(xi= k,hi= ?,xi+1= k?,hi+1= ??).<br />The joint kernel is described as<br />Kz(z,z?) =<br />nx<br />?<br />k=1<br />nh<br />?<br />?=1<br />nx<br />?<br />k?=1<br />1<br />m−1<br />nh<br />?<br />??=1<br />?m−1<br />nh<br />?<br />??=1<br />ck?k???(z)ck?k???(z?),<br />(10)<br />where ck?k???(z) =<br />kernel of (10) is described as<br />i=1δi<br />k?k???. The marginalized<br />K(x,x?) =<br />nx<br />?<br />k=1<br />nh<br />?<br />?=1<br />nx<br />?<br />k?=1<br />vk?k???(x)?vk?k???(x?) (11)<br />where<br />vk?k???(x) =<br />1<br />m − 1<br />1<br />m − 1<br />?<br />h<br />m−1<br />?<br />i=1<br />p(h|x)<br />m−1<br />?<br />i=1<br />nh<br />?<br />hi+1=1<br />δi<br />k?k???<br />=<br />nh<br />?<br />hi=1<br />p(hi,hi+1|x)δi<br />k?k???.<br />We call it the second order marginalized count kernel. As<br />in the first order case, the posterior probability p(hi,hi+<br />1|x) is obtained from forward and backward algorithms as<br />p(hi= ?,hi+1= ??|x) = a???e??xi+1<br />Note that this quantity is well known as ξi(?,??) parameter<br />in Baum-Welch algorithm, which gives estimation of tran-<br />sition probabilities in HMMs. The second order marginal-<br />ized count kernel is particularly useful, because it can uti-<br />lize second order information as well as hidden context in-<br />formation. Higher order extension is straightforward, but<br />not written here for brevity.<br />f?(i)b??(i + 1)<br />p(x)<br />.<br />Connections to the Fisher kernel<br />In the following, we will show that the Fisher ker-<br />nels (Jaakkola and Haussler, 1999) derived from latent<br />variable models are described as marginalized kernels.<br />This section will give a new analysis to explain the nature<br />of the Fisher kernel.<br />h:<br />x:<br />2<br />C<br />1<br />A<br />2<br />G<br />1<br />G<br />2<br />T<br />2<br />T<br />1<br />C<br />2<br />A<br />2<br />A<br />Fig. 4. Illustration of the second-order marginalized count kernel<br />(MCK2). Each feature is obtained by counting the number of<br />combinations of two adjacent symbols. The joint kernel is defined<br />as the dot product between these features, and MCK2 is obtained by<br />marginalization.<br />Definition of the Fisher kernel<br />model p(x|θ) is defined on X, where θ is a r-dimensional<br />parameter vector. Letˆθ denote parameter values which<br />are obtained by some learning algorithm (e.g., maximum<br />likelihood). Then the Fisher kernel (FK) (Jaakkola and<br />Haussler, 1999) between two objects is defined as<br />Assume a probabilistic<br />Kf(x,x?) = s(x,ˆθ)?Z−1(ˆθ)s(x?,ˆθ),<br />where s is the Fisher score<br />(12)<br />s(x,ˆθ) =<br />?∂<br />∂θ1<br />log p(x|ˆθ),...,<br />∂<br />∂θr<br />log p(x|ˆθ)<br />??<br />:= ∇θlog p(x|ˆθ),<br />and Z is the Fisher information matrix:<br />Z(ˆθ) = E<br />?<br />s(x,ˆθ)s(x,ˆθ)????ˆθ<br />?<br />x∈X<br />?<br />=<br />p(x|ˆθ)s(x,ˆθ)s(x,ˆθ)?.<br />The Fisher kernel is a general method which can be<br />applied for any objects. However, the Fisher kernel<br />is particularly effective for biological sequences when<br />combined with HMMs (Jaakkola et al., 2000; Karchin et<br />al., 2002; Pavlidis et al., 2001).<br />The Fisher Kernel from Latent Variable Models<br />a latent variable model p(x|θ) =<br />adopted, the Fisher score is described as<br />When<br />?<br />h∈Hp(x,h|θ) is<br />∇θlog p(x|ˆθ) =<br />?<br />h∈H∇θp(x,h|ˆθ)<br />p(x|ˆθ)<br />?<br />h∈H<br />?<br />h∈H<br />=<br />p(x,h|ˆθ)<br />p(x|ˆθ)<br />p(h|x,ˆθ)∇θlog p(x,h|ˆθ).<br />∇θp(x,h|ˆθ)<br />p(x,h|ˆθ)<br />=<br />S271</p>  <p>Page 5</p> <p>K.Tsuda et al.<br />So, the Fisher kernel is described as a marginalized kernel<br />Kf(x,x?) = ∇θlog p(x|ˆθ)?Z(ˆθ)−1∇θlog p(x?|ˆθ)<br />:=<br />h∈H<br />?<br />?<br />h?∈H<br />p(h|x,ˆθ)p(h?|x?,ˆθ)Kz(z,z?),<br />(13)<br />where the joint kernel is described as Kz(z,z?)<br />∇θlog p(x,h|ˆθ)?Z(ˆθ)−1∇θlog p(x?,h?|ˆθ).<br />Fisher kernel is one special case in the class of marginal-<br />ized kernels. One characteristic aspect of the Fisher kernel<br />is that the joint kernel is determined by the probabilistic<br />model, while, in our approach, the joint kernel is designed<br />to fit user’s purposes. Since the joint kernel of the Fisher<br />kernel is not suitable for every purpose, you have to check<br />whether it fits your purpose or not. If not, the joint kernel<br />should be engineered.<br />=<br />Thus the<br />The Fisher kernel from HMM<br />the Fisher kernel from HMM (8) and discuss its connec-<br />tion to marginalized count kernels. The joint distribution<br />of HMM is described as<br />In this section, we derive<br />p(x,h|θ) = qh1eh1x1[<br />m<br />?<br />i=2<br />ahi−1hiehixi]dhm.<br />As in the literature (Jaakkola et al., 2000), we take the<br />derivatives with respect to emission probabilities e only:<br />∂<br />∂e?k<br />log p(x,h|ˆθ) =mck?(z)<br />ˆ e?k<br />− m<br />nx<br />?<br />k=1<br />ck?(z),<br />(14)<br />where ˆ e?kis the estimated emission probability and ck?(z)<br />is defined as (6). Note that the second term of (14) comes<br />from the constraint of emission probabilities?nx<br />1. If we do not use the Fisher information matrix as in<br />(Jaakkola et al., 2000), the joint kernel is described as<br />k=1e?k=<br />∇elog p(x,h|ˆθ)?∇elog p(x?,h?|ˆθ).<br />This is rewritten as<br />nx<br />?<br />k=1<br />where ¯ ck?(z) = ˆ e?k<br />to the count kernel (6), however the count is centralized<br />and the dot product is taken with respect to the weight<br />αk?=mm?<br />proper normalization is needed for the Fisher kernel. Since<br />e?k represent the emission probability that symbol k is<br />produced from state ?, the weight becomes large when the<br />symbol k is rarely produced from state ?. It makes sense,<br />because the cooccurence of a rare symbol is a strong clue<br />of high similarity. However this weight is still argueable,<br />because a huge weight can appear when e?kis very small.<br />(15)<br />nh<br />?<br />?=1<br />mm?<br />ˆ e2<br />?k<br />(ck?(z) − ¯ ck?(z))(ck?(z?) − ¯ ck?(z?)),<br />(16)<br />?nx<br />k?=1ck??(z). This has a similar form<br />ˆ e2<br />?k. The weight is dependent on the length m, so a<br />DISCUSSION<br />In the previous section, we derived the Fisher kernel<br />only from emission probabilities. However, if you take<br />the derivatives of transition probabilities as well, you<br />obtain a different joint kernel from the one shown in<br />(16). How should we choose the subset of parameters to<br />take derivatives? More generally, you can derive a new<br />parameter as a function of a subset of original parameters.<br />How about taking the derivative with respect to the new<br />parameter?<br />As suggested in this example, one theoretical problem<br />about the Fisher kernel is that it depends not only on<br />the distribution itself, but also the parametrization which<br />a user has intentionally chosen. Consider two parametric<br />models:<br />p(x,h|θ), θ ∈ ?r1,<br />Let us assume that a joint distribution p(x,h) is repre-<br />sented by two different parametric models:<br />p(x,h|µ), µ ∈ ?r2, r1?= r2.<br />p(x,h) = p(x,h|ˆθ) = p(x,h|ˆ µ).<br />In general, the Fisher kernels derived from p(x|ˆθ) and<br />p(x|ˆ µ) are different although the underlying distribution<br />is the same‡. Since there is no admitted way to choose<br />proper parametrization so far, it is basically determined by<br />trial and error.<br />In (13) we represented the Fisher kernel as a function<br />of the joint kernel and posterior probabilities of hidden<br />variables. While the joint kernel is not invariant to<br />parametrization,<br />∇θlog p(x,h|ˆθ)?Z(ˆθ)−1∇θlog p(x?,h?|ˆθ)<br />?= ∇µlog p(x,h|ˆ µ)?Z(ˆ µ)−1∇µlog p(x?,h?|ˆ µ),<br />the posterior probabilities are invariant,<br />p(h|x,ˆθ) = p(h|x, ˆ µ).<br />Therefore choosing the parametrization amounts to<br />choosing the joint kernel. You may be able to derive the<br />joint kernel to fit your purpose by changing parametriza-<br />tion. However, in our opinion, this is an awkward and<br />indirect way.<br />In the Fisher kernel scheme, you have to control<br />two things (i.e., joint kernel and posterior probability)<br />simultanously by the choice of a parametric model. In our<br />opinion, there is no need to control them in such an unified<br />manner, because this scheme is sometimes too restrictive.<br />‡When there is one-to-one correspondence between two parameter spaces<br />aroundˆθ and ˆ µ, the Fisher kernel is invariant to parametrization because of<br />the Fisher information matrix (Jaakkola and Haussler, 1999). However it is<br />not the case in general.<br />S272</p>  <p>Page 6</p> <p>Marginalized kernels for biological sequences<br />For example, when you would like to incorporate the<br />second-order information into the Fisher kernel, you have<br />tousesecond-orderHMMs(Durbinetal.,1998).Sincethe<br />number of parameters of the second-order HMM is much<br />larger, it would be difficult to learn the parameters reliably<br />with a small sample set. This drawback is caused by the<br />fact that the joint kernel is tied to the probabilistic model.<br />In our approach, the joint kernel and the probabilistic<br />model are completely separated, so you can utilize second<br />order information with a first order HMM as in the second<br />order marginalized count kernel.<br />RESULTS<br />In this section, we illustrate the performance of marginal-<br />ized kernels in classification experiments using bacterial<br />gyrB amino acid sequences. gyrB—gyrase subunit B—is<br />a DNA topoisomerase (type II) which plays essential roles<br />in fundamental mechanisms of living organisms such as<br />DNA replication, transcription, recombination and repair<br />etc. One more important feature of gyrB is its capability<br />of being an evolutionary and taxonomic marker alternat-<br />ing popular 16S rRNA (Kasai et al., 1998). Our dataset<br />consists of 84 amino acid sequences of gyrB from five<br />genera in Actinobacteria which are Corynebacterium, My-<br />cobacterium, Gordonia, Nocardia and Rhodococcus, re-<br />spectively (Kasai et al., 2000). For brevity these genera<br />will be called genus 1 to 5, respectively. The number of se-<br />quences in each genus is listed as 9, 32, 15, 14 and 14. The<br />sequences are, by their nature, quite similar in terms of se-<br />quence similarity. Pairwise identity for each sequence is at<br />least 62 and 99% at most. For computing distance matrix<br />based on the sequence similarity, one can use the BLAST<br />scores (Altschul et al., 1990). However, since such scores<br />cannot directly be converted into positive semidefinite ker-<br />nels, kernel methods cannot be applied to them in princi-<br />ple.<br />In order to investigate how well the kernels re-<br />flect underlying genera, we performed two kinds of<br />experiments—clustering and supervised classification.<br />The following kernels are compared:<br />•<br />•<br />•<br />•<br />•<br />CK1: Count kernel (5)<br />CK2: Second-order count kernel (9)<br />FK: Fisher kernel (16)<br />MCK1: Marginalized count kernel (7)<br />MCK2: Second-order marginalized count kernel (11)<br />As the first experiment, K-Means clustering is per-<br />formed in feature spaces corresponding to kernels (see<br />M¨ uller et al. (2001) for details). The number of clusters<br />is determined as five (i.e., the true number). In FK and<br />MCKs, we used complete-connection HMMs with 3, 5<br />and 7 states. Note that FK is normalized by the sequence<br />lengths. In training HMMs, all 84 sequences are used.<br />One can also train HMMs in a classwise manner (Tsuda<br />et al., 2002). However, we did not do so because the<br />number of sequences is not large enough. For evaluating<br />clusters, we used the adjusted Rand index (ARI) (Yeung<br />and Ruzzo, 2001). The advantage of this index is that<br />you can compare two partitions whose number of clusters<br />are different. The ARI becomes 1 if the partitions are<br />completely correct. Also, the expectation of the ARI is 0<br />when partitions are randomly determined.<br />The kernel matrices by FK and MCKs are shown in<br />Figure 5. Additionally, the ideal kernel is shown for<br />reference,where K(x,x?)is1foranytwosequencesinthe<br />same genus, and −1 otherwise. Here, the number of HMM<br />states is three in all cases. For fair visualization, each<br />kernel matrix is normalized in the same manner: First,<br />the kernel matrix is ‘centralized’ as Kc := K − 1nK −<br />K1n+1nK1nwhere 1nis the n×n matrix whose elements<br />are all 1/n. Here n denotes the number of sequences, i.e.,<br />n = 84 in this experiment. Then, Kcis normalized by<br />the Frobenious norm as Kc/?Kc?F. As seen in the figure,<br />MCK2 is the best to recover the underlying structure. This<br />result is quantitatively shown by ARI in Figure 6, where<br />CK1 and CK2 correspond to the MCKs with only one<br />HMM state. Notably the Fisher kernel was worse than<br />MCK1, which shows that the joint kernel of the Fisher<br />kernel (16) is not appropriate for this task.<br />In order to see how genera are separated by introducing<br />the second order information and hidden variables, we<br />performed the following supervised classification experi-<br />ments as well. First, we pick up two genera out of three<br />genera (3,4,5). Genera 1 and 2 were not used because<br />they can be separated easily by all kernels. The sequences<br />of two genera are randomly divided into 25% training<br />and 75% testing samples. Kernels are compared due to<br />the test error by the kernel Fisher discriminant analysis<br />(KFDA) (Roth and Steinhage, 2000), which compares<br />favorably with the SVM in many benchmarks. Note<br />that the regularization parameter ? of KFDA (Roth and<br />Steinhage, 2000) is determined such that the test error is<br />minimized§. The test errors of five kernels are shown in<br />Table 1. The second order kernels (i.e., CK2 and MCK2)<br />were significantly better than the first order kernels. This<br />result coincides with the common understanding that<br />higher order information of protein sequences is essential<br />for classification and structure prediction (e.g., Asai et<br />al. (1993)). Comparing CK2 and MCK2, MCK2 always<br />performed better, which indicates that incorporating<br />hidden variables (i.e., context information) is meaningful<br />at least in this task.<br />§Forregularizationparameter?,10equallyspacedpointsonthelogscaleare<br />taken from [10−4,102]. Among these candidates, the optimal one is chosen.<br />S273</p>  <p>Page 7</p> <p>K.Tsuda et al.<br />1  <br />0. 5<br />0   <br />0.5 <br />1   <br />Ideal<br /> 0.02<br />0    <br />0.02 <br />0.04 <br />FK<br /> 0.02<br />0    <br />0.02 <br />0.04 <br />MCK1<br /> 0.02<br />0    <br />0.02 <br />0.04 <br />MCK2<br />Fig. 5. (Upperleft) Ideal kernel matrix to illustrate the true clusters. (Upperright) Kernel matrix of the Fisher kernel. (Lowerleft) Kernel<br />matrix of the first-order marginalized count kernel. (Lowerright) Kernel matrix of the second-order marginalized count kernel.<br />Table 1. Mean error rates (%) of supervised classification between two bacterial genera ([·] shows the standard deviation). The best result in each task is<br />written in bold face<br />GeneraCK1CK2 FKMCK1 MCK2<br />3–4<br />3–5<br />4–5<br />24.5 [9.67]<br />12.7 [8.93]<br />25.6 [13.0]<br />9.10 [7.87]<br />6.43 [7.76]<br />13.5 [15.5]<br />10.4 [9.15]<br />10.9 [10.1]<br />23.1 [14.3]<br />12.8 [9.85]<br />10.4 [8.17]<br />20.0 [14.6]<br />8.48 [7.76]<br />5.71 [7.72]<br />11.6 [14.6]<br />CONCLUSION<br />In this paper, we proposed marginalized kernels, which<br />provide a new reasonable way to design a kernel from<br />latent variable models. The Fisher kernel was described as<br />a special case of marginalized kernels, which have added a<br />new aspect to the Fisher kernel theory. Finally, we showed<br />that marginalized count kernels perform well in protein<br />classification experiments.<br />Our work provides a general framework from which<br />diverse kernels are expected to be constructed. In future<br />works, we would like to derive useful kernels as many as<br />possible not only in bioinformatics but also in other areas<br />in information technology.<br />ACKNOWLEDGEMENT<br />The authors gratefully acknowledge that the bacterial<br />gyrB amino acid sequences are offered by courtesy<br />of Identification and Classification of Bacteria (ICB)<br />S274</p>  <p>Page 8</p> <p>Marginalized kernels for biological sequences<br />1357<br />0.2<br />0.3<br />0.4<br />0.5<br />0.6<br />0.7<br />0.8<br />0.9<br />1<br />Number of HMM States<br />ARI<br />FK<br />MCK1<br />MCK2<br />Fig. 6. Evaluation of kernels in clustering in terms of the adjusted<br />Rand index (ARI). The x-axis corresponds to the number of states<br />in HMM, from which the kernels are derived. The Fisher kernel<br />(FK), the marginalized count kernels of first-order (MCK1) and<br />second-order (MCK2) are compared. Note that the count kernels<br />of first-order (CK1) and second-order (CK2) correspond to MCK1<br />and MCK2 at one HMM state, respectively.<br />database team (Watanabe et al., 2001). The authors<br />would like to thank S.-I.Amari, K.-R.M¨ uller, G.R¨ atsch,<br />M.Kawanabe, H.Nakahara and S.Sonnenburg for fruitful<br />discussions.<br />REFERENCES<br />Altschul,S., Gish,W., Myers,E. and Lipman,D. (1990) Basic local<br />alignment search tool. J. Mol. Biol., 215, 403–410.<br />Asai,K., Hayamizu,S. and Handa,K. (1993) Prediction of protein<br />secondary structure by the hidden Markov model. CABIOS<br />(currently Bioinformatics), 9, 141–146.<br />Durbin,R., Eddy,S., Krogh,A. and Mitchison,G. (1998) Biological<br />Sequence Analysis: Probabilistic Models of Proteins and Nucleic<br />Acids. Cambridge University Press.<br />Frakes,W. and Baeza-Yates,R., (eds.) (1992) Information Retrieval:<br />Data Structure and Algorithms. Prentice Hall.<br />Haussler,D. (1999) Convolution kernels on discrete structures.<br />Technical Report UCSC-CRL-99-10. UC Santa Cruz.<br />Jaakkola,T., Diekhans,M. and Haussler,D. (2000) A discriminative<br />framework for detecting remote protein homologies. J. Comp.<br />Biol., 7, 95–114.<br />Jaakkola,T. and Haussler,D. (1999) Exploiting generative mod-<br />els in discriminative classifiers. In Kearns,M., Solla,S. and<br />Cohn,D. (eds), Advances in Neural Information Processing Sys-<br />tems 11. pp. 487–493.<br />Karchin,R., Karplus,K. and Haussler,D. (2002) Classifying G-<br />protein coupled receptors with support vector machines. Bioin-<br />formatics, 18, 147–159.<br />Kasai,H.,Bairoch,A., Watanabe,K.,<br />Gasteiger,E. and Yamamoto,S. (1998) Construction of the gyrB<br />database for the identification and classification of bacteria.<br />Genome Informatics 1998. Universal Academic Press, pp.<br />13–21.<br />Kasai,H., Ezaki,T. and Harayama,S. (2000) Differentiation of<br />phylogenetically related slowly growing mycrobacteria by their<br />gyrB sequences. J. Clin. Microbiol., 38, 301–308.<br />M¨ uller,K.-R., Mika,S., R¨ atsch,G., Tsuda,K. and Sch¨ olkopf,B.<br />(2001) An introduction to kernel-based learning algorithms.<br />IEEE Trans. Neural Networks, 12, 181–201.<br />Pavlidis,P.,Furey,T.,Liberto,M.,Haussler,D.andGrundy,W.(2001)<br />Promotor region-based classification of genes. In Proceedings<br />PSB 2001. pp. 151–163.<br />Rabiner,L. (1989) A tutorial on hidden Markov models and<br />selected applications in speech recognition. Proc. IEEE, 77, 257–<br />285.<br />Roth,V.andSteinhage,V.(2000)Nonlineardiscriminantanalysisus-<br />ing kernel functions. In Solla,S., Leen,T. and M¨ uller,K.-R. (eds),<br />Advances in Neural Information Processing Systems 12. MIT<br />Press, pp. 568–574.<br />Tipping,M. (1999) Deriving cluster analytic distance functions<br />from gaussian mixture models. In Willshaw,D. and Mur-<br />ray,A. (eds), Proceedings of ICANN’99. IEE Press, pp. 815–820.<br />Tsuda,K., Kawanabe,M., R¨ atsch,G., Sonnenburg,S. and M¨ uller,K.-<br />R.(2002)Anewdiscriminativekernelfromprobabilisticmodels.<br />Neural Comput., In press.<br />Watanabe,K., Nelson,J., Harayama,S. and Kasai,H. (2001) ICB<br />database: the gyrB database for identification and classification<br />of bacteria. Nucleic Acids Res., 29, 344–345.<br />Yeung,K. and Ruzzo,W. (2001) Principal component analysis for<br />clustering gene expression data. Bioinformatics, 17, 763–774.<br />Isono,K.,Harayama,S.,<br />S275</p>  <a href="https://www.researchgate.net/profile/Koji_Tsuda/publication/11216584_Marginalized_kernels_for_biological_sequences/links/00b7d52705ee89c23a000000.pdf">Download full-text</a> </div> <div id="rgw21_56ab1fd7ae687" class="c-box pub-resource-container js-toggle" style=""> <h4 class="lf js-expand-list"><small><a href="javascript:" class="lf">View other sources <span class="ico-expand-list"></span></a></small></h4> <h4 class="lf js-collapse-list hidden"><small><a href="javascript:" class="lf">Hide other sources <span class="ico-collapse-list"></span></a></small></h4> <div class="clear"></div> <div class="scroll-wrapper hidden"> <ul class="files-list c-list" id="rgw22_56ab1fd7ae687">  <li class="c-list-item pub-resource-item"  data-type="fulltext" id="rgw23_56ab1fd7ae687"> <span class="ico-pub"></span> <div class="file-content"> <div class="download"> <a href="profile/Koji_Tsuda/publication/11216584_Marginalized_kernels_for_biological_sequences/links/00b7d52705ee89c23a000000.pdf" class="publication-viewer" title="S268.full.pdf">S268.full.pdf</a> </div>  <div class="details">  <span> Available from <a href="profile/Koji_Tsuda">Koji Tsuda</a> &middot; Jan 20, 2016 </span>   </div>    </div> </li>  <li class="c-list-item pub-resource-item"  data-type="fulltext" id="rgw24_56ab1fd7ae687"> <span class="ico-pub"></span> <div class="file-content"> <div class="download"> <a href="http://bioinformatics.oxfordjournals.org/content/18/suppl_1/S268.full.pdf" target="_blank" rel="nofollow" class="publication-viewer" title="Marginalized kernels for biological sequences">Marginalized kernels for biological sequences</a> </div>  <div class="details">   Available from <a href="http://bioinformatics.oxfordjournals.org/content/18/suppl_1/S268.full.pdf" target="_blank" rel="nofollow">oxfordjournals.org</a>  </div>    </div> </li>  </ul> </div> </div> </noscript> <div class="clearfix"> <div class="action-container">  </div> <div class="pub-legal"> Data provided are for informational purposes only. Although carefully collected, accuracy cannot be guaranteed. The impact factor represents a rough estimation of the journal's impact factor and does not reflect the actual current impact factor. Publisher conditions are provided by RoMEO. Differing provisions from the publisher's actual policy or licence agreement may be applicable. </div>  <div id="rgw31_56ab1fd7ae687" class="citations-container"> <div class="tab-container"> <ul class="tab-list"> <li class="lf tab-item  js-citations"> <a href="javascript:void(0);" class="tab-link"> References  </small> </a> </li>  <li class="lf tab-item tab-item-active js-cited-in js-cited-in-tooltip"> <a href="javascript:void(0);" class="tab-link"> Cited In <small> (139) </small> </a> </li>    <li class="rf"> <div class="dropdown js-citations-sorter dropdown-right-align" style="position: relative; bottom: -1px;display:none;"> <a href="javascript:void(0);" class="dropdown-toggle"> Sorted by: <strong class="js-current-sorting"> Order of availability  </strong> <span class="caret"></span> </a> <ul class="dropdown-menu"> <li><a href="javascript:void(0);" data-sort="normal">Order of availability</a></li> <li><a href="javascript:void(0);" data-sort="original">Appearance in publication</a></li> </ul> </div> </li>  </ul> <div class="tab-section tab-section-active js-citations-list-container"> <div id="rgw32_56ab1fd7ae687" class="pub-citations-list">  <ul class="c-list">  <li class="c-list-item li-publication   includes-citation-list"  id="rgw33_56ab1fd7ae687" >  <div class="indent-left">  <div id="rgw34_56ab1fd7ae687" class="js-publication-item-fulltext fulltext-thumb">    <a class="publication-preview ga-publication-viewer js-publication-item-fulltext-content" href="publication/273125269_Indefinite_kernel_ridge_regression_and_its_application_on_QSAR_modelling">       </a>   <div class="preview-source-info"> <a class="details js-show-source ga-source-url" href="profile/Benjamin_Li" >Source</a>  <div class="tooltip-content" style="display: none"> Available from: Benjamin Yee Shing Li </div> </div>   </div>  </div>  <div class="indent-right">      </div>  <ul class="citation-contexts" id="rgw35_56ab1fd7ae687">  <li class="citation-context-item"> "However , for some kinds of data such as structural objects like 3D conformations of molecular structures and DNA sequences, features are hard to be extracted. To apply kernelized method over these structural objects, a significant level of efforts is devoted to the design of customized kernels such as graph kernels and string kernels [9] [10] [11]. " </li>  </ul>   <div  style="margin-top: -2px">  <h5 class="pub-type-and-title">  <span class="publication-type">Article:</span>    <a class="js-publication-title-link js-go-to-publication ga-publication-item" href="publication/273125269_Indefinite_kernel_ridge_regression_and_its_application_on_QSAR_modelling"> <span class="publication-title js-publication-title">Indefinite kernel ridge regression and its application on QSAR modelling</span> </a>     </h5>  </div>    <div class="authors">     <a href="researcher/85205020_Benjamin_Yee_Shing_Li" class="authors js-author-name ga-publications-authors">Benjamin Yee Shing Li</a> &middot;     <a href="researcher/14503873_Lam_Fat_Yeung" class="authors js-author-name ga-publications-authors">Lam Fat Yeung</a> &middot;     <a href="researcher/74904926_King_Tim_Ko" class="authors js-author-name ga-publications-authors">King Tim Ko</a>      </div>        <div class="abstract"> <span class="shorten"> <a href="javascript:" class="js-toggle-abstract">[Show abstract]</a> </span> <span class="full"> <a href="javascript:" class="js-toggle-abstract">[Hide abstract]</a><br/>  <strong>ABSTRACT:</strong> Recently, the use of indefinite kernels in machine learning has attracted numerous attentions. However most works are focused on the classification techniques and less are devoted to regression models. In this paper to adapt indefinite kernels to ridge regression model, an indefinite kernel ridge regression model is proposed. Instead of performing spectral transformation on the kernel matrix, a less restrictive semi-definite proxy kernel can be constructed to approximate the kernel which normally is positive semi-definite. The sensitivity of the distance between this indefinite kernel and the proxy kernel is controlled by a parameter ρ. </span> </div>    <div class="publication-meta publication-meta">   <span class="ico-publication-fulltext reset-background"></span> Full-text   &middot; Article &middot; Feb 2015  &middot; Neurocomputing  </div>        <div class="publication-actions"> <div class="btn-group">  <a class="btn btn-plain action-download primary  open-viewer" href="profile/Benjamin_Li/publication/273125269_Indefinite_kernel_ridge_regression_and_its_application_on_QSAR_modelling/links/54f80ab20cf210398e93ea29.pdf?origin=publication_list">  <span class="js-btn-label">Download</span> </a>    </div> </div>      </li>  <li class="c-list-item li-publication   includes-citation-list"  id="rgw36_56ab1fd7ae687" >  <div class="indent-left">  <div id="rgw37_56ab1fd7ae687" class="js-publication-item-fulltext fulltext-thumb">    <a class="publication-preview ga-publication-viewer js-publication-item-fulltext-content" href="publication/254257268_Discriminative_Feature_Fusion_for_Image_Classification">       </a>   <div class="preview-source-info"> <a class="details js-show-source ga-source-url" href="profile/Basura_Fernando" >Source</a>  <div class="tooltip-content" style="display: none"> Available from: Basura Fernando </div> </div>   </div>  </div>  <div class="indent-right">      </div>  <ul class="citation-contexts" id="rgw38_56ab1fd7ae687">  <li class="citation-context-item"> "We claim that this kernel is not well suited to our logistic regression context . In the following, we rather make use of the information provided by our model to design a new marginalized kernel [14] [16], which is defined in its original form as fol- lows: " </li>  </ul>   <div  style="margin-top: -2px">  <h5 class="pub-type-and-title">  <span class="publication-type">Conference Paper:</span>    <a class="js-publication-title-link js-go-to-publication ga-publication-item" href="publication/254257268_Discriminative_Feature_Fusion_for_Image_Classification"> <span class="publication-title js-publication-title">Discriminative Feature Fusion for Image Classification</span> </a>     </h5>  </div>    <div class="authors">     <a href="researcher/70701639_Basura_Fernando" class="authors js-author-name ga-publications-authors">Basura Fernando</a> &middot;     <a href="researcher/15519860_Elisa_Fromont" class="authors js-author-name ga-publications-authors">Elisa Fromont</a> &middot;     <a href="researcher/70568361_Damien_Muselet" class="authors js-author-name ga-publications-authors">Damien Muselet</a> &middot;     <a href="researcher/10762347_Marc_Sebban" class="authors js-author-name ga-publications-authors">Marc Sebban</a>      </div>        <div class="abstract"> <span class="shorten"> <a href="javascript:" class="js-toggle-abstract">[Show abstract]</a> </span> <span class="full"> <a href="javascript:" class="js-toggle-abstract">[Hide abstract]</a><br/>  <strong>ABSTRACT:</strong> Bag-of-words-based image classification approaches mostly rely on low level local shape features. However, it has been shown that combining multiple cues such as color, texture, or shape is a challenging and promising task which can improve the classification accuracy. Most of the state-of-the-art feature fusion methods usually aim to weight the cues without considering their statistical dependence in the application at hand. In this paper, we present a new logistic regression-based fusion method, called LRFF, which takes advantage of the different cues without being tied to any of them. We also design a new marginalized kernel by making use of the output of the regression model. We show that such kernels, surprisingly ignored so far by the computer vision community, are particularly well suited to achieve image classification tasks. We compare our approach with existing methods that combine color and shape on three datasets. The proposed learning-based feature fusion process clearly outperforms the state-of-the art fusion methods for image classification. </span> </div>    <div class="publication-meta publication-meta">   <span class="ico-publication-fulltext reset-background"></span> Full-text   &middot; Conference Paper &middot; Jun 2012  </div>        <div class="publication-actions"> <div class="btn-group">  <a class="btn btn-plain action-download primary  open-viewer" href="profile/Basura_Fernando/publication/254257268_Discriminative_Feature_Fusion_for_Image_Classification/links/004635293c79ddb86a000000.pdf?origin=publication_list">  <span class="js-btn-label">Download</span> </a>    </div> </div>      </li>  <li class="c-list-item li-publication   includes-citation-list"  id="rgw39_56ab1fd7ae687" >  <div class="indent-left">  <div id="rgw40_56ab1fd7ae687" class="js-publication-item-fulltext fulltext-thumb">    <a class="publication-preview ga-publication-viewer js-publication-item-fulltext-content" href="publication/224254886_Hybrid_generative-discriminative_classification_using_posterior_divergence">       </a>   <div class="preview-source-info"> <a class="details js-show-source ga-source-url" href="deref/http%3A%2F%2Fwww.cnbc.cmu.edu%2F%7Etai%2Fpapers%2FLi_PosDiv_fixed.pdf" target="_blank" >Source</a>  <div class="tooltip-content" style="display: none"> Available from: cmu.edu </div> </div>   </div>  </div>  <div class="indent-right">      </div>  <ul class="citation-contexts" id="rgw41_56ab1fd7ae687">  <li class="citation-context-item"> "ˆ y = sign(P(y = +1 | x, θ) − 1/2) is generally inferior to discriminative models which are more powerful in capturing decision boundaries among different classes and more widely used in recognition tasks. At present, several hybrid generative-discriminative schemes have been proposed to combine the strengths of these two classes of models in a number of applications, from scene classification [3], object recognition [5], speech recognition [19] to biological sequence analysis [6] [21], resulting in state-of-the-art performance. These hybrid schemes sought to integrate the intra-class information from generative models and the complementary inter-class information from discriminative methods. " </li>  </ul>   <div  style="margin-top: -2px">  <h5 class="pub-type-and-title">  <span class="publication-type">Conference Paper:</span>    <a class="js-publication-title-link js-go-to-publication ga-publication-item" href="publication/224254886_Hybrid_generative-discriminative_classification_using_posterior_divergence"> <span class="publication-title js-publication-title">Hybrid generative-discriminative classification using posterior divergence</span> </a>     </h5>  </div>    <div class="authors">     <a href="researcher/69821672_Xiong_Li" class="authors js-author-name ga-publications-authors">Xiong Li</a> &middot;     <a href="researcher/10726208_Tai_Sing_Lee" class="authors js-author-name ga-publications-authors">Tai Sing Lee</a> &middot;     <a href="researcher/11760291_Yuncai_Liu" class="authors js-author-name ga-publications-authors">Yuncai Liu</a>      </div>        <div class="abstract"> <span class="shorten"> <a href="javascript:" class="js-toggle-abstract">[Show abstract]</a> </span> <span class="full"> <a href="javascript:" class="js-toggle-abstract">[Hide abstract]</a><br/>  <strong>ABSTRACT:</strong> Integrating generative models and discriminative models in a hybrid scheme has shown some success in recognition tasks. In such scheme, generative models are used to derive feature maps for outputting a set of fixed length features that are used by discriminative models to perform classification. In this paper, we present a method, called posterior divergence, to derive feature maps from the log likelihood function implied in the incremental expectation-maximization algorithm. These feature maps evaluate a sample in three complementary measures: (1) how much the sample affects the model; (2) how well the sample fits the model; (3) how uncertain the fit is. We prove that the linear classification error rate using the outputs of the derived feature maps is at least as low as that of plug-in estimation. We present efficient algorithms for computing these feature maps for semi-supervised learning and supervised learning. We evaluate the proposed method on three typical applications, i.e. scene recognition, face and non-face classification and protein sequence analysis, and demonstrate improvements over related methods. </span> </div>    <div class="publication-meta publication-meta">  <span class="ico-publication-preview reset-background"></span> Preview    &middot; Conference Paper &middot; Jul 2011  </div>        <div class="publication-actions"> <div class="btn-group">  <a class="btn btn-plain action-request-external  " href="javascript:;" data-context="pubCit">  <span class="js-btn-label">Request full-text</span> </a>    </div> </div>      </li>  </ul>    <a class="show-more-rebranded js-show-more rf text-gray-lighter">Show more</a> <span class="ajax-loading-small list-loading" style="display: none"></span>  <div class="clearfix"></div>  <div class="publication-detail-sidebar-legal">Note: This list is based on the publications in our database and might not be exhaustive.</div> <div class="clearfix"></div>  </div> </div> </div> </div> </div> </div> <div class="clearfix">     <div id="rgw26_56ab1fd7ae687" class="c-box pub-resource-container js-toggle" style="display: none;"> <h4 class="lf js-expand-list"><small><a href="javascript:" class="lf">View other sources <span class="ico-expand-list"></span></a></small></h4> <h4 class="lf js-collapse-list hidden"><small><a href="javascript:" class="lf">Hide other sources <span class="ico-collapse-list"></span></a></small></h4> <div class="clear"></div> <div class="scroll-wrapper hidden"> <ul class="files-list c-list" id="rgw27_56ab1fd7ae687">  </ul> </div> </div>   <div id="rgw17_56ab1fd7ae687" class="similar-publications"> <h2>Similar Publications</h2> <ul class="list-bordered">  <li class="c-list-item li-publication-teaser" id="rgw18_56ab1fd7ae687"> <div> <h5> <a href="publication/289587194_Fast_Kronecker_product_kernel_methods_via_sampled_vec_trick" class="color-inherit ga-similar-publication-title"><span class="publication-title">Fast Kronecker product kernel methods via sampled vec trick</span></a>  </h5>  <div class="authors"> <a href="researcher/13960335_Antti_Airola" class="authors ga-similar-publication-author">Antti Airola</a>, <a href="researcher/39244917_Tapio_Pahikkala" class="authors ga-similar-publication-author">Tapio Pahikkala</a>  </div>  </div> <div class="clear"></div> </li>  <li class="c-list-item li-publication-teaser" id="rgw19_56ab1fd7ae687"> <div> <h5> <a href="publication/282904456_A_new_efficient_hybrid_intelligent_method_for_nonlinear_dynamical_systems_identification_The_Wavelet_Kernel_Fuzzy_Neural_Network" class="color-inherit ga-similar-publication-title"><span class="publication-title">A new efficient hybrid intelligent method for nonlinear dynamical systems identification: The Wavelet Kernel Fuzzy Neural Network</span></a>  </h5>  <div class="authors"> <a href="researcher/2049817612_Hichem_Loussifi" class="authors ga-similar-publication-author">Hichem Loussifi</a>, <a href="researcher/70687948_Khaled_Nouri" class="authors ga-similar-publication-author">Khaled Nouri</a>, <a href="researcher/17902406_Naceur_Benhadj_Braiek" class="authors ga-similar-publication-author">Naceur Benhadj Braiek</a>  </div>  </div> <div class="clear"></div> </li>  <li class="c-list-item li-publication-teaser" id="rgw20_56ab1fd7ae687"> <div> <h5> <a href="publication/283161063_Diagnosing_Tropical_Cyclone_Rapid_Intensification_Using_Kernel_Methods_and_Reanalysis_Datasets" class="color-inherit ga-similar-publication-title"><span class="publication-title">Diagnosing Tropical Cyclone Rapid Intensification Using Kernel Methods and Reanalysis Datasets</span></a>  </h5>  <div class="authors"> <a href="researcher/70659180_Andrew_Mercer" class="authors ga-similar-publication-author">Andrew Mercer</a>, <a href="researcher/2075534530_Alexandria_Grimes" class="authors ga-similar-publication-author">Alexandria Grimes</a>  </div>  </div> <div class="clear"></div> </li>  </ul> </div> </div> </div> </div> </div></div></div>
<div class="clear"></div><div id="rgw51_56ab1fd7ae687" class="default-footer"> <div id="footer" class="clearfix"> <span class="footer-left"> &copy; 2008&dash;2016 researchgate.net. All rights reserved. </span> <span class="footer-right"> <a href="https://www.researchgate.net/about">About us</a><span class="footer-link-separator">&nbsp;&middot;&nbsp;</span> <a href="https://www.researchgate.net/contact">Contact us</a><span class="footer-link-separator">&nbsp;&middot;&nbsp;</span> <a href="careers">Careers</a><span class="footer-link-separator">&nbsp;&middot;&nbsp;</span> <a href="developers">Developers</a><span class="footer-link-separator">&nbsp;&middot;&nbsp;</span> <a href="blog" target="_blank">News</a><span class="footer-link-separator">&nbsp;&middot;&nbsp;</span> <a href="application.PrivacyPolicy.html">Privacy</a><span class="footer-link-separator">&nbsp;&middot;&nbsp;</span> <a href="application.TermsAndConditions.html">Terms</a> <span class="footer-link-separator" style="padding: 0 5px;">&nbsp;|&nbsp;</span> <a href="advertising?_ref=ft">Advertising</a> <span class="footer-link-separator">&nbsp;&middot;&nbsp;</span> <a href="recruiters?_ref=ft">Recruiting</a> </span> </div>  </div></div>
<div id="rgw52_56ab1fd7ae687">  <div class="header-wrapper-logged-out"> <div id="header"> <div class="header-content"> <a href="" class="g-l-logo"> <svg width="149" height="19"> <image xlink:href="https://c5.rgstatic.net/m/235107188705592/images/template/brand-header-logo.svg" src="https://c5.rgstatic.net/m/238113351022438/images/template/brand-header-logo.png" width="149" height="19"/> </svg> </a> <a href="" class="g-l-logo-ico"> <svg width="127" height="17" viewbox="0 0 127 22"> <image xlink:href="https://c5.rgstatic.net/m/235107188705592/images/template/brand-header-logo.svg" src="https://c5.rgstatic.net/m/238113351022438/images/template/brand-header-logo.png" width="127" height="17"/> </svg> </a> <div id="rgw53_56ab1fd7ae687" class="header-login-wrapper js-header-login"> <div class="dropdown-right-align"> <div class="login-signup-container lf"> <a href="https://www.researchgate.net/signup.SignUp.html?ev=su_chnl_index&amp;hdrsu=1&amp;_sg=ZtAnySYxo0IbqSufCsWjxcuY7_T5bahbTyyTtv3nLx7Uxi05lqeh4-5PnK2gQ6Zi" class="dropdown-toggle lf">Join for free</a> </div> <div class="dropdown lf" style="height: 20px;"> <a href="https://www.researchgate.net/application.Login.html" class="js-login-url dropdown-toggle lf">Log in <span class="caret"></span></a> <div class="dropdown-menu"> <div class="header-login-form-wrapper"> <!--[if IE 6]><p class="box-warning" style="margin-bottom: 0;">Sorry, ResearchGate no longer supports the version of Internet Explorer you are using. <a href="http://whatbrowser.org/" rel="nofollow" target="_blank">Update your web browser</a> and then log in. </p><![endif]--> <form method="post" action="https://www.researchgate.net/application.Login.html" class="form-big header-login-form js-login-form" name="loginForm" id="headerLoginForm"> <input type="hidden" name="request_token" value="B8gSzaHs4UiHGqPUieODnKn1JuMV+BZ6n2X7Q73ZsjmyzxTuZYQiiZYwXpANKMRPLVqcNsDRqy78PiOZIGReYH5YwsAU7TiwKxgikwt5cPcziuvWbgY++hAaZqU86u6DcoCmYh6i9JQV6Kst/inoX5v7OClYI8SzOVml5rguz0UTcJJDvrI4IKT3FoCWU1O6nhnlphi6SX/mk2SZUsw8ASIS7m2Ceobgqnk2NT2993qDvz5hx8OUBT7EQnO131KsT8OAOwa5Hcqagkyr+dI5J8lH575dnFv3HYQokX7WPSU="/> <input type="hidden" name="urlAfterLogin" value="publication/11216584_Marginalized_kernels_for_biological_sequences"/> <input type="hidden" name="invalidPasswordCount" value="0"/> <input type="hidden" name="headerLogin" value="yes"/> <label for="input-header-login">Email</label> <div class="login-input"> <div class="info-tip-wrapper"> <span class="ico-info js-info"></span> </div> <input type="email" value="" name="login" class="login js-login-input text" id="input-header-login" tabindex="1"/> </div> <div class="clear"></div> <label class="lf" for="input-header-password"> Password </label> <a class="rf forgot-password js-forgot-password" href="application.LostPassword.html">Forgot password?</a> <div class="clear"></div> <input type="password" value="" name="password" class="password js-password-input text" id="input-header-password" tabindex="2"/> <div class="clear"></div> <label class="remember-me" for="headerLoginCookie"> <input type="checkbox" checked="checked" value="yes" name="setLoginCookie" class="lf checkbox" id="headerLoginCookie" tabindex="3"/> Keep me logged in </label> <div class="clear"></div> <input value="Log in" name="loginSubmit" class="btn btn-promote btn-fullwidth btn-large allow-leave js-submit-button" type="submit" tabindex="4"/> </form> <div class="connectors"> <div class="text">or log in with</div> <div class="connector-actions"> <a href="connector/linkedin/" class="li-connect js-li-connect" data-redirect-url="cHVibGljYXRpb24vMTEyMTY1ODRfTWFyZ2luYWxpemVkX2tlcm5lbHNfZm9yX2Jpb2xvZ2ljYWxfc2VxdWVuY2Vz"> <span class="icon ico-linkedin-round-grey"></span> <span class="icon ico-linkedin-round"></span> </a> <a href="connector/facebook/" class="fb-connect middle js-fb-connect" data-redirect-url="cHVibGljYXRpb24vMTEyMTY1ODRfTWFyZ2luYWxpemVkX2tlcm5lbHNfZm9yX2Jpb2xvZ2ljYWxfc2VxdWVuY2Vz"> <span class="icon ico-facebook-round-grey"></span> <span class="icon ico-facebook-round"></span> </a> <a href="connector/google/" class="g-connect js-g-connect" data-redirect-url="cHVibGljYXRpb24vMTEyMTY1ODRfTWFyZ2luYWxpemVkX2tlcm5lbHNfZm9yX2Jpb2xvZ2ljYWxfc2VxdWVuY2Vz"> <span class="icon ico-google-round-grey"></span> <span class="icon ico-google-round"></span> </a> </div> </div> </div> </div> </div> </div> </div> </div> </div> <script type="application/ld+json">
{ "@context" : "http://schema.org",
  "@type" : "Organization",
  "name" : "ResearchGate",
  "url" : "http://www.researchgate.net",
  "logo" : "http://www.researchgate.net/images/template/rg_logo_square_brand.png",
  "sameAs" : [ "https://www.facebook.com/ResearchGate",
    "https://twitter.com/ResearchGate",
    "https://plus.google.com/+researchgate",
    "https://www.linkedin.com/company/researchgate"] 
}
</script> </div> </div><div class="c-signup-bar" id="rgw54_56ab1fd7ae687"> <div class="banner-contents">   <span class="message">ResearchGate is the professional network for scientists and researchers.</span> <a href="signup.SignUp.html?ev=su_banner" class="btn btn-large btn-promote">Join for free</a>  </div> </div></div>
<script>
rgConfig.backendTime = 640;
</script>
<script src="//c5.rgstatic.net/m/2277196935388619/javascript/lib/yui3/yui/yui-min.js" type="text/javascript"></script>
        <script src="//c5.rgstatic.net/m/2448732603281275/javascript/yuiLoaderConfig-min.js" type="text/javascript"></script>
        <script src="//c5.rgstatic.net/m/23178613132105398/javascript/vendor/babel-core/browser-polyfill.min.js" type="text/javascript"></script>
<script>
(function (){
if (typeof YRG === "undefined") {
var xmlHttpRequest = new XMLHttpRequest();
xmlHttpRequest.open("post", "go.Error.html");
xmlHttpRequest.setRequestHeader("Content-Type", "application/json");
xmlHttpRequest.setRequestHeader("Accept", "application/json"); var loadedScripts = "";
if (window.performance && window.performance.getEntriesByType) {
    var result = [];
    var resources = performance.getEntriesByType("resource");
    for (var i in resources) {
        if (resources.hasOwnProperty(i)) {
            result.push({
                name: resources[i].name,
                duration: resources[i].duration
            });
        }
    }
    loadedScripts += "&loadedScripts=" + encodeURIComponent(JSON.stringify(result));
}
if (typeof YUI === "undefined") {
    loadedScripts += "&yuiLoaded=false";
} else {
    loadedScripts += "&yuiLoaded=true";
}
xmlHttpRequest.send("Type=InformationException&message=" + encodeURIComponent("Error loading YUI") + loadedScripts);
}
})();
</script>
<script>if (typeof YRG !== 'undefined') { YRG.use('rg-base',function(Y){Y.applyConfig({ignore: ["css-rg","css-rg2","css-ie","css-modules-publicprofile","css-pow-publicliterature-FollowPublicationPromo","css-pow-application-PdfJsReader","css-pow-publicliterature-PublicationInlineReader"]});Y.use(["rg.core.pagespeed.Monitoring"],function(Y){(function(){Y.rg.createInitialWidget({"data":{"content":{"data":{"profileSmallHeader":{"data":{"profileBadge":{"data":{"fullname":"Koji Tsuda","imageUrl":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/AS%3A272439313694743%401441966002280_m\/Koji_Tsuda.png","profileStats":[],"profileFollowButton":null,"profileReputationScoreNumber":null,"profileUrl":"profile\/Koji_Tsuda","institution":"The University of Tokyo","institutionUrl":false,"widgetId":"rgw4_56ab1fd7ae687"},"id":"rgw4_56ab1fd7ae687","partials":[],"templateName":"publicprofile\/stubs\/ProfileBadge.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicprofile.ProfileBadge.html?accountId=1854273","viewClass":null,"yuiModules":["css-modules-publicprofile"],"stylesheets":["modules\/publicprofile.css"],"_isYUI":true},"widgetId":"rgw3_56ab1fd7ae687"},"id":"rgw3_56ab1fd7ae687","partials":[],"templateName":"publicprofile\/stubs\/ProfileSmallHeader.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicprofile.ProfileSmallHeader.html?publicationUid=11216584","viewClass":null,"yuiModules":["css-modules-publicprofile"],"stylesheets":["modules\/publicprofile.css"],"_isYUI":true},"publication":{"data":{"publicationUid":11216584,"title":"Marginalized kernels for biological sequences","journalTitle":"Bioinformatics","journalDetailsTooltip":{"data":{"journalTitle":"Bioinformatics","journalAbbrev":"BIOINFORMATICS","publisher":"Oxford University Press (OUP)","issn":"1367-4803","impactFactor":"4.98","fiveYearImpactFactor":"8.14","citedHalfLife":"6.90","immediacyIndex":"1.17","eigenFactor":"0.20","articleInfluence":"3.57","widgetId":"rgw6_56ab1fd7ae687"},"id":"rgw6_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/JournalInfo.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.JournalInfo.html?issn=1367-4803","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"affiliation":"Computational Biology Research Center, National Institute of Advanced Industrial Science and Technology (AIST), 2-41-6 Aomi Koto-ku, Tokyo 135-0064, Japan. ","type":"Article","details":{"doi":"10.1093\/bioinformatics\/18.suppl_1.S268","journalInfos":{"journal":"","publicationDate":"02\/2002;","publicationDateRobot":"2002-02","article":"18 Suppl 1(Suppl 1):S268-75.","journalTitle":"Bioinformatics","journalUrl":"journal\/1367-4803_Bioinformatics","impactFactor":4.98}},"source":{"sourceUrl":"http:\/\/www.ncbi.nlm.nih.gov\/pubmed\/12169556","sourceName":"PubMed"},"publicationActions":null,"publicationCoins":{"data":{"tags":[{"key":"ctx_ver","value":"Z39.88-2004"},{"key":"rft_val_fmt","value":"info:ofi\/fmt:kev:mtx:journal"},{"key":"rfr_id","value":"info:sid\/researchgate.net:researchgate"},{"key":"rft_id","value":"info:doi\/10.1093\/bioinformatics\/18.suppl_1.S268"},{"key":"rft.atitle","value":"Marginalized kernels for biological sequences"},{"key":"rft.title","value":"Bioinformatics (Oxford, England)"},{"key":"rft.jtitle","value":"Bioinformatics (Oxford, England)"},{"key":"rft.volume","value":"18 Suppl 1"},{"key":"rft.issue","value":"Suppl 1"},{"key":"rft.date","value":"2002"},{"key":"rft.pages","value":"S268-75"},{"key":"rft.issn","value":"1367-4803"},{"key":"rft.au","value":"Koji Tsuda,Taishin Kin,Kiyoshi Asai"},{"key":"rft.genre","value":"article"}],"widgetId":"rgw7_56ab1fd7ae687"},"id":"rgw7_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationCoins.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationCoins.html?publicationUid=11216584","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"publicationAuthors":{"data":{"publicationUid":11216584,"peopleItems":[{"data":{"authorNameOnPublication":"Koji Tsuda","accountUrl":"profile\/Koji_Tsuda","accountKey":"Koji_Tsuda","imageUrl":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/AS%3A272439313694743%401441966002280_m\/Koji_Tsuda.png","imageWidth":20,"imageHeight":20,"peopleItem":{"data":{"displayName":"Koji Tsuda","profile":{"professionalInstitution":{"professionalInstitutionName":"The University of Tokyo","professionalInstitutionUrl":"institution\/The_University_of_Tokyo"}},"professionalInstitutionName":"The University of Tokyo","professionalInstitutionUrl":"institution\/The_University_of_Tokyo","url":"profile\/Koji_Tsuda","imageUrl":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/AS%3A272439313694743%401441966002280_l\/Koji_Tsuda.png","imageSize":"l","imageHeight":90,"imageWidth":90,"enableFollowButton":false,"enableHideButton":false,"enableConnectionButton":false,"followButton":null,"isClaimedAuthor":true,"hasExtraContainer":false,"showStatsWidgets":false,"statsWidgets":[],"additionalCssClasses":null,"contentWidgetsRight":[],"contentWidgetsMain":[],"showHideButton":false,"accountKey":"Koji_Tsuda","hasInfoPopup":false,"hasTeaserPopup":true,"showContactAuthorButton":true,"widgetId":"rgw10_56ab1fd7ae687"},"id":"rgw10_56ab1fd7ae687","partials":[],"templateName":"application\/stubs\/PeopleItem.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.PeopleAccountItem.html?entityId=1854273&enableUnfollow=0&imageSize=l&enableFollowButton=0&showContactAuthorButton=1","viewClass":"views.application.PeopleItemView","yuiModules":["rg.views.application.PeopleItemView"],"stylesheets":[],"_isYUI":true},"peopleItemRenderable":true,"accountInstitution":"The University of Tokyo","score":null,"largeTooltip":false,"useRebrandedImageStyle":null,"authorCount":3,"accountCount":2,"publicationUid":11216584,"widgetId":"rgw9_56ab1fd7ae687"},"id":"rgw9_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationDetailAccountItem.html","templateExtensions":[],"attrs":{"preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationDetailAccountItem.html?accountId=1854273&context=pubdetail_authors_xflw&showContactAuthorButton=1&authorCount=3&accountCount=2&publicationUid=11216584","viewClass":"views.publicliterature.PublicationDetailAccountItemView","yuiModules":["rg.views.publicliterature.PublicationDetailAccountItemView"],"stylesheets":[],"_isYUI":true},{"data":{"authorUrl":"researcher\/39227296_Taishin_Kin","authorNameOnPublication":"Taishin Kin","imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg","imageSize":"s","imageWidth":20,"imageHeight":20,"peopleItem":{"data":{"displayName":"Taishin Kin","profile":{"professionalInstitution":{"professionalInstitutionName":false,"professionalInstitutionUrl":null}},"professionalInstitutionName":false,"professionalInstitutionUrl":null,"url":"researcher\/39227296_Taishin_Kin","imageUrl":"https:\/\/c5.rgstatic.net\/m\/237738464651637\/images\/template\/default\/profile\/profile_default_l.jpg","imageSize":"l","imageHeight":90,"imageWidth":90,"enableFollowButton":true,"enableHideButton":false,"enableConnectionButton":false,"followButton":null,"isClaimedAuthor":false,"hasExtraContainer":false,"showStatsWidgets":false,"statsWidgets":[],"additionalCssClasses":null,"contentWidgetsRight":[],"contentWidgetsMain":[],"showHideButton":false,"widgetId":"rgw12_56ab1fd7ae687"},"id":"rgw12_56ab1fd7ae687","partials":[],"templateName":"application\/stubs\/PeopleItem.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.PeopleAuthorItem.html?entityId=39227296&imageSize=l&enableAuthorInviteButton=0","viewClass":"views.application.PeopleItemView","yuiModules":["rg.views.application.PeopleItemView"],"stylesheets":[],"_isYUI":true},"peopleItemRenderable":true,"largeTooltip":false,"useRebrandedImageStyle":null,"widgetId":"rgw11_56ab1fd7ae687"},"id":"rgw11_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationDetailAuthorItem.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationDetailAuthorItem.html?authorUid=39227296&authorNameOnPublication=Taishin%20Kin","viewClass":"views.publicliterature.PublicationDetailAccountItemView","yuiModules":["rg.views.publicliterature.PublicationDetailAccountItemView"],"stylesheets":[],"_isYUI":true},{"data":{"authorNameOnPublication":"Kiyoshi Asai","accountUrl":"profile\/Kiyoshi_Asai","accountKey":"Kiyoshi_Asai","imageUrl":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/AS%3A279014078992401%401443533549046_m\/Kiyoshi_Asai.png","imageWidth":20,"imageHeight":20,"peopleItem":{"data":{"displayName":"Kiyoshi Asai","profile":{"professionalInstitution":{"professionalInstitutionName":"The University of Tokyo","professionalInstitutionUrl":"institution\/The_University_of_Tokyo"}},"professionalInstitutionName":"The University of Tokyo","professionalInstitutionUrl":"institution\/The_University_of_Tokyo","url":"profile\/Kiyoshi_Asai","imageUrl":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/AS%3A279014078992401%401443533549046_l\/Kiyoshi_Asai.png","imageSize":"l","imageHeight":90,"imageWidth":90,"enableFollowButton":false,"enableHideButton":false,"enableConnectionButton":false,"followButton":null,"isClaimedAuthor":true,"hasExtraContainer":false,"showStatsWidgets":false,"statsWidgets":[],"additionalCssClasses":null,"contentWidgetsRight":[],"contentWidgetsMain":[],"showHideButton":false,"accountKey":"Kiyoshi_Asai","hasInfoPopup":false,"hasTeaserPopup":true,"showContactAuthorButton":true,"widgetId":"rgw14_56ab1fd7ae687"},"id":"rgw14_56ab1fd7ae687","partials":[],"templateName":"application\/stubs\/PeopleItem.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.PeopleAccountItem.html?entityId=2947656&enableUnfollow=0&imageSize=l&enableFollowButton=0&showContactAuthorButton=1","viewClass":"views.application.PeopleItemView","yuiModules":["rg.views.application.PeopleItemView"],"stylesheets":[],"_isYUI":true},"peopleItemRenderable":true,"accountInstitution":"The University of Tokyo","score":null,"largeTooltip":false,"useRebrandedImageStyle":null,"authorCount":3,"accountCount":2,"publicationUid":11216584,"widgetId":"rgw13_56ab1fd7ae687"},"id":"rgw13_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationDetailAccountItem.html","templateExtensions":[],"attrs":{"preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationDetailAccountItem.html?accountId=2947656&context=pubdetail_authors_xflw&showContactAuthorButton=1&authorCount=3&accountCount=2&publicationUid=11216584","viewClass":"views.publicliterature.PublicationDetailAccountItemView","yuiModules":["rg.views.publicliterature.PublicationDetailAccountItemView"],"stylesheets":[],"_isYUI":true}],"hasMore":false,"nextOffset":6,"useRebrandedImageStyle":null,"widgetId":"rgw8_56ab1fd7ae687"},"id":"rgw8_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationDetailAuthorList.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationDetailAuthorList.html?publicationUid=11216584&context=pubdetail_authors_xflw&showContactAuthorButton=1","viewClass":"views.publicliterature.PublicationDetailAuthorListView","yuiModules":["rg.views.publicliterature.PublicationDetailAuthorListView"],"stylesheets":[],"_isYUI":true},"publicationAbstract":{"data":{"publicationUid":11216584,"abstract":"<noscript><\/noscript><div>MOTIVATION: Kernel methods such as support vector machines require a kernel function between objects to be defined a priori. Several works have been done to derive kernels from probability distributions, e.g., the Fisher kernel. However, a general methodology to design a kernel is not fully developed. RESULTS: We propose a reasonable way of designing a kernel when objects are generated from latent variable models (e.g., HMM). First of all, a joint kernel is designed for complete data which include both visible and hidden variables. Then a marginalized kernel for visible data is obtained by taking the expectation with respect to hidden variables. We will show that the Fisher kernel is a special case of marginalized kernels, which gives another viewpoint to the Fisher kernel theory. Although our approach can be applied to any object, we particularly derive several marginalized kernels useful for biological sequences (e.g., DNA and proteins). The effectiveness of marginalized kernels is illustrated in the task of classifying bacterial gyrase subunit B (gyrB) amino acid sequences.<\/div>","canEdit":false,"isAdmin":false,"isArtifact":false,"showFullAbstract":false,"widgetId":"rgw15_56ab1fd7ae687"},"id":"rgw15_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationAbstract.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationAbstract.html?publicationUid=11216584","viewClass":"views.publicliterature.PublicationAbstractView","yuiModules":["rg.views.publicliterature.PublicationAbstractView"],"stylesheets":[],"_isYUI":true},"publicationKeywords":null,"publicationState":null,"isGuest":true,"isAdminEditingAllowed":false,"isArtifact":false,"figureListWidget":null,"previewImage":"https:\/\/i1.rgstatic.net\/publication\/11216584_Marginalized_kernels_for_biological_sequences\/links\/00b7d52705ee89c23a000000\/smallpreview.png","nativeAdDisclosure":null,"showFollowPublicationButton":false,"followPublicationPromo":{"data":{"widgetId":"rgw16_56ab1fd7ae687"},"id":"rgw16_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/FollowPublicationPromo.html","templateExtensions":[],"attrs":{"context":null,"preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.FollowPublicationPromo.html","viewClass":null,"yuiModules":["css-pow-publicliterature-FollowPublicationPromo"],"stylesheets":["pow\/publicliterature\/FollowPublicationPromo.css"],"_isYUI":true},"widgetId":"rgw5_56ab1fd7ae687"},"id":"rgw5_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationDetailItem.html","templateExtensions":["generalHelpers"],"attrs":{"preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationDetailItem.html?publicationUid=11216584&showActionBar=0&showContactAuthorButton=1&showRequestFulltextExperience=0&showNoRgAuthorsRequestFulltextExperience=0&showFollowPublicationPromo=1","viewClass":"views.publicliterature.PublicationDetailItemView","yuiModules":["rg.views.publicliterature.PublicationDetailItemView"],"stylesheets":[],"_isYUI":true},"similarPublications":{"data":{"publicationListItems":[{"data":{"milestones":{"moreContext":"experimentMilestoneClickedCitationContextReadMore","clickThrough":"experimentMilestoneClickedToPublicationFromReferenceWithNoFulltext","clickThroughWithContext":"experimentMilestoneClickedCitationContextReadMore"},"nextPublicationMilestone":"experimentMilestoneRequestFulltextNoPreview","nextPublicationViewId":null,"authorsPartOne":[{"id":13960335,"url":"researcher\/13960335_Antti_Airola","fullname":"Antti Airola","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":39244917,"url":"researcher\/39244917_Tapio_Pahikkala","fullname":"Tapio Pahikkala","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"authorsPartTwo":null,"surplusAuthors":null,"additionalCssClasses":null,"isFulltext":null,"isSlurp":false,"isNoText":true,"publicationType":"Article","publicationDate":"Jan 2016","journal":null,"showEnrichedPublicationItem":false,"citationCount":0,"commentCount":0,"requestFulltextButton":null,"publicationUrl":"publication\/289587194_Fast_Kronecker_product_kernel_methods_via_sampled_vec_trick","usePlainButton":true,"publicationUid":289587194,"feedStubs":null,"hasFeedStubs":false,"impactPoints":false,"url":"publication\/289587194_Fast_Kronecker_product_kernel_methods_via_sampled_vec_trick","title":"Fast Kronecker product kernel methods via sampled vec trick","displayTitleAsLink":true,"authors":[{"id":13960335,"url":"researcher\/13960335_Antti_Airola","fullname":"Antti Airola","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":39244917,"url":"researcher\/39244917_Tapio_Pahikkala","fullname":"Tapio Pahikkala","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"displayAuthorsAsLinks":true,"displayAuthors":true,"displayJournalMetadata":true,"collapseTitle":false,"collapseAuthors":false,"collapseAbstract":false,"details":[],"abstract":false,"description":false,"swapJournalAndAuthorPositions":false,"showAbstract":false,"type":"Article","showType":true,"showPublicationIcon":false,"showSeeInContextCta":false,"publicationUrlWithContext":"publication\/289587194_Fast_Kronecker_product_kernel_methods_via_sampled_vec_trick","isReader":null,"isAuthor":null,"isRequester":false,"hasFulltext":null,"actions":[],"actionWidgets":[],"publicationItemFulltext":null,"linkId":null,"context":null,"contextId":null,"eventCode":"","isCitation":false,"isPendingCitationRequest":false,"canRemoveCitation":false,"citationSourcePublicationUid":false,"publicationCitationUid":false,"showPublicationDownloadDataUsagePermissionDialog":false,"origin":"publication_list","keywordList":null,"hasKeywordList":false,"showOpenReviewButton":false,"openReviewUrl":"publication\/289587194_Fast_Kronecker_product_kernel_methods_via_sampled_vec_trick\/review","additionalRightSideTopWidgets":[],"additionalRightSideBottomWidgets":[],"requestEndpointPrimary":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","requestEndpointImage":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","doi":false,"hasMicrodata":false,"microdataPropertyName":null,"showPublicationPreview":true,"showPublicationPreviewOnRightSide":false,"showActionsOnRightSide":false,"swapJournalAndAbstractPositions":false,"displayUpdatedExpanderPlugin":false,"showActions":false,"showAuthors":true,"isResearch":false,"publicationLinkText":"Read preview","widgetId":"rgw18_56ab1fd7ae687"},"id":"rgw18_56ab1fd7ae687","partials":{"publicationItem":"publicliterature\/stubs\/partials\/publicationItem.html"},"templateName":"publicliterature\/stubs\/PublicationTeaserItem.html","templateExtensions":["generalHelpers"],"attrs":{"opensViewer":false,"requestFulltextInvitationDialogUrl":"literature.PublicationRequestFulltextInvitationDialog.html","followContext":"publication_item","publicationBookmarkEndpointUrl":"literature.AjaxFollowPublication.html","preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationTeaserItem.html?options%5BshowImpactPoints%5D=1&options%5BshowActorActions%5D=1&options%5BshowAbstract%5D=0&options%5BeventCode%5D=&publicationUid=289587194","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},{"data":{"milestones":{"moreContext":"experimentMilestoneClickedCitationContextReadMore","clickThrough":"experimentMilestoneClickedToPublicationFromReferenceWithNoFulltext","clickThroughWithContext":"experimentMilestoneClickedCitationContextReadMore"},"nextPublicationMilestone":"experimentMilestoneRequestFulltextNoPreview","nextPublicationViewId":null,"authorsPartOne":[{"id":2049817612,"url":"researcher\/2049817612_Hichem_Loussifi","fullname":"Hichem Loussifi","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":70687948,"url":"researcher\/70687948_Khaled_Nouri","fullname":"Khaled Nouri","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":17902406,"url":"researcher\/17902406_Naceur_Benhadj_Braiek","fullname":"Naceur Benhadj Braiek","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"authorsPartTwo":null,"surplusAuthors":null,"additionalCssClasses":null,"isFulltext":null,"isSlurp":false,"isNoText":true,"publicationType":"Article","publicationDate":"Mar 2016","journal":"Communications in Nonlinear Science and Numerical Simulation","showEnrichedPublicationItem":false,"citationCount":0,"commentCount":0,"requestFulltextButton":null,"publicationUrl":"publication\/282904456_A_new_efficient_hybrid_intelligent_method_for_nonlinear_dynamical_systems_identification_The_Wavelet_Kernel_Fuzzy_Neural_Network","usePlainButton":true,"publicationUid":282904456,"feedStubs":null,"hasFeedStubs":false,"impactPoints":"2.87","url":"publication\/282904456_A_new_efficient_hybrid_intelligent_method_for_nonlinear_dynamical_systems_identification_The_Wavelet_Kernel_Fuzzy_Neural_Network","title":"A new efficient hybrid intelligent method for nonlinear dynamical systems identification: The Wavelet Kernel Fuzzy Neural Network","displayTitleAsLink":true,"authors":[{"id":2049817612,"url":"researcher\/2049817612_Hichem_Loussifi","fullname":"Hichem Loussifi","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":70687948,"url":"researcher\/70687948_Khaled_Nouri","fullname":"Khaled Nouri","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":17902406,"url":"researcher\/17902406_Naceur_Benhadj_Braiek","fullname":"Naceur Benhadj Braiek","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"displayAuthorsAsLinks":true,"displayAuthors":true,"displayJournalMetadata":true,"collapseTitle":false,"collapseAuthors":false,"collapseAbstract":false,"details":["Communications in Nonlinear Science and Numerical Simulation 03\/2016; 32:10-30. DOI:10.1016\/j.cnsns.2015.08.010"],"abstract":false,"description":false,"swapJournalAndAuthorPositions":false,"showAbstract":false,"type":"Article","showType":true,"showPublicationIcon":false,"showSeeInContextCta":false,"publicationUrlWithContext":"publication\/282904456_A_new_efficient_hybrid_intelligent_method_for_nonlinear_dynamical_systems_identification_The_Wavelet_Kernel_Fuzzy_Neural_Network","isReader":null,"isAuthor":null,"isRequester":false,"hasFulltext":null,"actions":[],"actionWidgets":[],"publicationItemFulltext":null,"linkId":null,"context":null,"contextId":null,"eventCode":"","isCitation":false,"isPendingCitationRequest":false,"canRemoveCitation":false,"citationSourcePublicationUid":false,"publicationCitationUid":false,"showPublicationDownloadDataUsagePermissionDialog":false,"origin":"publication_list","keywordList":null,"hasKeywordList":false,"showOpenReviewButton":false,"openReviewUrl":"publication\/282904456_A_new_efficient_hybrid_intelligent_method_for_nonlinear_dynamical_systems_identification_The_Wavelet_Kernel_Fuzzy_Neural_Network\/review","additionalRightSideTopWidgets":[],"additionalRightSideBottomWidgets":[],"requestEndpointPrimary":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","requestEndpointImage":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","doi":false,"hasMicrodata":false,"microdataPropertyName":null,"showPublicationPreview":true,"showPublicationPreviewOnRightSide":false,"showActionsOnRightSide":false,"swapJournalAndAbstractPositions":false,"displayUpdatedExpanderPlugin":false,"showActions":false,"showAuthors":true,"isResearch":false,"publicationLinkText":"Read preview","widgetId":"rgw19_56ab1fd7ae687"},"id":"rgw19_56ab1fd7ae687","partials":{"publicationItem":"publicliterature\/stubs\/partials\/publicationItem.html"},"templateName":"publicliterature\/stubs\/PublicationTeaserItem.html","templateExtensions":["generalHelpers"],"attrs":{"opensViewer":false,"requestFulltextInvitationDialogUrl":"literature.PublicationRequestFulltextInvitationDialog.html","followContext":"publication_item","publicationBookmarkEndpointUrl":"literature.AjaxFollowPublication.html","preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationTeaserItem.html?options%5BshowImpactPoints%5D=1&options%5BshowActorActions%5D=1&options%5BshowAbstract%5D=0&options%5BeventCode%5D=&publicationUid=282904456","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},{"data":{"milestones":{"moreContext":"experimentMilestoneClickedCitationContextReadMore","clickThrough":"experimentMilestoneClickedToPublicationFromReferenceWithNoFulltext","clickThroughWithContext":"experimentMilestoneClickedCitationContextReadMore"},"nextPublicationMilestone":"experimentMilestoneRequestFulltextNoPreview","nextPublicationViewId":null,"authorsPartOne":[{"id":70659180,"url":"researcher\/70659180_Andrew_Mercer","fullname":"Andrew Mercer","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":2075534530,"url":"researcher\/2075534530_Alexandria_Grimes","fullname":"Alexandria Grimes","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"authorsPartTwo":null,"surplusAuthors":null,"additionalCssClasses":null,"isFulltext":null,"isSlurp":false,"isNoText":true,"publicationType":"Article","publicationDate":"Dec 2015","journal":"Procedia Computer Science","showEnrichedPublicationItem":false,"citationCount":0,"commentCount":0,"requestFulltextButton":null,"publicationUrl":"publication\/283161063_Diagnosing_Tropical_Cyclone_Rapid_Intensification_Using_Kernel_Methods_and_Reanalysis_Datasets","usePlainButton":true,"publicationUid":283161063,"feedStubs":null,"hasFeedStubs":false,"impactPoints":false,"url":"publication\/283161063_Diagnosing_Tropical_Cyclone_Rapid_Intensification_Using_Kernel_Methods_and_Reanalysis_Datasets","title":"Diagnosing Tropical Cyclone Rapid Intensification Using Kernel Methods and Reanalysis Datasets","displayTitleAsLink":true,"authors":[{"id":70659180,"url":"researcher\/70659180_Andrew_Mercer","fullname":"Andrew Mercer","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":2075534530,"url":"researcher\/2075534530_Alexandria_Grimes","fullname":"Alexandria Grimes","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"displayAuthorsAsLinks":true,"displayAuthors":true,"displayJournalMetadata":true,"collapseTitle":false,"collapseAuthors":false,"collapseAbstract":false,"details":["Procedia Computer Science 12\/2015; 61:422-427. DOI:10.1016\/j.procs.2015.09.179"],"abstract":false,"description":false,"swapJournalAndAuthorPositions":false,"showAbstract":false,"type":"Article","showType":true,"showPublicationIcon":false,"showSeeInContextCta":false,"publicationUrlWithContext":"publication\/283161063_Diagnosing_Tropical_Cyclone_Rapid_Intensification_Using_Kernel_Methods_and_Reanalysis_Datasets","isReader":null,"isAuthor":null,"isRequester":false,"hasFulltext":null,"actions":[],"actionWidgets":[],"publicationItemFulltext":null,"linkId":null,"context":null,"contextId":null,"eventCode":"","isCitation":false,"isPendingCitationRequest":false,"canRemoveCitation":false,"citationSourcePublicationUid":false,"publicationCitationUid":false,"showPublicationDownloadDataUsagePermissionDialog":false,"origin":"publication_list","keywordList":null,"hasKeywordList":false,"showOpenReviewButton":false,"openReviewUrl":"publication\/283161063_Diagnosing_Tropical_Cyclone_Rapid_Intensification_Using_Kernel_Methods_and_Reanalysis_Datasets\/review","additionalRightSideTopWidgets":[],"additionalRightSideBottomWidgets":[],"requestEndpointPrimary":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","requestEndpointImage":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","doi":false,"hasMicrodata":false,"microdataPropertyName":null,"showPublicationPreview":true,"showPublicationPreviewOnRightSide":false,"showActionsOnRightSide":false,"swapJournalAndAbstractPositions":false,"displayUpdatedExpanderPlugin":false,"showActions":false,"showAuthors":true,"isResearch":false,"publicationLinkText":"Read preview","widgetId":"rgw20_56ab1fd7ae687"},"id":"rgw20_56ab1fd7ae687","partials":{"publicationItem":"publicliterature\/stubs\/partials\/publicationItem.html"},"templateName":"publicliterature\/stubs\/PublicationTeaserItem.html","templateExtensions":["generalHelpers"],"attrs":{"opensViewer":false,"requestFulltextInvitationDialogUrl":"literature.PublicationRequestFulltextInvitationDialog.html","followContext":"publication_item","publicationBookmarkEndpointUrl":"literature.AjaxFollowPublication.html","preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationTeaserItem.html?options%5BshowImpactPoints%5D=1&options%5BshowActorActions%5D=1&options%5BshowAbstract%5D=0&options%5BeventCode%5D=&publicationUid=283161063","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true}],"viewClass":"sidebar","widgetId":"rgw17_56ab1fd7ae687"},"id":"rgw17_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/SimilarPublications.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.SimilarPublications.html?referencePublicationId=11216584&view=sidebar","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"publicationFulltextPreview":null,"publicationResourcesFulltext":{"data":{"publicationUid":11216584,"publicationResourceList":{"data":{"publicationResourceItems":[{"data":{"publicationUid":11216584,"publicationType":"article","linkId":"00b7d52705ee89c23a000000","fileName":"S268.full.pdf","fileUrl":"profile\/Koji_Tsuda\/publication\/11216584_Marginalized_kernels_for_biological_sequences\/links\/00b7d52705ee89c23a000000.pdf","name":"Koji Tsuda","nameUrl":"profile\/Koji_Tsuda","canRemove":false,"fileIcon":"ico-pub","canPreview":true,"hasPublisherLink":null,"hasMetaData":true,"hide":false,"fulltext":"ico-pub","origin":"publication_list","isLastLink":false,"isUserLink":true,"uploadDate":"Jan 20, 2016","fileSize":"176.99 KB","widgetId":"rgw23_56ab1fd7ae687"},"id":"rgw23_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationResourceItem.html","templateExtensions":["generalHelpers"],"attrs":{"fileRequiredPublicationTypes":["code","coverPage","experimentFindings","method","negativeResults","presentation","rawData","researchProposal","workingPaper","artifact","dataset"]},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationResourceItem.html?publicationUid=11216584&linkId=00b7d52705ee89c23a000000&hide=0&disableJavascript=disableJavascript&useAlternativeTemplate=0","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},{"data":{"publicationUid":11216584,"publicationType":"article","linkId":"0e5ffa60f0c404bcbfc9d2d9","fileName":"Marginalized kernels for biological sequences","fileUrl":"http:\/\/bioinformatics.oxfordjournals.org\/content\/18\/suppl_1\/S268.full.pdf","name":"oxfordjournals.org","nameUrl":"http:\/\/bioinformatics.oxfordjournals.org\/content\/18\/suppl_1\/S268.full.pdf","canRemove":false,"fileIcon":"ico-pub","canPreview":true,"hasPublisherLink":null,"hasMetaData":true,"hide":false,"fulltext":"ico-pub","origin":"publication_list","isLastLink":false,"isUserLink":false,"widgetId":"rgw24_56ab1fd7ae687"},"id":"rgw24_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationResourceItem.html","templateExtensions":["generalHelpers"],"attrs":{"fileRequiredPublicationTypes":["code","coverPage","experimentFindings","method","negativeResults","presentation","rawData","researchProposal","workingPaper","artifact","dataset"]},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationResourceItem.html?publicationUid=11216584&linkId=0e5ffa60f0c404bcbfc9d2d9&hide=0&disableJavascript=disableJavascript&useAlternativeTemplate=0","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true}],"widgetId":"rgw22_56ab1fd7ae687"},"id":"rgw22_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationResourceList.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationResourceList.html?publicationUid=11216584&limit=3&disableJavascript=disableJavascript&type=fulltextFile","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"resourceCount":2,"hidden":false,"showMore":false,"fulltext":true,"publicationDownloadCount":{"data":{"value":77,"valueFormatted":"77","widgetId":"rgw25_56ab1fd7ae687"},"id":"rgw25_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationDownloadCount.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationDownloadCount.html?publicationUid=11216584","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"widgetId":"rgw21_56ab1fd7ae687"},"id":"rgw21_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationResources.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationResources.html?publicationUid=11216584&type=fulltextFile&disableJavascript=1","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"publicationResourcesAttachments":{"data":{"publicationUid":11216584,"publicationResourceList":{"data":{"publicationResourceItems":[],"widgetId":"rgw27_56ab1fd7ae687"},"id":"rgw27_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationResourceList.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationResourceList.html?publicationUid=11216584&limit=3&disableJavascript=disableJavascript&type=attachmentFile","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"resourceCount":0,"hidden":true,"showMore":false,"fulltext":false,"publicationDownloadCount":{"data":{"value":77,"valueFormatted":"77","widgetId":"rgw28_56ab1fd7ae687"},"id":"rgw28_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationDownloadCount.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationDownloadCount.html?publicationUid=11216584","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"widgetId":"rgw26_56ab1fd7ae687"},"id":"rgw26_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationResources.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationResources.html?publicationUid=11216584&type=attachmentFile","viewClass":"views.publicliterature.PublicationResourcesView","yuiModules":["rg.views.publicliterature.PublicationResourcesView"],"stylesheets":[],"_isYUI":true},"publicationText":{"data":{"hide":false,"pages":[{"page":1,"text":"BIOINFORMATICS\nVol. 18 Suppl. 1 2002\nPages S268\u2013S275\nMarginalized kernels for biological sequences\nKoji Tsuda, Taishin Kin and Kiyoshi Asai\nComputational Biology Research Center, National Institute of Advanced Industrial\nScience and Technology (AIST), 2-41-6 Aomi Koto-ku, Tokyo, 135-0064, Japan\nReceived on January 24, 2002; revised and accepted on April 1, 2002\nABSTRACT\nMotivation: Kernel methods such as support vector\nmachines require a kernel function between objects to be\ndefined a priori. Several works have been done to derive\nkernels from probability distributions, e.g., the Fisher\nkernel. However, a general methodology to design a kernel\nis not fully developed.\nResults: We propose a reasonable way of designing a\nkernel when objects are generated from latent variable\nmodels (e.g., HMM). First of all, a joint kernel is designed\nfor complete data which include both visible and hidden\nvariables. Then a marginalized kernel for visible data\nis obtained by taking the expectation with respect to\nhidden variables. We will show that the Fisher kernel\nis a special case of marginalized kernels, which gives\nanother viewpoint to the Fisher kernel theory. Although\nour approach can be applied to any object, we particularly\nderive several marginalized kernels useful for biological\nsequences (e.g., DNA and proteins). The effectiveness of\nmarginalized kernels is illustrated in the task of classifying\nbacterial gyrase subunit B (gyrB) amino acid sequences.\nContact: koji.tsuda@aist.go.jp\nKeywords: kernel design; marginalized kernels; the\nFisher kernel; biological sequence classification; string\nkernels.\nINTRODUCTION\nInkernelmethodssuchassupportvectormachines(M\u00a8 uller\net al., 2001), a kernel function between two objects should\nbe determined a priori. In supervised learning algorithms,\nthe objective function to be optimized is clearly stated\n(e.g., the expected risk), so one should determine the\nkernel to optimize this function or its approximations\n(e.g., the leave-one-out error). However, in unsupervised\nlearning algorithms such as clustering, the choice of\nkernel is quite subjective. The kernel is determined to\nreflect the user\u2019s notion of similarity, which cannot be\njustified nor falsified completely. However, it is not an\neasy task to describe your notion of similarity as a positive\nsemidefinite kernel.\nDNA and proteins are symbol sequences which may\nhave different lengths. So they have similar character-\nistics to other symbol sequences such as texts (Frakes\nand Baeza-Yates, 1992). In order to measure similarity\nbetween such sequences, it is common to extract count\nfeatures, which represent the number of each symbol\ncontained in a sequence. Then, the similarity is obtained\nas the weighted dot product between these features,\nwhere a smaller weight is assigned for the symbols which\nappear frequently. Although this approach (i.e., vector\nspace representation (Frakes and Baeza-Yates, 1992))\nachieved a great success for texts, it is not appropriate for\nbiological sequences. The primal reason is that the context\nchanges frequently in one sequence. For example, a\nDNA sequence has coding and noncoding regions, whose\nstatistical properties are quite different. The residuals \u2018A\u2019\nin coding and noncoding regions have different meanings.\nAlthough it is difficult to determine the boundaries of the\nregions, they should be counted separately. If a sequence\nof hidden variables which describe the context are avail-\nable, it would be easier to design a kernel (Figure 1).\nHowever, hidden variables are unknown in general, and\nhave to be estimated.\nIn this paper, we propose a new reasonable way to\ndesign a kernel. First, a kernel between sequences is\ndefined depending both on visible and hidden variables.\nBecause this kernel requires both visible and hidden\nvariables for calculation, we call it a joint kernel. Since\nhidden information is assumed to be available, the kernel\ncan be designed depending on the hidden context of\nsequences. However, the problem is that such hidden\ninformation is actually not available. To cope with this\nproblem, the posterior distribution of hidden variables\nare estimated by means of a probabilistic model such as\nHMM. Then, we obtain a marginalized kernel by taking\nexpectation of the joint kernel with respect to hidden\nvariables.\nWe will show that the Fisher kernel (Jaakkola and\nHaussler, 1999)\u2014which has been successfully applied\nto many tasks e.g., protein classification (Jaakkola et\nal., 2000; Karchin et al., 2002) and promoter region\ndetection (Pavlidis et al., 2001)\u2014is a special case of\nmarginalized kernels. This reveals the joint kernel im-\nplicitly assumed in the Fisher kernel, which helps us to\nunderstand the Fisher kernel more in detail. For biological\nS268\nc ? Oxford University Press 2002"},{"page":2,"text":"Marginalized kernels for biological sequences\nh:\nx:\n2\nC\n1\nA\n2\nG\n1\nG\n2\nT\n2\nT\n1\nC\n2\nA\n2\nA\nFig. 1. A DNA sequence with hidden context information. Sup-\npose the hidden variable (\u2018h\u2019 in the figure) indicates e.g., cod-\ning\/noncoding regions. If hidden variables are known, it would be\nmuch easier to design a kernel function between sequences.\nsequences, we propose useful kernels, called marginalized\ncount kernels (MCKs). In order to illustrate the effec-\ntiveness of our kernels, we will perform experiments to\nclassify bacterial gyrB amino acid sequences (Kasai et\nal., 2000). As a result, it is shown that MCKs compare\nfavorably to the Fisher kernel.\nMETHODS\nMarginalized kernels\nLet us describe a visible variable as x \u2208 X, where the\ndomain X is a finite set\u2020. Our task is to define a kernel\nK(x,x?) between two visible variables x,x?. Suppose we\nhave a hidden variable h \u2208 H, where H is a finite set.\nBy utilizing the hidden information in h, the joint kernel\nKz(z,z?) is designed between two combined variables\nz = (x,h),z?= (x?,h?). The marginalized kernel in X\nis derived by taking the expectation with respect to hidden\nvariables:\nK(x,x?) =\n?\nh\u2208H\n?\nh?\u2208H\np(h|x)p(h?|x?)Kz(z,z?).\n(1)\nThe posterior distribution p(h|x) is unknown in general,\nand has to be estimated from the data e.g., by HMMs.\nThe calculation of K can be intractable when the cardi-\nnality of H is too large. However, for useful stochastic\nmodels such as HMMs, there are algorithms which\nenable efficient computation as shown in later sections.\nSince the class of positive semidefinite (Mercer) kernels\nare closed under addition and multiplication (Haussler,\n1999), the marginalized kernel K is positive semidefinite\nas long as the joint kernel is positive semidefinite. In\nconvolution kernels (Haussler, 1999), sub-kernels for the\nparts are aggregated into a kernel for the whole set. This\nis constrastive to our approach deriving the kernel for the\npart (i.e., visible variables) from the kernel for the whole\nset (i.e., visible and hidden variables).\nMarginalized kernel from Gaussian mixture\nFor intuitive understanding, we provide a simple example\nof marginalized kernels. Here we derive a marginalized\n\u2020Here we determine X as a finite set for simplicity, but the results in this\npaper can be easily extended to continuous domains.\nkernel from Gaussian mixture with m-components. The\nvisible variable is a point in the d-dimensional space\nx \u2208 ?d, and the hidden variable is an index of component\nh \u2208 {1,...,m}. The probabilistic model is written as\np(x|\u03b8) =\ncomponent is a Gaussian distribution with mean \u00b5hand\ncovariance matrix A\u22121\n?m\nh=1p(h)q(x|h,\u00b5h, Ah), where the hth\nh:\nq(x|h,\u00b5h, Ah) =exp(\u22121\n2(x \u2212 \u00b5h)?Ah(x \u2212 \u00b5h))\n2\u03c0|A\u22121\nh|1\/2\n. (2)\nLet us define the joint kernel as Kz(z,z?) = I(h =\nh?)(x?Ahx?), where I(t) is the indicator function which\nis 1 if the condition t is true and 0 otherwise. Note that Kz\nis a positive semidefinite kernel. The marginalized kernel\nis obtained as\nK(x,x?) =\nm\n?\nh=1\np(h|x)p(h|x?)x?Ahx?.\n(3)\nWe illustrate the shape of this kernel in Figure 2. Here, the\nkernel is converted to the distance in the feature space\nD(x,x?) =\nFixing x?at a point, the contours of distance D(x,x?)\nare shown. When x?belongs to one cluster, the contour\nshape is similar to the shape of the cluster. The shape\ngradually changes when the point x?moves to one cluster\nto the other. In comparison with the Euclidean distance,\nthis distance emphasizes the cluster structure. This kind\nof kernel is considered to be useful in visualizing cluster\nstructure in a high dimensional space (Tipping, 1999).\n?\nK(x,x) + K(x?,x?) \u2212 2K(x,x?).\n(4)\nMarginalized count kernel\nNext we propose an important example of marginal-\nized kernels for biological sequences. Let x\n(x1,...,xm),\nxi \u2208 {1,...,nx} denote a symbol se-\nquence of length m. Assume that each sequence can have\na different length. One simple kernel for such sequences\nis the count kernel:\n=\nK(x,x?) =\nnx\n?\nk=1\nck(x)ck(x?),\n(5)\nwhere ck(x) is the number of symbol k normalized by the\nlength:\nck(x) =1\nm\nm\n?\ni=1\nI(xi= k).\nThe count kernel is often used in text processing litera-\ntures (Frakes and Baeza-Yates, 1992), but it is not suit-\nable for biological sequences because of frequent context\nS269"},{"page":3,"text":"K.Tsuda et al.\nFig. 2. Contours of the marginalized kernel-based distance from a\nspecified point. The left figure shows sample points generated from\nthe Gaussian mixture model. Here the crossing point of dotted lines\nindicates the central point from which the kernel-based distance\n(4) is measured. The right figure shows the distance contours.\nThe contour shapes are adapted to the shape of cluster which the\ncentral point belongs to. Compared with the Euclidean distance, this\ndistance emphasizes the cluster structure.\nchanges. We are going to extend the count kernel to in-\nclude (hidden) context information.\nAssume that there is a sequence of hidden variables h =\n(h1,...,hm), hi \u2208 {1,...,nh}. Define the combined\nsequence as\nz := (z1,...,zm) = ({x1,h1},...,{xm,hm})\nwhere each zi can have nz = nxnhsymbols. The count\nkernel for z can be defined as\nKz(z,z?) =\nnx\n?\nk=1\nnh\n?\n?=1\nm\n?\ni=1\nck?(z)ck?(z?),\nck?(z) =1\nm\nI(xi= k,hi= ?).\n(6)\nWhen h is regarded as the context information, symbols\nare counted separately in each context (Figure 3). For\nexample, if the sequences are DNA and the hidden\ncontexts are exon\/intron, the frequencies of \u2018A\u2019, \u2018C\u2019, \u2018G\u2019,\n\u2018T\u2019 are counted and compared separately for exon\/intron.\nSetting (6) as a joint kernel, the marginalized count\nkernel is defined as\nK(x,x?) =\n?\nh\n?\nh?\np(h|x)p(h?|x?)Kz(z,z?),\n(7)\nwhere\nrewritten as\n?\nh\n=\n?nh\nh1=1\u00b7\u00b7\u00b7?nh\nhm=1. This kernel (7) is\nK(x,x?) =\nnx\n?\nk=1\nnh\n?\n?=1\n\u03b3kl(x)\u03b3kl(x?),\n(T,1) = 0\n(T,2) = 2\n(G,1) = 1\n(G,2) = 1\n(C,1) = 1\n(C,2) = 1\n(A,1) = 1\n(A,2) = 2\nh:\nx:\n2\nC\n1\nA\n2\nG\n1\nG\n2\nT\n2\nT\n1\nC\n2\nA\n2\nA\nFig. 3. Illustration of the marginalized count kernel (MCK1). Each\nfeature is obtained by counting the number of combined symbols,\nand the joint kernel is defined as the dot product between these\nfeatures. Finally, MCK1 is obtained by marginalizing the joint\nkernel.\nwhere the marginalized counts \u03b3kl(x) are described as\n\u03b3kl(x) =1\nm\n?\nh\nm\n?\ni=1\np(h|x)\nm\n?\ni=1\nI(xi= k,hi= ?)\n=1\nm\nnh\n?\nhi=1\np(hi|x)I(xi= k,hi= ?).\nFortunately, the sum over all hidden variables h can be\nreplaced by the sum over each hi, which reduces the\ncomputational cost.\nWhen the probability distribution p(x) is represented\nas HMM, the posterior probability p(hi|x) is computed\neasily by the forward-backward algorithm (Durbin et al.,\n1998). An HMM is described as\np(x|\u03b8) =\n?\nh\nqh1eh1x1[\nm\n?\ni=2\nahi\u22121hiehixi]dhm,\n(8)\nwheretheparameters\u03b8 = {a,e,q,d}aretransitionproba-\nbilities,emissionprobabilities,initialstatedistributionand\nterminal state distribution, respectively. The forward and\nbackward algorithms provide the following probabilities,\nrespectively:\nfk(i) = p(x1,...,xi,hi= k),\nbk(i) = p(xi+1,...,xm|hi= k).\nThen the posterior probability is described as\np(hi= ?|x) =\nf?(i)b?(i)\np(x)\nwhich is known as \u03b3i(?) in HMM literatures (Rabiner,\n1989).\nSecond-order marginalized count kernel\nWhen adjacent relations between symbols have essential\nmeanings, the count kernel is obviously not sufficient. In\nS270"},{"page":4,"text":"Marginalized kernels for biological sequences\nsuch cases, it would be better to count the number of\ncombinations of two adjacent symbols (Figure 4). The dot\nproduct of such counts is described as\nK(x,x?) =\nnx\n?\nk=1\nnx\n?\nk?=1\n1\nm \u2212 1\nckk?(x)ckk?(x?),\n(9)\nckk?(x) =\nm\u22121\n?\ni=1\nI(xi= k)I(xi+1= k?).\nWe call it the second order count kernel. Incorporating\nhiddenvariablesto(9),onecaneasilyextendthemarginal-\nized count kernel to second order: Let us define\n\u03b4i\nk?k??? := I(xi= k,hi= ?,xi+1= k?,hi+1= ??).\nThe joint kernel is described as\nKz(z,z?) =\nnx\n?\nk=1\nnh\n?\n?=1\nnx\n?\nk?=1\n1\nm\u22121\nnh\n?\n??=1\n?m\u22121\nnh\n?\n??=1\nck?k???(z)ck?k???(z?),\n(10)\nwhere ck?k???(z) =\nkernel of (10) is described as\ni=1\u03b4i\nk?k???. The marginalized\nK(x,x?) =\nnx\n?\nk=1\nnh\n?\n?=1\nnx\n?\nk?=1\nvk?k???(x)?vk?k???(x?) (11)\nwhere\nvk?k???(x) =\n1\nm \u2212 1\n1\nm \u2212 1\n?\nh\nm\u22121\n?\ni=1\np(h|x)\nm\u22121\n?\ni=1\nnh\n?\nhi+1=1\n\u03b4i\nk?k???\n=\nnh\n?\nhi=1\np(hi,hi+1|x)\u03b4i\nk?k???.\nWe call it the second order marginalized count kernel. As\nin the first order case, the posterior probability p(hi,hi+\n1|x) is obtained from forward and backward algorithms as\np(hi= ?,hi+1= ??|x) = a???e??xi+1\nNote that this quantity is well known as \u03bei(?,??) parameter\nin Baum-Welch algorithm, which gives estimation of tran-\nsition probabilities in HMMs. The second order marginal-\nized count kernel is particularly useful, because it can uti-\nlize second order information as well as hidden context in-\nformation. Higher order extension is straightforward, but\nnot written here for brevity.\nf?(i)b??(i + 1)\np(x)\n.\nConnections to the Fisher kernel\nIn the following, we will show that the Fisher ker-\nnels (Jaakkola and Haussler, 1999) derived from latent\nvariable models are described as marginalized kernels.\nThis section will give a new analysis to explain the nature\nof the Fisher kernel.\nh:\nx:\n2\nC\n1\nA\n2\nG\n1\nG\n2\nT\n2\nT\n1\nC\n2\nA\n2\nA\nFig. 4. Illustration of the second-order marginalized count kernel\n(MCK2). Each feature is obtained by counting the number of\ncombinations of two adjacent symbols. The joint kernel is defined\nas the dot product between these features, and MCK2 is obtained by\nmarginalization.\nDefinition of the Fisher kernel\nmodel p(x|\u03b8) is defined on X, where \u03b8 is a r-dimensional\nparameter vector. Let\u02c6\u03b8 denote parameter values which\nare obtained by some learning algorithm (e.g., maximum\nlikelihood). Then the Fisher kernel (FK) (Jaakkola and\nHaussler, 1999) between two objects is defined as\nAssume a probabilistic\nKf(x,x?) = s(x,\u02c6\u03b8)?Z\u22121(\u02c6\u03b8)s(x?,\u02c6\u03b8),\nwhere s is the Fisher score\n(12)\ns(x,\u02c6\u03b8) =\n?\u2202\n\u2202\u03b81\nlog p(x|\u02c6\u03b8),...,\n\u2202\n\u2202\u03b8r\nlog p(x|\u02c6\u03b8)\n??\n:= \u2207\u03b8log p(x|\u02c6\u03b8),\nand Z is the Fisher information matrix:\nZ(\u02c6\u03b8) = E\n?\ns(x,\u02c6\u03b8)s(x,\u02c6\u03b8)????\u02c6\u03b8\n?\nx\u2208X\n?\n=\np(x|\u02c6\u03b8)s(x,\u02c6\u03b8)s(x,\u02c6\u03b8)?.\nThe Fisher kernel is a general method which can be\napplied for any objects. However, the Fisher kernel\nis particularly effective for biological sequences when\ncombined with HMMs (Jaakkola et al., 2000; Karchin et\nal., 2002; Pavlidis et al., 2001).\nThe Fisher Kernel from Latent Variable Models\na latent variable model p(x|\u03b8) =\nadopted, the Fisher score is described as\nWhen\n?\nh\u2208Hp(x,h|\u03b8) is\n\u2207\u03b8log p(x|\u02c6\u03b8) =\n?\nh\u2208H\u2207\u03b8p(x,h|\u02c6\u03b8)\np(x|\u02c6\u03b8)\n?\nh\u2208H\n?\nh\u2208H\n=\np(x,h|\u02c6\u03b8)\np(x|\u02c6\u03b8)\np(h|x,\u02c6\u03b8)\u2207\u03b8log p(x,h|\u02c6\u03b8).\n\u2207\u03b8p(x,h|\u02c6\u03b8)\np(x,h|\u02c6\u03b8)\n=\nS271"},{"page":5,"text":"K.Tsuda et al.\nSo, the Fisher kernel is described as a marginalized kernel\nKf(x,x?) = \u2207\u03b8log p(x|\u02c6\u03b8)?Z(\u02c6\u03b8)\u22121\u2207\u03b8log p(x?|\u02c6\u03b8)\n:=\nh\u2208H\n?\n?\nh?\u2208H\np(h|x,\u02c6\u03b8)p(h?|x?,\u02c6\u03b8)Kz(z,z?),\n(13)\nwhere the joint kernel is described as Kz(z,z?)\n\u2207\u03b8log p(x,h|\u02c6\u03b8)?Z(\u02c6\u03b8)\u22121\u2207\u03b8log p(x?,h?|\u02c6\u03b8).\nFisher kernel is one special case in the class of marginal-\nized kernels. One characteristic aspect of the Fisher kernel\nis that the joint kernel is determined by the probabilistic\nmodel, while, in our approach, the joint kernel is designed\nto fit user\u2019s purposes. Since the joint kernel of the Fisher\nkernel is not suitable for every purpose, you have to check\nwhether it fits your purpose or not. If not, the joint kernel\nshould be engineered.\n=\nThus the\nThe Fisher kernel from HMM\nthe Fisher kernel from HMM (8) and discuss its connec-\ntion to marginalized count kernels. The joint distribution\nof HMM is described as\nIn this section, we derive\np(x,h|\u03b8) = qh1eh1x1[\nm\n?\ni=2\nahi\u22121hiehixi]dhm.\nAs in the literature (Jaakkola et al., 2000), we take the\nderivatives with respect to emission probabilities e only:\n\u2202\n\u2202e?k\nlog p(x,h|\u02c6\u03b8) =mck?(z)\n\u02c6 e?k\n\u2212 m\nnx\n?\nk=1\nck?(z),\n(14)\nwhere \u02c6 e?kis the estimated emission probability and ck?(z)\nis defined as (6). Note that the second term of (14) comes\nfrom the constraint of emission probabilities?nx\n1. If we do not use the Fisher information matrix as in\n(Jaakkola et al., 2000), the joint kernel is described as\nk=1e?k=\n\u2207elog p(x,h|\u02c6\u03b8)?\u2207elog p(x?,h?|\u02c6\u03b8).\nThis is rewritten as\nnx\n?\nk=1\nwhere \u00af ck?(z) = \u02c6 e?k\nto the count kernel (6), however the count is centralized\nand the dot product is taken with respect to the weight\n\u03b1k?=mm?\nproper normalization is needed for the Fisher kernel. Since\ne?k represent the emission probability that symbol k is\nproduced from state ?, the weight becomes large when the\nsymbol k is rarely produced from state ?. It makes sense,\nbecause the cooccurence of a rare symbol is a strong clue\nof high similarity. However this weight is still argueable,\nbecause a huge weight can appear when e?kis very small.\n(15)\nnh\n?\n?=1\nmm?\n\u02c6 e2\n?k\n(ck?(z) \u2212 \u00af ck?(z))(ck?(z?) \u2212 \u00af ck?(z?)),\n(16)\n?nx\nk?=1ck??(z). This has a similar form\n\u02c6 e2\n?k. The weight is dependent on the length m, so a\nDISCUSSION\nIn the previous section, we derived the Fisher kernel\nonly from emission probabilities. However, if you take\nthe derivatives of transition probabilities as well, you\nobtain a different joint kernel from the one shown in\n(16). How should we choose the subset of parameters to\ntake derivatives? More generally, you can derive a new\nparameter as a function of a subset of original parameters.\nHow about taking the derivative with respect to the new\nparameter?\nAs suggested in this example, one theoretical problem\nabout the Fisher kernel is that it depends not only on\nthe distribution itself, but also the parametrization which\na user has intentionally chosen. Consider two parametric\nmodels:\np(x,h|\u03b8), \u03b8 \u2208 ?r1,\nLet us assume that a joint distribution p(x,h) is repre-\nsented by two different parametric models:\np(x,h|\u00b5), \u00b5 \u2208 ?r2, r1?= r2.\np(x,h) = p(x,h|\u02c6\u03b8) = p(x,h|\u02c6 \u00b5).\nIn general, the Fisher kernels derived from p(x|\u02c6\u03b8) and\np(x|\u02c6 \u00b5) are different although the underlying distribution\nis the same\u2021. Since there is no admitted way to choose\nproper parametrization so far, it is basically determined by\ntrial and error.\nIn (13) we represented the Fisher kernel as a function\nof the joint kernel and posterior probabilities of hidden\nvariables. While the joint kernel is not invariant to\nparametrization,\n\u2207\u03b8log p(x,h|\u02c6\u03b8)?Z(\u02c6\u03b8)\u22121\u2207\u03b8log p(x?,h?|\u02c6\u03b8)\n?= \u2207\u00b5log p(x,h|\u02c6 \u00b5)?Z(\u02c6 \u00b5)\u22121\u2207\u00b5log p(x?,h?|\u02c6 \u00b5),\nthe posterior probabilities are invariant,\np(h|x,\u02c6\u03b8) = p(h|x, \u02c6 \u00b5).\nTherefore choosing the parametrization amounts to\nchoosing the joint kernel. You may be able to derive the\njoint kernel to fit your purpose by changing parametriza-\ntion. However, in our opinion, this is an awkward and\nindirect way.\nIn the Fisher kernel scheme, you have to control\ntwo things (i.e., joint kernel and posterior probability)\nsimultanously by the choice of a parametric model. In our\nopinion, there is no need to control them in such an unified\nmanner, because this scheme is sometimes too restrictive.\n\u2021When there is one-to-one correspondence between two parameter spaces\naround\u02c6\u03b8 and \u02c6 \u00b5, the Fisher kernel is invariant to parametrization because of\nthe Fisher information matrix (Jaakkola and Haussler, 1999). However it is\nnot the case in general.\nS272"},{"page":6,"text":"Marginalized kernels for biological sequences\nFor example, when you would like to incorporate the\nsecond-order information into the Fisher kernel, you have\ntousesecond-orderHMMs(Durbinetal.,1998).Sincethe\nnumber of parameters of the second-order HMM is much\nlarger, it would be difficult to learn the parameters reliably\nwith a small sample set. This drawback is caused by the\nfact that the joint kernel is tied to the probabilistic model.\nIn our approach, the joint kernel and the probabilistic\nmodel are completely separated, so you can utilize second\norder information with a first order HMM as in the second\norder marginalized count kernel.\nRESULTS\nIn this section, we illustrate the performance of marginal-\nized kernels in classification experiments using bacterial\ngyrB amino acid sequences. gyrB\u2014gyrase subunit B\u2014is\na DNA topoisomerase (type II) which plays essential roles\nin fundamental mechanisms of living organisms such as\nDNA replication, transcription, recombination and repair\netc. One more important feature of gyrB is its capability\nof being an evolutionary and taxonomic marker alternat-\ning popular 16S rRNA (Kasai et al., 1998). Our dataset\nconsists of 84 amino acid sequences of gyrB from five\ngenera in Actinobacteria which are Corynebacterium, My-\ncobacterium, Gordonia, Nocardia and Rhodococcus, re-\nspectively (Kasai et al., 2000). For brevity these genera\nwill be called genus 1 to 5, respectively. The number of se-\nquences in each genus is listed as 9, 32, 15, 14 and 14. The\nsequences are, by their nature, quite similar in terms of se-\nquence similarity. Pairwise identity for each sequence is at\nleast 62 and 99% at most. For computing distance matrix\nbased on the sequence similarity, one can use the BLAST\nscores (Altschul et al., 1990). However, since such scores\ncannot directly be converted into positive semidefinite ker-\nnels, kernel methods cannot be applied to them in princi-\nple.\nIn order to investigate how well the kernels re-\nflect underlying genera, we performed two kinds of\nexperiments\u2014clustering and supervised classification.\nThe following kernels are compared:\n\u2022\n\u2022\n\u2022\n\u2022\n\u2022\nCK1: Count kernel (5)\nCK2: Second-order count kernel (9)\nFK: Fisher kernel (16)\nMCK1: Marginalized count kernel (7)\nMCK2: Second-order marginalized count kernel (11)\nAs the first experiment, K-Means clustering is per-\nformed in feature spaces corresponding to kernels (see\nM\u00a8 uller et al. (2001) for details). The number of clusters\nis determined as five (i.e., the true number). In FK and\nMCKs, we used complete-connection HMMs with 3, 5\nand 7 states. Note that FK is normalized by the sequence\nlengths. In training HMMs, all 84 sequences are used.\nOne can also train HMMs in a classwise manner (Tsuda\net al., 2002). However, we did not do so because the\nnumber of sequences is not large enough. For evaluating\nclusters, we used the adjusted Rand index (ARI) (Yeung\nand Ruzzo, 2001). The advantage of this index is that\nyou can compare two partitions whose number of clusters\nare different. The ARI becomes 1 if the partitions are\ncompletely correct. Also, the expectation of the ARI is 0\nwhen partitions are randomly determined.\nThe kernel matrices by FK and MCKs are shown in\nFigure 5. Additionally, the ideal kernel is shown for\nreference,where K(x,x?)is1foranytwosequencesinthe\nsame genus, and \u22121 otherwise. Here, the number of HMM\nstates is three in all cases. For fair visualization, each\nkernel matrix is normalized in the same manner: First,\nthe kernel matrix is \u2018centralized\u2019 as Kc := K \u2212 1nK \u2212\nK1n+1nK1nwhere 1nis the n\u00d7n matrix whose elements\nare all 1\/n. Here n denotes the number of sequences, i.e.,\nn = 84 in this experiment. Then, Kcis normalized by\nthe Frobenious norm as Kc\/?Kc?F. As seen in the figure,\nMCK2 is the best to recover the underlying structure. This\nresult is quantitatively shown by ARI in Figure 6, where\nCK1 and CK2 correspond to the MCKs with only one\nHMM state. Notably the Fisher kernel was worse than\nMCK1, which shows that the joint kernel of the Fisher\nkernel (16) is not appropriate for this task.\nIn order to see how genera are separated by introducing\nthe second order information and hidden variables, we\nperformed the following supervised classification experi-\nments as well. First, we pick up two genera out of three\ngenera (3,4,5). Genera 1 and 2 were not used because\nthey can be separated easily by all kernels. The sequences\nof two genera are randomly divided into 25% training\nand 75% testing samples. Kernels are compared due to\nthe test error by the kernel Fisher discriminant analysis\n(KFDA) (Roth and Steinhage, 2000), which compares\nfavorably with the SVM in many benchmarks. Note\nthat the regularization parameter ? of KFDA (Roth and\nSteinhage, 2000) is determined such that the test error is\nminimized\u00a7. The test errors of five kernels are shown in\nTable 1. The second order kernels (i.e., CK2 and MCK2)\nwere significantly better than the first order kernels. This\nresult coincides with the common understanding that\nhigher order information of protein sequences is essential\nfor classification and structure prediction (e.g., Asai et\nal. (1993)). Comparing CK2 and MCK2, MCK2 always\nperformed better, which indicates that incorporating\nhidden variables (i.e., context information) is meaningful\nat least in this task.\n\u00a7Forregularizationparameter?,10equallyspacedpointsonthelogscaleare\ntaken from [10\u22124,102]. Among these candidates, the optimal one is chosen.\nS273"},{"page":7,"text":"K.Tsuda et al.\n1  \n0. 5\n0   \n0.5 \n1   \nIdeal\n 0.02\n0    \n0.02 \n0.04 \nFK\n 0.02\n0    \n0.02 \n0.04 \nMCK1\n 0.02\n0    \n0.02 \n0.04 \nMCK2\nFig. 5. (Upperleft) Ideal kernel matrix to illustrate the true clusters. (Upperright) Kernel matrix of the Fisher kernel. (Lowerleft) Kernel\nmatrix of the first-order marginalized count kernel. (Lowerright) Kernel matrix of the second-order marginalized count kernel.\nTable 1. Mean error rates (%) of supervised classification between two bacterial genera ([\u00b7] shows the standard deviation). The best result in each task is\nwritten in bold face\nGeneraCK1CK2 FKMCK1 MCK2\n3\u20134\n3\u20135\n4\u20135\n24.5 [9.67]\n12.7 [8.93]\n25.6 [13.0]\n9.10 [7.87]\n6.43 [7.76]\n13.5 [15.5]\n10.4 [9.15]\n10.9 [10.1]\n23.1 [14.3]\n12.8 [9.85]\n10.4 [8.17]\n20.0 [14.6]\n8.48 [7.76]\n5.71 [7.72]\n11.6 [14.6]\nCONCLUSION\nIn this paper, we proposed marginalized kernels, which\nprovide a new reasonable way to design a kernel from\nlatent variable models. The Fisher kernel was described as\na special case of marginalized kernels, which have added a\nnew aspect to the Fisher kernel theory. Finally, we showed\nthat marginalized count kernels perform well in protein\nclassification experiments.\nOur work provides a general framework from which\ndiverse kernels are expected to be constructed. In future\nworks, we would like to derive useful kernels as many as\npossible not only in bioinformatics but also in other areas\nin information technology.\nACKNOWLEDGEMENT\nThe authors gratefully acknowledge that the bacterial\ngyrB amino acid sequences are offered by courtesy\nof Identification and Classification of Bacteria (ICB)\nS274"},{"page":8,"text":"Marginalized kernels for biological sequences\n1357\n0.2\n0.3\n0.4\n0.5\n0.6\n0.7\n0.8\n0.9\n1\nNumber of HMM States\nARI\nFK\nMCK1\nMCK2\nFig. 6. Evaluation of kernels in clustering in terms of the adjusted\nRand index (ARI). The x-axis corresponds to the number of states\nin HMM, from which the kernels are derived. The Fisher kernel\n(FK), the marginalized count kernels of first-order (MCK1) and\nsecond-order (MCK2) are compared. Note that the count kernels\nof first-order (CK1) and second-order (CK2) correspond to MCK1\nand MCK2 at one HMM state, respectively.\ndatabase team (Watanabe et al., 2001). The authors\nwould like to thank S.-I.Amari, K.-R.M\u00a8 uller, G.R\u00a8 atsch,\nM.Kawanabe, H.Nakahara and S.Sonnenburg for fruitful\ndiscussions.\nREFERENCES\nAltschul,S., Gish,W., Myers,E. and Lipman,D. (1990) Basic local\nalignment search tool. J. Mol. Biol., 215, 403\u2013410.\nAsai,K., Hayamizu,S. and Handa,K. (1993) Prediction of protein\nsecondary structure by the hidden Markov model. CABIOS\n(currently Bioinformatics), 9, 141\u2013146.\nDurbin,R., Eddy,S., Krogh,A. and Mitchison,G. (1998) Biological\nSequence Analysis: Probabilistic Models of Proteins and Nucleic\nAcids. Cambridge University Press.\nFrakes,W. and Baeza-Yates,R., (eds.) (1992) Information Retrieval:\nData Structure and Algorithms. Prentice Hall.\nHaussler,D. (1999) Convolution kernels on discrete structures.\nTechnical Report UCSC-CRL-99-10. UC Santa Cruz.\nJaakkola,T., Diekhans,M. and Haussler,D. (2000) A discriminative\nframework for detecting remote protein homologies. J. Comp.\nBiol., 7, 95\u2013114.\nJaakkola,T. and Haussler,D. (1999) Exploiting generative mod-\nels in discriminative classifiers. In Kearns,M., Solla,S. and\nCohn,D. (eds), Advances in Neural Information Processing Sys-\ntems 11. pp. 487\u2013493.\nKarchin,R., Karplus,K. and Haussler,D. (2002) Classifying G-\nprotein coupled receptors with support vector machines. Bioin-\nformatics, 18, 147\u2013159.\nKasai,H.,Bairoch,A., Watanabe,K.,\nGasteiger,E. and Yamamoto,S. (1998) Construction of the gyrB\ndatabase for the identification and classification of bacteria.\nGenome Informatics 1998. Universal Academic Press, pp.\n13\u201321.\nKasai,H., Ezaki,T. and Harayama,S. (2000) Differentiation of\nphylogenetically related slowly growing mycrobacteria by their\ngyrB sequences. J. Clin. Microbiol., 38, 301\u2013308.\nM\u00a8 uller,K.-R., Mika,S., R\u00a8 atsch,G., Tsuda,K. and Sch\u00a8 olkopf,B.\n(2001) An introduction to kernel-based learning algorithms.\nIEEE Trans. Neural Networks, 12, 181\u2013201.\nPavlidis,P.,Furey,T.,Liberto,M.,Haussler,D.andGrundy,W.(2001)\nPromotor region-based classification of genes. In Proceedings\nPSB 2001. pp. 151\u2013163.\nRabiner,L. (1989) A tutorial on hidden Markov models and\nselected applications in speech recognition. Proc. IEEE, 77, 257\u2013\n285.\nRoth,V.andSteinhage,V.(2000)Nonlineardiscriminantanalysisus-\ning kernel functions. In Solla,S., Leen,T. and M\u00a8 uller,K.-R. (eds),\nAdvances in Neural Information Processing Systems 12. MIT\nPress, pp. 568\u2013574.\nTipping,M. (1999) Deriving cluster analytic distance functions\nfrom gaussian mixture models. In Willshaw,D. and Mur-\nray,A. (eds), Proceedings of ICANN\u201999. IEE Press, pp. 815\u2013820.\nTsuda,K., Kawanabe,M., R\u00a8 atsch,G., Sonnenburg,S. and M\u00a8 uller,K.-\nR.(2002)Anewdiscriminativekernelfromprobabilisticmodels.\nNeural Comput., In press.\nWatanabe,K., Nelson,J., Harayama,S. and Kasai,H. (2001) ICB\ndatabase: the gyrB database for identification and classification\nof bacteria. Nucleic Acids Res., 29, 344\u2013345.\nYeung,K. and Ruzzo,W. (2001) Principal component analysis for\nclustering gene expression data. Bioinformatics, 17, 763\u2013774.\nIsono,K.,Harayama,S.,\nS275"}],"fullTextUrl":"https:\/\/www.researchgate.net\/profile\/Koji_Tsuda\/publication\/11216584_Marginalized_kernels_for_biological_sequences\/links\/00b7d52705ee89c23a000000.pdf","widgetId":"rgw29_56ab1fd7ae687"},"id":"rgw29_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationText.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationText.html?publicationUid=11216584&hide=0","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"publicationDetailAd":{"data":{"googleDfpSlot":null,"widgetId":"rgw30_56ab1fd7ae687"},"id":"rgw30_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationDetailAd.html","templateExtensions":["generalHelpers"],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationDetailAd.html?publicationUid=11216584&slotId=336x280_Publications_ATF_Right&collapseSlotMode=never&fallbackContainerEnabled=1","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"publicationRelations":null,"publicationRelationsReverse":null,"publicationUid":11216584,"showSignUpDialog":false,"selectNewSignUpDialog":false,"publicationQuestions":null,"publicationCitations":{"data":{"publicationUid":11216584,"publicationCitationsList":{"data":{"citationItems":[{"data":{"milestones":{"moreContext":"experimentMilestoneClickedCitationContextReadMore","clickThrough":"experimentMilestoneClickedToPublicationFromCitationWithFulltext","clickThroughWithContext":"experimentMilestoneClickedCitationContextReadMore"},"nextPublicationMilestone":"experimentMilestoneRequestFulltextNoPreview","nextPublicationViewId":null,"authorsPartOne":[{"id":85205020,"url":"researcher\/85205020_Benjamin_Yee_Shing_Li","fullname":"Benjamin Yee Shing Li","last":false,"imageUrl":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/AS%3A279086434930696%401443550799610_m"},{"id":14503873,"url":"researcher\/14503873_Lam_Fat_Yeung","fullname":"Lam Fat Yeung","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":74904926,"url":"researcher\/74904926_King_Tim_Ko","fullname":"King Tim Ko","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"authorsPartTwo":null,"surplusAuthors":null,"additionalCssClasses":[["includes-citation-list"]],"isFulltext":true,"isSlurp":false,"isNoText":false,"publicationType":"Article","publicationDate":"Feb 2015","journal":"Neurocomputing","showEnrichedPublicationItem":false,"citationCount":1,"commentCount":0,"requestFulltextButton":null,"publicationUrl":"publication\/273125269_Indefinite_kernel_ridge_regression_and_its_application_on_QSAR_modelling","usePlainButton":true,"publicationUid":273125269,"feedStubs":null,"hasFeedStubs":false,"impactPoints":"2.08","url":"publication\/273125269_Indefinite_kernel_ridge_regression_and_its_application_on_QSAR_modelling","title":"Indefinite kernel ridge regression and its application on QSAR modelling","displayTitleAsLink":true,"authors":[{"id":85205020,"url":"researcher\/85205020_Benjamin_Yee_Shing_Li","fullname":"Benjamin Yee Shing Li","last":false,"imageUrl":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/AS%3A279086434930696%401443550799610_m"},{"id":14503873,"url":"researcher\/14503873_Lam_Fat_Yeung","fullname":"Lam Fat Yeung","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":74904926,"url":"researcher\/74904926_King_Tim_Ko","fullname":"King Tim Ko","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"displayAuthorsAsLinks":true,"displayAuthors":true,"displayJournalMetadata":true,"collapseTitle":false,"collapseAuthors":false,"collapseAbstract":false,"details":["Neurocomputing 02\/2015; 158. DOI:10.1016\/j.neucom.2015.01.060"],"abstract":"Recently, the use of indefinite kernels in machine learning has attracted numerous attentions. However most works are focused on the classification techniques and less are devoted to regression models. In this paper to adapt indefinite kernels to ridge regression model, an indefinite kernel ridge regression model is proposed. Instead of performing spectral transformation on the kernel matrix, a less restrictive semi-definite proxy kernel can be constructed to approximate the kernel which normally is positive semi-definite. The sensitivity of the distance between this indefinite kernel and the proxy kernel is controlled by a parameter \u03c1.","description":false,"swapJournalAndAuthorPositions":false,"showAbstract":true,"type":"Article","showType":true,"showPublicationIcon":false,"showSeeInContextCta":false,"publicationUrlWithContext":"publication\/273125269_Indefinite_kernel_ridge_regression_and_its_application_on_QSAR_modelling","isReader":null,"isAuthor":null,"isRequester":false,"hasFulltext":true,"actions":[{"type":"download","text":"Download","url":"profile\/Benjamin_Li\/publication\/273125269_Indefinite_kernel_ridge_regression_and_its_application_on_QSAR_modelling\/links\/54f80ab20cf210398e93ea29.pdf?origin=publication_list","active":false,"primary":true,"extraClass":"open-viewer","icon":null,"data":[]}],"actionWidgets":[],"publicationItemFulltext":{"data":{"isDataset":false,"isResearch":false,"isFulltext":true,"source":{"isPreview":false,"sourceUrl":"profile\/Benjamin_Li","sourceName":"Benjamin Yee Shing Li","hasSourceUrl":true},"publicationUid":273125269,"publicationUrl":"publication\/273125269_Indefinite_kernel_ridge_regression_and_its_application_on_QSAR_modelling","eventCode":"","citationCount":0,"additionalContentWidgets":[],"disableViewer":true,"showLoggedOutRequestButton":false,"context":null,"previewUrl":"https:\/\/i1.rgstatic.net\/publication\/273125269_Indefinite_kernel_ridge_regression_and_its_application_on_QSAR_modelling\/links\/54f80ab20cf210398e93ea29\/smallpreview.png","linkId":"54f80ab20cf210398e93ea29","origin":"publication_list","showRequestCount":false,"publish":false,"request":false,"showAction":false,"displayAsLink":true,"documentViewerUrl":"publicliterature.PublicLiteratureDocumentViewer.html?publicationId=273125269&reference=54f80ab20cf210398e93ea29&eventCode=&origin=publication_list","widgetId":"rgw34_56ab1fd7ae687"},"id":"rgw34_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationItemFulltext.html","templateExtensions":["generalHelpers"],"attrs":{"preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationItemFulltext.html?publicationUid=273125269&showAction=1&eventCode=&hideSource=0&displayAsLink=1&showPublishAsSelectFile=0","viewClass":"views.publicliterature.PublicationItemFulltextView","yuiModules":["rg.views.publicliterature.PublicationItemFulltextView"],"stylesheets":[],"_isYUI":true},"linkId":"54f80ab20cf210398e93ea29","context":null,"contextId":null,"eventCode":"","isCitation":true,"isPendingCitationRequest":false,"canRemoveCitation":false,"citationSourcePublicationUid":11216584,"publicationCitationUid":false,"showPublicationDownloadDataUsagePermissionDialog":false,"origin":"publication_list","keywordList":null,"hasKeywordList":false,"showOpenReviewButton":false,"openReviewUrl":"publication\/273125269_Indefinite_kernel_ridge_regression_and_its_application_on_QSAR_modelling\/review","additionalRightSideTopWidgets":[{"data":{"citationContexts":["However , for some kinds of data such as structural objects like 3D conformations of molecular structures and DNA sequences, features are hard to be extracted. To apply kernelized method over these structural objects, a significant level of efforts is devoted to the design of customized kernels such as graph kernels and string kernels [9] [10] [11]. "],"widgetId":"rgw35_56ab1fd7ae687"},"id":"rgw35_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationItemCitationContexts.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationItemCitationContexts.html","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true}],"additionalRightSideBottomWidgets":[],"requestEndpointPrimary":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","requestEndpointImage":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","doi":false,"hasMicrodata":false,"microdataPropertyName":null,"showPublicationPreview":true,"showPublicationPreviewOnRightSide":false,"showActionsOnRightSide":false,"swapJournalAndAbstractPositions":false,"displayUpdatedExpanderPlugin":false,"showActions":true,"showAuthors":true,"isResearch":false,"publicationLinkText":"Read full-text","widgetId":"rgw33_56ab1fd7ae687"},"id":"rgw33_56ab1fd7ae687","partials":{"publicationItem":"publicliterature\/stubs\/partials\/publicationItem.html"},"templateName":"publicliterature\/stubs\/PublicationItem.html","templateExtensions":["generalHelpers"],"attrs":{"opensViewer":true,"requestFulltextInvitationDialogUrl":"literature.PublicationRequestFulltextInvitationDialog.html","followContext":"publication_item","publicationBookmarkEndpointUrl":"literature.AjaxFollowPublication.html","preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationItem.html?showEnrichedPublicationItem=0&showRequestButton=1&publicationUid=273125269&additionalCssClasses%5B0%5D=includes-citation-list&citationContextItemVersion=old&isIncomingCitation=1","viewClass":"views.publicliterature.PublicationItemView","yuiModules":["rg.views.publicliterature.PublicationItemView"],"stylesheets":[],"_isYUI":true},{"data":{"milestones":{"moreContext":"experimentMilestoneClickedCitationContextReadMore","clickThrough":"experimentMilestoneClickedToPublicationFromCitationWithFulltext","clickThroughWithContext":"experimentMilestoneClickedCitationContextReadMore"},"nextPublicationMilestone":"experimentMilestoneRequestFulltextNoPreview","nextPublicationViewId":null,"authorsPartOne":[{"id":70701639,"url":"researcher\/70701639_Basura_Fernando","fullname":"Basura Fernando","last":false,"imageUrl":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/AS%3A272462717911047%401441971582228_m\/Basura_Fernando.png"},{"id":15519860,"url":"researcher\/15519860_Elisa_Fromont","fullname":"Elisa Fromont","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":70568361,"url":"researcher\/70568361_Damien_Muselet","fullname":"Damien Muselet","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":10762347,"url":"researcher\/10762347_Marc_Sebban","fullname":"Marc Sebban","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"authorsPartTwo":null,"surplusAuthors":null,"additionalCssClasses":[["includes-citation-list"]],"isFulltext":true,"isSlurp":false,"isNoText":false,"publicationType":"Conference Paper","publicationDate":"Jun 2012","journal":null,"showEnrichedPublicationItem":false,"citationCount":24,"commentCount":0,"requestFulltextButton":null,"publicationUrl":"publication\/254257268_Discriminative_Feature_Fusion_for_Image_Classification","usePlainButton":true,"publicationUid":254257268,"feedStubs":null,"hasFeedStubs":false,"impactPoints":false,"url":"publication\/254257268_Discriminative_Feature_Fusion_for_Image_Classification","title":"Discriminative Feature Fusion for Image Classification","displayTitleAsLink":true,"authors":[{"id":70701639,"url":"researcher\/70701639_Basura_Fernando","fullname":"Basura Fernando","last":false,"imageUrl":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/AS%3A272462717911047%401441971582228_m\/Basura_Fernando.png"},{"id":15519860,"url":"researcher\/15519860_Elisa_Fromont","fullname":"Elisa Fromont","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":70568361,"url":"researcher\/70568361_Damien_Muselet","fullname":"Damien Muselet","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":10762347,"url":"researcher\/10762347_Marc_Sebban","fullname":"Marc Sebban","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"displayAuthorsAsLinks":true,"displayAuthors":true,"displayJournalMetadata":true,"collapseTitle":false,"collapseAuthors":false,"collapseAbstract":false,"details":["CVPR, Providence, RI; 06\/2012"],"abstract":"Bag-of-words-based image classification approaches mostly rely on low level local shape features. However, it has been shown that combining multiple cues such as color, texture, or shape is a challenging and promising task which can improve the classification accuracy. Most of the state-of-the-art feature fusion methods usually aim to weight the cues without considering their statistical dependence in the application at hand. In this paper, we present a new logistic regression-based fusion method, called LRFF, which takes advantage of the different cues without being tied to any of them. We also design a new marginalized kernel by making use of the output of the regression model. We show that such kernels, surprisingly ignored so far by the computer vision community, are particularly well suited to achieve image classification tasks. We compare our approach with existing methods that combine color and shape on three datasets. The proposed learning-based feature fusion process clearly outperforms the state-of-the art fusion methods for image classification.","description":false,"swapJournalAndAuthorPositions":false,"showAbstract":true,"type":"Conference Paper","showType":true,"showPublicationIcon":false,"showSeeInContextCta":false,"publicationUrlWithContext":"publication\/254257268_Discriminative_Feature_Fusion_for_Image_Classification","isReader":null,"isAuthor":null,"isRequester":false,"hasFulltext":true,"actions":[{"type":"download","text":"Download","url":"profile\/Basura_Fernando\/publication\/254257268_Discriminative_Feature_Fusion_for_Image_Classification\/links\/004635293c79ddb86a000000.pdf?origin=publication_list","active":false,"primary":true,"extraClass":"open-viewer","icon":null,"data":[]}],"actionWidgets":[],"publicationItemFulltext":{"data":{"isDataset":false,"isResearch":false,"isFulltext":true,"source":{"isPreview":false,"sourceUrl":"profile\/Basura_Fernando","sourceName":"Basura Fernando","hasSourceUrl":true},"publicationUid":254257268,"publicationUrl":"publication\/254257268_Discriminative_Feature_Fusion_for_Image_Classification","eventCode":"","citationCount":0,"additionalContentWidgets":[],"disableViewer":true,"showLoggedOutRequestButton":false,"context":null,"previewUrl":"https:\/\/i1.rgstatic.net\/publication\/254257268_Discriminative_Feature_Fusion_for_Image_Classification\/links\/004635293c79ddb86a000000\/smallpreview.png","linkId":"004635293c79ddb86a000000","origin":"publication_list","showRequestCount":false,"publish":false,"request":false,"showAction":false,"displayAsLink":true,"documentViewerUrl":"publicliterature.PublicLiteratureDocumentViewer.html?publicationId=254257268&reference=004635293c79ddb86a000000&eventCode=&origin=publication_list","widgetId":"rgw37_56ab1fd7ae687"},"id":"rgw37_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationItemFulltext.html","templateExtensions":["generalHelpers"],"attrs":{"preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationItemFulltext.html?publicationUid=254257268&showAction=1&eventCode=&hideSource=0&displayAsLink=1&showPublishAsSelectFile=0","viewClass":"views.publicliterature.PublicationItemFulltextView","yuiModules":["rg.views.publicliterature.PublicationItemFulltextView"],"stylesheets":[],"_isYUI":true},"linkId":"004635293c79ddb86a000000","context":null,"contextId":null,"eventCode":"","isCitation":true,"isPendingCitationRequest":false,"canRemoveCitation":false,"citationSourcePublicationUid":11216584,"publicationCitationUid":false,"showPublicationDownloadDataUsagePermissionDialog":false,"origin":"publication_list","keywordList":null,"hasKeywordList":false,"showOpenReviewButton":false,"openReviewUrl":"publication\/254257268_Discriminative_Feature_Fusion_for_Image_Classification\/review","additionalRightSideTopWidgets":[{"data":{"citationContexts":["We claim that this kernel is not well suited to our logistic regression context . In the following, we rather make use of the information provided by our model to design a new marginalized kernel [14] [16], which is defined in its original form as fol- lows: "],"widgetId":"rgw38_56ab1fd7ae687"},"id":"rgw38_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationItemCitationContexts.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationItemCitationContexts.html","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true}],"additionalRightSideBottomWidgets":[],"requestEndpointPrimary":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","requestEndpointImage":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","doi":false,"hasMicrodata":false,"microdataPropertyName":null,"showPublicationPreview":true,"showPublicationPreviewOnRightSide":false,"showActionsOnRightSide":false,"swapJournalAndAbstractPositions":false,"displayUpdatedExpanderPlugin":false,"showActions":true,"showAuthors":true,"isResearch":false,"publicationLinkText":"Read full-text","widgetId":"rgw36_56ab1fd7ae687"},"id":"rgw36_56ab1fd7ae687","partials":{"publicationItem":"publicliterature\/stubs\/partials\/publicationItem.html"},"templateName":"publicliterature\/stubs\/PublicationItem.html","templateExtensions":["generalHelpers"],"attrs":{"opensViewer":true,"requestFulltextInvitationDialogUrl":"literature.PublicationRequestFulltextInvitationDialog.html","followContext":"publication_item","publicationBookmarkEndpointUrl":"literature.AjaxFollowPublication.html","preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationItem.html?showEnrichedPublicationItem=0&showRequestButton=1&publicationUid=254257268&additionalCssClasses%5B0%5D=includes-citation-list&citationContextItemVersion=old&isIncomingCitation=1","viewClass":"views.publicliterature.PublicationItemView","yuiModules":["rg.views.publicliterature.PublicationItemView"],"stylesheets":[],"_isYUI":true},{"data":{"milestones":{"moreContext":"experimentMilestoneClickedCitationContextReadMore","clickThrough":"experimentMilestoneClickedToPublicationFromCitationWithSlurp","clickThroughWithContext":"experimentMilestoneClickedCitationContextReadMore"},"nextPublicationMilestone":"experimentMilestoneRequestFulltextSlurp","nextPublicationViewId":null,"authorsPartOne":[{"id":69821672,"url":"researcher\/69821672_Xiong_Li","fullname":"Xiong Li","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":10726208,"url":"researcher\/10726208_Tai_Sing_Lee","fullname":"Tai Sing Lee","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":11760291,"url":"researcher\/11760291_Yuncai_Liu","fullname":"Yuncai Liu","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"authorsPartTwo":null,"surplusAuthors":null,"additionalCssClasses":[["includes-citation-list"]],"isFulltext":false,"isSlurp":true,"isNoText":false,"publicationType":"Conference Paper","publicationDate":"Jul 2011","journal":null,"showEnrichedPublicationItem":false,"citationCount":12,"commentCount":0,"requestFulltextButton":null,"publicationUrl":"publication\/224254886_Hybrid_generative-discriminative_classification_using_posterior_divergence","usePlainButton":true,"publicationUid":224254886,"feedStubs":null,"hasFeedStubs":false,"impactPoints":false,"url":"publication\/224254886_Hybrid_generative-discriminative_classification_using_posterior_divergence","title":"Hybrid generative-discriminative classification using posterior divergence","displayTitleAsLink":true,"authors":[{"id":69821672,"url":"researcher\/69821672_Xiong_Li","fullname":"Xiong Li","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":10726208,"url":"researcher\/10726208_Tai_Sing_Lee","fullname":"Tai Sing Lee","last":false,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"},{"id":11760291,"url":"researcher\/11760291_Yuncai_Liu","fullname":"Yuncai Liu","last":true,"imageUrl":"https:\/\/c5.rgstatic.net\/m\/2951093203564\/images\/template\/default\/profile\/profile_default_s.jpg"}],"displayAuthorsAsLinks":true,"displayAuthors":true,"displayJournalMetadata":true,"collapseTitle":false,"collapseAuthors":false,"collapseAbstract":false,"details":["Computer Vision and Pattern Recognition (CVPR), 2011 IEEE Conference on; 07\/2011"],"abstract":"Integrating generative models and discriminative models in a hybrid scheme has shown some success in recognition tasks. In such scheme, generative models are used to derive feature maps for outputting a set of fixed length features that are used by discriminative models to perform classification. In this paper, we present a method, called posterior divergence, to derive feature maps from the log likelihood function implied in the incremental expectation-maximization algorithm. These feature maps evaluate a sample in three complementary measures: (1) how much the sample affects the model; (2) how well the sample fits the model; (3) how uncertain the fit is. We prove that the linear classification error rate using the outputs of the derived feature maps is at least as low as that of plug-in estimation. We present efficient algorithms for computing these feature maps for semi-supervised learning and supervised learning. We evaluate the proposed method on three typical applications, i.e. scene recognition, face and non-face classification and protein sequence analysis, and demonstrate improvements over related methods.","description":false,"swapJournalAndAuthorPositions":false,"showAbstract":true,"type":"Conference Paper","showType":true,"showPublicationIcon":false,"showSeeInContextCta":false,"publicationUrlWithContext":"publication\/224254886_Hybrid_generative-discriminative_classification_using_posterior_divergence","isReader":null,"isAuthor":null,"isRequester":false,"hasFulltext":false,"actions":[{"type":"request-external","text":"Request full-text","url":"javascript:;","active":false,"primary":false,"extraClass":null,"icon":null,"data":[{"key":"context","value":"pubCit"}]}],"actionWidgets":[],"publicationItemFulltext":{"data":{"isDataset":false,"isResearch":false,"isFulltext":true,"source":{"isPreview":true,"sourceUrl":"deref\/http%3A%2F%2Fwww.cnbc.cmu.edu%2F%7Etai%2Fpapers%2FLi_PosDiv_fixed.pdf","sourceName":"cmu.edu","hasSourceUrl":true},"publicationUid":224254886,"publicationUrl":"publication\/224254886_Hybrid_generative-discriminative_classification_using_posterior_divergence","eventCode":"","citationCount":0,"additionalContentWidgets":[],"disableViewer":true,"showLoggedOutRequestButton":false,"context":null,"previewUrl":"https:\/\/i1.rgstatic.net\/publication\/224254886_Hybrid_generative-discriminative_classification_using_posterior_divergence\/links\/0ffc58f70cf255165fc7e8ab\/smallpreview.png","linkId":"0ffc58f70cf255165fc7e8ab","origin":"publication_list","showRequestCount":false,"publish":false,"request":false,"showAction":false,"displayAsLink":true,"documentViewerUrl":"publicliterature.PublicLiteratureDocumentViewer.html?publicationId=224254886&reference=0ffc58f70cf255165fc7e8ab&eventCode=&origin=publication_list","widgetId":"rgw40_56ab1fd7ae687"},"id":"rgw40_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationItemFulltext.html","templateExtensions":["generalHelpers"],"attrs":{"preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationItemFulltext.html?publicationUid=224254886&showAction=1&eventCode=&hideSource=0&displayAsLink=1&showPublishAsSelectFile=0","viewClass":"views.publicliterature.PublicationItemFulltextView","yuiModules":["rg.views.publicliterature.PublicationItemFulltextView"],"stylesheets":[],"_isYUI":true},"linkId":null,"context":null,"contextId":null,"eventCode":"","isCitation":true,"isPendingCitationRequest":false,"canRemoveCitation":false,"citationSourcePublicationUid":11216584,"publicationCitationUid":false,"showPublicationDownloadDataUsagePermissionDialog":false,"origin":"publication_list","keywordList":null,"hasKeywordList":false,"showOpenReviewButton":false,"openReviewUrl":"publication\/224254886_Hybrid_generative-discriminative_classification_using_posterior_divergence\/review","additionalRightSideTopWidgets":[{"data":{"citationContexts":["\u02c6 y = sign(P(y = +1 | x, \u03b8) \u2212 1\/2) is generally inferior to discriminative models which are more powerful in capturing decision boundaries among different classes and more widely used in recognition tasks. At present, several hybrid generative-discriminative schemes have been proposed to combine the strengths of these two classes of models in a number of applications, from scene classification [3], object recognition [5], speech recognition [19] to biological sequence analysis [6] [21], resulting in state-of-the-art performance. These hybrid schemes sought to integrate the intra-class information from generative models and the complementary inter-class information from discriminative methods. "],"widgetId":"rgw41_56ab1fd7ae687"},"id":"rgw41_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationItemCitationContexts.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationItemCitationContexts.html","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true}],"additionalRightSideBottomWidgets":[],"requestEndpointPrimary":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","requestEndpointImage":"literature.PublicationPromoRequestFulltextAction.html?dbw=true","doi":false,"hasMicrodata":false,"microdataPropertyName":null,"showPublicationPreview":true,"showPublicationPreviewOnRightSide":false,"showActionsOnRightSide":false,"swapJournalAndAbstractPositions":false,"displayUpdatedExpanderPlugin":false,"showActions":true,"showAuthors":true,"isResearch":false,"publicationLinkText":"Read preview","widgetId":"rgw39_56ab1fd7ae687"},"id":"rgw39_56ab1fd7ae687","partials":{"publicationItem":"publicliterature\/stubs\/partials\/publicationItem.html"},"templateName":"publicliterature\/stubs\/PublicationItem.html","templateExtensions":["generalHelpers"],"attrs":{"opensViewer":true,"requestFulltextInvitationDialogUrl":"literature.PublicationRequestFulltextInvitationDialog.html","followContext":"publication_item","publicationBookmarkEndpointUrl":"literature.AjaxFollowPublication.html","preSignUpDialogContext":null},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationItem.html?showEnrichedPublicationItem=0&showRequestButton=1&publicationUid=224254886&additionalCssClasses%5B0%5D=includes-citation-list&citationContextItemVersion=old&isIncomingCitation=1","viewClass":"views.publicliterature.PublicationItemView","yuiModules":["rg.views.publicliterature.PublicationItemView"],"stylesheets":[],"_isYUI":true}],"hasCitations":true,"isPublicationAuthor":false,"isPublicationVisitor":false,"publicationUid":11216584,"publicationLink":"publication\/11216584_Marginalized_kernels_for_biological_sequences","hasShowMore":true,"newOffset":3,"pageSize":10,"widgetId":"rgw32_56ab1fd7ae687"},"id":"rgw32_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationIncomingCitationsList.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationIncomingCitationsList.html?publicationUid=11216584&citedInPage=1&swapJournalAndAuthorPositions=0&showAbstract=1&showType=1&showPublicationPreview=1&totalCount=139","viewClass":"views.publicliterature.PublicationIncomingCitationsListView","yuiModules":["rg.views.publicliterature.PublicationIncomingCitationsListView"],"stylesheets":[],"_isYUI":true},"hasCitations":false,"citationsCount":0,"hasIncomingCitations":true,"incomingCitationsCount":139,"showCitationsSorter":true,"showAbstract":true,"showType":true,"showPublicationPreview":true,"swapJournalAndAuthorPositions":false,"sort":"","sortOriginal":false,"citationList":"incoming","showsIncoming":true,"showSorting":false,"usePlainButton":null,"useEnrichedContext":null,"widgetId":"rgw31_56ab1fd7ae687"},"id":"rgw31_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationCitations.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationCitations.html?publicationUid=11216584&citationList=&citedInPage=1&sort=","viewClass":"views.publicliterature.PublicationCitationsView","yuiModules":["rg.views.publicliterature.PublicationCitationsView"],"stylesheets":[],"_isYUI":true},"publicationReviewPromo":null,"publicationUsedInReviews":null,"publicationPdfJsReader":{"data":{"isPreview":false,"licenseUrl":null,"licenseInfo":null,"defaultLinkData":{"linkId":"00b7d52705ee89c23a000000","name":"Koji Tsuda","date":null,"nameLink":"profile\/Koji_Tsuda","filename":"S268.full.pdf","downloadLink":"https:\/\/www.researchgate.net\/profile\/Koji_Tsuda\/publication\/11216584_Marginalized_kernels_for_biological_sequences\/links\/00b7d52705ee89c23a000000.pdf?inViewer=0&pdfJsDownload=0&origin=publication_detail","viewerUrl":"https:\/\/www.researchgate.net\/profile\/Koji_Tsuda\/publication\/11216584_Marginalized_kernels_for_biological_sequences\/links\/00b7d52705ee89c23a000000.pdf?inViewer=1&pdfJsDownload=1&origin=publication_detail","downloadHash":"7956bc04b908662491d59a9c0cbad405","showFileSizeNote":false,"fileSize":"176.99 KB","noFollow":false,"isDefault":true,"doi":null},"displayableLinks":[{"linkId":"00b7d52705ee89c23a000000","name":"Koji Tsuda","date":null,"nameLink":"profile\/Koji_Tsuda","filename":"S268.full.pdf","downloadLink":"https:\/\/www.researchgate.net\/profile\/Koji_Tsuda\/publication\/11216584_Marginalized_kernels_for_biological_sequences\/links\/00b7d52705ee89c23a000000.pdf?inViewer=0&pdfJsDownload=0&origin=publication_detail","viewerUrl":"https:\/\/www.researchgate.net\/profile\/Koji_Tsuda\/publication\/11216584_Marginalized_kernels_for_biological_sequences\/links\/00b7d52705ee89c23a000000.pdf?inViewer=1&pdfJsDownload=1&origin=publication_detail","downloadHash":"7956bc04b908662491d59a9c0cbad405","showFileSizeNote":false,"fileSize":"176.99 KB","noFollow":false,"isDefault":true,"doi":null}],"hasDisplayableLinks":false,"reader":{"data":{"pdfCommentsActive":false,"publicationType":"Article","onlyRenderFirstPage":false,"readMoreExperimentGoal":"goalPublicationPdfClicksReadMore","readMoreExperimentViewId":null,"comments":null,"figures":[],"figureAssetIds":[],"figureOverlayUrls":{"clickOnOverlay":"publication.PublicationFigures.html?_sg=ZfRhmOTAbch_CerWB-koJ1yJF5LCcVYMLcJ2H-_jmjr1oCzxLtkwgbmvJlzxliPzBGYuuthp2eWfDZBdvzI3pA.sWpJiv2Xq7SfPqLFaqMw0PyX-Q5N-yo8RlYmsq1OliQ-2AwtOG826aAO1VhH9ty2TGfKpbl9W5VJqM040sb2-g","clickOnPill":"publication.PublicationFigures.html?_sg=EJ8qG0Bj3agCAK7efXZIIwjIKGD4R-B4yHh36pFOfG4F2yssjwrUbBzn4SIgna7gCo-gZnPIoPEx5Pty9Hm42A.a7DeSPZsEtnT_ZSYPjtvy7De1ajdWz6zrfYw8Ht4KPMl9-bSc5q_r0jC9nqHbFbDU3sTG5TkXF9716eLfagCWg"},"canSelect":false,"javascriptPath":"https:\/\/www.researchgate.net\/c\/o1o9o3\/","downloadUrl":"https%3A%2F%2Fwww.researchgate.net%2Fprofile%2FKoji_Tsuda%2Fpublication%2F11216584_Marginalized_kernels_for_biological_sequences%2Flinks%2F00b7d52705ee89c23a000000.pdf%3FinViewer%3D1%26pdfJsDownload%3D1%26origin%3Dpublication_detail","viewerUrl":"https:\/\/www.researchgate.net\/c\/o1o9o3\/javascript\/lib\/pdfjs\/web\/viewer.html","commentEndpoint":"https:\/\/www.researchgate.net\/rgformat\/api\/comment","experimentEndpoint":"https:\/\/www.researchgate.net\/rgformat\/api\/experiment?key=vjDj7MruijEJycNASblUceZacAWfSmkWTG_dN1pKfpMiZutNu3i36lxRos7-blKdVaeZONkaxg8bq4Zrngkbeg","urlHash":"6e4295ff7fc7caf58dd75b0e02dee7d9","downloadTrackUrl":"application.PdfJsReader.ajaxTrackDownload.html?msrp=z1GVXZ4rExQsemh32SnUFGF00o2FblpaakX0JVoF9Lu35HH_mOHiwOdNmR7g5Eq1mLZMZHPN3_YnaDry0sh1fuHd3qxHXp6bh704IYKJGsI.HRv8CFzLJG8BNePOKxkL6vVl7Lbhg6jXZM2xvsHU_54Aq8rL5ygr5GnEyTu9OCoQfPNo1b99BNCawNtKGEpm6w.I7OyOONhoYbBdnKtSfdF0aiHDDD4tEpl8EJOwbQHUiOPZ2diF5QFI0DVK8Y9CfLtE4WErWgys2dKJUCzeKMjbQ","viewportMilestoneTrackUrl":"application.PdfJsReader.ajaxTrackViewportMilestone.html","linkId":"00b7d52705ee89c23a000000","trackedDownloads":{"00b7d52705ee89c23a000000":{"v":false,"d":false}},"assetId":"AS:102515337072650@1401452969693","readerDocId":null,"assetType":"fulltext","interactionType":{"comment":"comment","highlight":"highlight"},"publicationUid":11216584,"commentCursorPromo":null,"widgetId":"rgw43_56ab1fd7ae687"},"id":"rgw43_56ab1fd7ae687","partials":[],"templateName":"application\/stubs\/PdfJsReader.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.PdfJsReader.html?fileHref=https%3A%2F%2Fwww.researchgate.net%2Fprofile%2FKoji_Tsuda%2Fpublication%2F11216584_Marginalized_kernels_for_biological_sequences%2Flinks%2F00b7d52705ee89c23a000000.pdf%3FinViewer%3D1%26pdfJsDownload%3D1%26origin%3Dpublication_detail&assetId=AS%3A102515337072650%401401452969693&publicationUid=11216584&linkId=00b7d52705ee89c23a000000&onlyShowFirstPage=0","viewClass":null,"yuiModules":["css-pow-application-PdfJsReader"],"stylesheets":["pow\/application\/PdfJsReader.css"],"_isYUI":true},"showHeader":true,"title":"Marginalized kernels for biological sequences","publicationType":"Article","downloadTrackUrl":"publicliterature.PublicationInlineReader.ajaxTrackDownload.html?msrp=1yLWXlPqEE4uWdA2b9m1xrwKWg0g-jXb53Q_4NeRv37qiFZl53G6ScrBrNmXvS620eG6Mt08Fl-JV8fr_qxRQYyG4A3Rz75IEHlFM1A-kgA.X8Y9l2Hc-CKBOJZau9XsnBLF5yvpAJIHBFeqzrlmH-1Lp5kyna338Gw2wCXu-nwZpjxbeue5k7JoVTN90FjbjA.uXE1HZgTRTq_WSlxK86g83kY0LcUEbgcLvt3I6NI5IcmpiuyTtEECRZ05HjX88WfTwzqKp84d0Ap--VxiSwT3w","publicationUid":11216584,"trackedDownloads":{"00b7d52705ee89c23a000000":{"v":false,"d":false}},"inlinePdf":false,"publicationComments":null,"showDownloadButton":true,"socialShare":{"data":{"shareItems":[{"data":{"name":"Facebook","url":"http:\/\/www.facebook.com\/share.php?u={{url}}{{#title}}&t={{title}}{{\/title}}","width":600,"height":350,"useUtmTags":true,"utmSource":"facebook","utmMedium":"rgShare","utmCampaign":"shareFullTextPublication","cssClass":"","trackClick":true,"trackUrl":"https:\/\/www.researchgate.net\/application.ShareFacebook.ajaxTrackSocialShare.html","widgetId":"rgw45_56ab1fd7ae687"},"id":"rgw45_56ab1fd7ae687","partials":{"shareIcon":"application\/stubs\/partials\/shareFacebookBlogIcon.html"},"templateName":"application\/stubs\/BaseShare.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.ShareFacebook.html?provider=Facebook&shareIcon=shareIconBlog&utmSource=facebook&utmMedium=rgShare&utmCampaign=shareFullTextPublication&trackClick=1","viewClass":"views.application.BaseShareView","yuiModules":["rg.views.application.BaseShareView"],"stylesheets":[],"_isYUI":true},{"data":{"name":"Twitter","url":"http:\/\/twitter.com\/intent\/tweet?text={{#title}}{{title}}: {{\/title}}{{url}}&via=researchgate","width":600,"height":350,"useUtmTags":true,"utmSource":"twitter","utmMedium":"rgShare","utmCampaign":"shareFullTextPublication","cssClass":"","trackClick":true,"trackUrl":"https:\/\/www.researchgate.net\/application.ShareTwitter.ajaxTrackSocialShare.html","widgetId":"rgw46_56ab1fd7ae687"},"id":"rgw46_56ab1fd7ae687","partials":{"shareIcon":"application\/stubs\/partials\/shareTwitterBlogIcon.html"},"templateName":"application\/stubs\/BaseShare.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.ShareTwitter.html?provider=Twitter&shareIcon=shareIconBlog&utmSource=twitter&utmMedium=rgShare&utmCampaign=shareFullTextPublication&trackClick=1","viewClass":"views.application.BaseShareView","yuiModules":["rg.views.application.BaseShareView"],"stylesheets":[],"_isYUI":true},{"data":{"name":"Google+","url":"https:\/\/plus.google.com\/share?url={{url}}","width":600,"height":600,"useUtmTags":true,"utmSource":"googleplus","utmMedium":"rgShare","utmCampaign":"shareFullTextPublication","cssClass":"","trackClick":true,"trackUrl":"https:\/\/www.researchgate.net\/application.ShareGooglePlus.ajaxTrackSocialShare.html","widgetId":"rgw47_56ab1fd7ae687"},"id":"rgw47_56ab1fd7ae687","partials":{"shareIcon":"application\/stubs\/partials\/shareGooglePlusBlogIcon.html"},"templateName":"application\/stubs\/BaseShare.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.ShareGooglePlus.html?provider=Google%2B&shareIcon=shareIconBlog&utmSource=googleplus&utmMedium=rgShare&utmCampaign=shareFullTextPublication&trackClick=1","viewClass":"views.application.BaseShareView","yuiModules":["rg.views.application.BaseShareView"],"stylesheets":[],"_isYUI":true},{"data":{"name":"LinkedIn","url":"http:\/\/www.linkedin.com\/shareArticle?mini=true&url={{url}}{{#title}}&title={{title}}{{\/title}}&source=ResearchGate","width":520,"height":570,"useUtmTags":true,"utmSource":"linkedin","utmMedium":"rgShare","utmCampaign":"shareFullTextPublication","cssClass":"","trackClick":true,"trackUrl":"https:\/\/www.researchgate.net\/application.ShareLinkedIn.ajaxTrackSocialShare.html","widgetId":"rgw48_56ab1fd7ae687"},"id":"rgw48_56ab1fd7ae687","partials":{"shareIcon":"application\/stubs\/partials\/shareLinkedInBlogIcon.html"},"templateName":"application\/stubs\/BaseShare.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.ShareLinkedIn.html?provider=LinkedIn&shareIcon=shareIconBlog&utmSource=linkedin&utmMedium=rgShare&utmCampaign=shareFullTextPublication&trackClick=1","viewClass":"views.application.BaseShareView","yuiModules":["rg.views.application.BaseShareView"],"stylesheets":[],"_isYUI":true},{"data":{"name":"Reddit","url":"https:\/\/www.reddit.com\/submit?url={{url}}{{#title}}&title={{title}}{{\/title}}","width":600,"height":600,"useUtmTags":true,"utmSource":"reddit","utmMedium":"rgShare","utmCampaign":"shareFullTextPublication","cssClass":"","trackClick":true,"trackUrl":"https:\/\/www.researchgate.net\/application.ShareReddit.ajaxTrackSocialShare.html","widgetId":"rgw49_56ab1fd7ae687"},"id":"rgw49_56ab1fd7ae687","partials":{"shareIcon":"application\/stubs\/partials\/shareRedditBlogIcon.html"},"templateName":"application\/stubs\/BaseShare.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.ShareReddit.html?provider=Reddit&shareIcon=shareIconBlog&utmSource=reddit&utmMedium=rgShare&utmCampaign=shareFullTextPublication&trackClick=1","viewClass":"views.application.BaseShareView","yuiModules":["rg.views.application.BaseShareView"],"stylesheets":[],"_isYUI":true}],"widgetId":"rgw44_56ab1fd7ae687"},"id":"rgw44_56ab1fd7ae687","partials":[],"templateName":"application\/stubs\/SocialShare.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.SocialShare.html?campaign=shareFullTextPublication&trackClick=1&shareIcon=shareIconBlog","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true},"widgetId":"rgw42_56ab1fd7ae687"},"id":"rgw42_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicationInlineReader.html","templateExtensions":["generalHelpers"],"attrs":{"showFulltextDownloadedSignupDialog":true,"preSignUpDialogContext":null,"requestFulltext":false},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicationInlineReader.html","viewClass":"views.publicliterature.PublicationInlineReaderView","yuiModules":["rg.views.publicliterature.PublicationInlineReaderView","css-pow-publicliterature-PublicationInlineReader"],"stylesheets":["pow\/publicliterature\/PublicationInlineReader.css"],"_isYUI":true},"useFulltextOptimizedLayout":false,"publicationActions":null,"requestFulltextPromo":null,"currentUrl":"publication\/11216584_Marginalized_kernels_for_biological_sequences","isLeaderboardAd":false,"enableStickyBox":null,"googleDfpSlotMobileBottom":null,"fullTextExitPopup":null,"showExitPopupDialog":false,"widgetId":"rgw2_56ab1fd7ae687"},"id":"rgw2_56ab1fd7ae687","partials":{"romeo_legal_notice":"publicliterature\/stubs\/partials\/romeo_legal_notice.html"},"templateName":"publicliterature\/stubs\/PublicPublicationDetails_NewLayout.html","templateExtensions":["generalHelpers"],"attrs":{"publicationUid":11216584},"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicPublicationDetailsOld.html?publicationUid=11216584&isTestOldDesign=0","viewClass":"views.publicliterature.PublicPublicationDetailsOldView","yuiModules":["rg.views.publicliterature.PublicPublicationDetailsOldView"],"stylesheets":[],"_isYUI":true},"widgetId":"rgw1_56ab1fd7ae687"},"id":"rgw1_56ab1fd7ae687","partials":[],"templateName":"publicliterature\/stubs\/PublicPublicationDetails.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/publicliterature.PublicPublicationDetails.html","viewClass":"views.publicliterature.PublicPublicationDetailsView","yuiModules":["rg.views.publicliterature.PublicPublicationDetailsView"],"stylesheets":[],"_isYUI":true,"initState":[]});})();
(function(){Y.rg.createInitialWidget({"data":{"uaClass":"","headPrefix":[],"rootUrl":"https:\/\/www.researchgate.net\/","requestToken":"JepYZceksY+\/UOw5jXXF3yzNmlmvflynIo+DsY1Ozb58qjjWuT14CnOiwC6j40VjvyVJ+eJ7P78Plv0MBP03sDb8Pydea3KtExjBg\/4miEk0EBtHbEYMNn3\/iUyuhUZ9CJMifnV4RbBY2o9PUSe9zDT9fm7Q+5e\/D8UMN0w6b\/DuDaasJWh7n9c9a9p1ke45Vl9C7HjlET8a7udQVwuqKYJnZqFgB2zKZoMUoUbMsU913MFIuYf89w3kZKgBG1Q+blJXNoHMTPAtrypPSzz3\/VUYhIAPIkg0pxCz6C+YnFQ=","faviconCdnUrl":"https:\/\/c5.rgstatic.net\/m\/2390829798215018\/images\/favicon.ico","headerOutput":"<noscript><\/noscript><link rel=\"canonical\" href=\"https:\/\/www.researchgate.net\/publication\/11216584_Marginalized_kernels_for_biological_sequences\" \/>\n<link rel=\"dns-prefetch\" href=\"\/\/c5.rgstatic.net\" \/>\n<link rel=\"dns-prefetch\" href=\"\/\/i1.rgstatic.net\" \/>\n<meta property=\"twitter:card\" content=\"summary\" \/>\n<meta property=\"twitter:site\" content=\"@ResearchGate\" \/>\n<meta property=\"og:title\" content=\"Marginalized kernels for biological sequences\" \/>\n<meta property=\"og:description\" content=\"MOTIVATION: Kernel methods such as support vector machines require a kernel function between objects to be defined a priori. Several works have been done to derive kernels from probability...\" \/>\n<meta property=\"og:site_name\" content=\"ResearchGate\" \/>\n<meta property=\"og:image\" content=\"https:\/\/i1.rgstatic.net\/publication\/11216584_Marginalized_kernels_for_biological_sequences\/links\/00b7d52705ee89c23a000000\/smallpreview.png\" \/>\n<meta property=\"og:url\" content=\"https:\/\/www.researchgate.net\/publication\/11216584_Marginalized_kernels_for_biological_sequences\" \/>\n<meta property=\"rg:id\" content=\"PB:11216584\" \/>\n<meta name=\"DC.identifier\" scheme=\"DCTERMS.URI\" content=\"http:\/\/dx.doi.org\/10.1093\/bioinformatics\/18.suppl_1.S268\" \/>\n<meta name=\"gs_meta_revision\" content=\"1.1\" \/>\n<meta name=\"citation_title\" content=\"Marginalized kernels for biological sequences\" \/>\n<meta name=\"citation_author\" content=\"Koji Tsuda\" \/>\n<meta name=\"citation_author\" content=\"Taishin Kin\" \/>\n<meta name=\"citation_author\" content=\"Kiyoshi Asai\" \/>\n<meta name=\"citation_pmid\" content=\"12169556\" \/>\n<meta name=\"citation_publication_date\" content=\"2002\/02\/01\" \/>\n<meta name=\"citation_journal_title\" content=\"Bioinformatics\" \/>\n<meta name=\"citation_issn\" content=\"1367-4803\" \/>\n<meta name=\"citation_volume\" content=\"18 Suppl 1\" \/>\n<meta name=\"citation_issue\" content=\"Suppl 1\" \/>\n<meta name=\"citation_firstpage\" content=\"S268\" \/>\n<meta name=\"citation_lastpage\" content=\"75\" \/>\n<meta name=\"citation_doi\" content=\"10.1093\/bioinformatics\/18.suppl_1.S268\" \/>\n<meta name=\"citation_pdf_url\" content=\"https:\/\/www.researchgate.net\/profile\/Koji_Tsuda\/publication\/11216584_Marginalized_kernels_for_biological_sequences\/links\/00b7d52705ee89c23a000000.pdf\" \/>\n<meta name=\"citation_abstract_html_url\" content=\"https:\/\/www.researchgate.net\/publication\/11216584_Marginalized_kernels_for_biological_sequences\" \/>\n<meta name=\"citation_fulltext_html_url\" content=\"https:\/\/www.researchgate.net\/publication\/11216584_Marginalized_kernels_for_biological_sequences\" \/>\n<meta name=\"viewport\" content=\"width=device-width,initial-scale=1\" \/>\n<link href=\"\/\/c5.rgstatic.net\/m\/22664197317151888\/styles\/rg.css\" type=\"text\/css\" rel=\"stylesheet\"\/>\n<link href=\"\/\/c5.rgstatic.net\/m\/21004998181197492\/styles\/rg2.css\" type=\"text\/css\" rel=\"stylesheet\"\/>\n<!--[if lt IE 9]><link href=\"\/\/c5.rgstatic.net\/m\/238176252723686\/styles\/ie.css\" type=\"text\/css\" rel=\"stylesheet\"\/><![endif]-->\n<link href=\"\/\/c5.rgstatic.net\/m\/217752362214895\/styles\/modules\/publicprofile.css\" type=\"text\/css\" rel=\"stylesheet\"\/>\n<link href=\"\/\/c5.rgstatic.net\/m\/21993347442549\/styles\/pow\/publicliterature\/FollowPublicationPromo.css\" type=\"text\/css\" rel=\"stylesheet\"\/>\n<link href=\"\/\/c5.rgstatic.net\/m\/282514599719602\/styles\/pow\/application\/PdfJsReader.css\" type=\"text\/css\" rel=\"stylesheet\"\/>\n<link href=\"\/\/c5.rgstatic.net\/m\/23819663151220\/styles\/pow\/publicliterature\/PublicationInlineReader.css\" type=\"text\/css\" rel=\"stylesheet\"\/>\n<script src=\"\/\/c5.rgstatic.net\/m\/2321000301012716\/javascript\/vendor\/webfontloader\/webfontloader.js\" type=\"text\/javascript\"><\/script>\n <script>(function(i,s,o,g,r,a,m){i[\"GoogleAnalyticsObject\"]=r;i[r]=i[r]||function(){ (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o), m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m) })(window,document,\"script\",\"\/\/www.google-analytics.com\/analytics.js\",\"ga\");\n ga(\"create\",\"UA-58591210-1\");ga(\"set\",\"anonymizeIp\",true);ga('set', 'dimension1', 'publication full-text');ga('set', 'dimension2', 'Article');ga('set', 'dimension3', 'Logged out');ga('set', 'dimension4', 'https');ga(\"send\",\"pageview\");<\/script>\n","webfont":"<noscript><\/noscript><script> WebFontConfig = { custom: { families: ['source_sans_pro_italic', 'source_sans_pro_bold', 'source_sans_pro', 'martel'], urls : ['\/\/c5.rgstatic.net\/m\/231392577336386\/styles\/fonts.css'] } }; WebFont.load(WebFontConfig); <\/script>\n","correlationId":"rgreq-4f8fec31-4745-407b-bfd0-5c2e051f6cde","accountId":null,"module":"publicliterature","action":"publicliterature.PublicPublicationDetails","page":"publicationDetail","product":"publications","backendTime":620,"continent":"Asia","stylesHome":"\/\/c5.rgstatic.net\/m\/","staticHost":"c5.rgstatic.net","useEarlyFlush":false,"longRunningRequestIdentifier":"LongRunningRequest.publicliterature.PublicPublicationDetails","longRunningRequestFp":"e9e5bbe1366c7ee8c8e41d02f793c884ac9be19b","widgetId":"rgw50_56ab1fd7ae687"},"id":"rgw50_56ab1fd7ae687","partials":[],"templateName":"application\/stubs\/StaticHeader.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.StaticHeader.html","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true,"initState":[]});})();
(function(){
            Y.rg.core.pagespeed.Monitoring.monitorPage("\/\/glassmoni.researchgate.net", "publicliterature.PublicPublicationDetails.run.html.loggedOut.get", "59de9668d07aeb77ab4de5f24d382b5d199d2437", "rgreq-4f8fec31-4745-407b-bfd0-5c2e051f6cde", "60580be58e4c06421a4aa2ab7d8595311a4440a8");
        
            Y.rg.core.pagespeed.Monitoring.monitorPage("\/\/glassmoni.researchgate.net", "publicationDetail.loggedOut", "ed1993d9e20f6cefa83edacaa24401a18c071aea", "rgreq-4f8fec31-4745-407b-bfd0-5c2e051f6cde", "60580be58e4c06421a4aa2ab7d8595311a4440a8");
        })();
(function(){Y.rg.createInitialWidget({"data":{"year":"2016","inlinePromo":null,"isAdmin":false,"contactUrl":"https:\/\/www.researchgate.net\/contact","aboutUsUrl":"https:\/\/www.researchgate.net\/about","widgetId":"rgw51_56ab1fd7ae687"},"id":"rgw51_56ab1fd7ae687","partials":[],"templateName":"application\/stubs\/DefaultFooter.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.DefaultFooter.html","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true,"initState":[]});})();
(function(){Y.rg.createInitialWidget({"data":{"headerLogin":{"data":{"urlAfterLogin":"publication\/11216584_Marginalized_kernels_for_biological_sequences","requestToken":"B8gSzaHs4UiHGqPUieODnKn1JuMV+BZ6n2X7Q73ZsjmyzxTuZYQiiZYwXpANKMRPLVqcNsDRqy78PiOZIGReYH5YwsAU7TiwKxgikwt5cPcziuvWbgY++hAaZqU86u6DcoCmYh6i9JQV6Kst\/inoX5v7OClYI8SzOVml5rguz0UTcJJDvrI4IKT3FoCWU1O6nhnlphi6SX\/mk2SZUsw8ASIS7m2Ceobgqnk2NT2993qDvz5hx8OUBT7EQnO131KsT8OAOwa5Hcqagkyr+dI5J8lH575dnFv3HYQokX7WPSU=","loginUrl":"https:\/\/www.researchgate.net\/application.Login.html","signupUrl":"https:\/\/www.researchgate.net\/signup.SignUp.html?ev=su_chnl_index&hdrsu=1&_sg=ZtAnySYxo0IbqSufCsWjxcuY7_T5bahbTyyTtv3nLx7Uxi05lqeh4-5PnK2gQ6Zi","encodedUrlAfterLogin":"cHVibGljYXRpb24vMTEyMTY1ODRfTWFyZ2luYWxpemVkX2tlcm5lbHNfZm9yX2Jpb2xvZ2ljYWxfc2VxdWVuY2Vz","signupCallToAction":"Join for free","widgetId":"rgw53_56ab1fd7ae687"},"id":"rgw53_56ab1fd7ae687","partials":{"partial":"application\/stubs\/partials\/headerLoginDefault.html"},"templateName":"application\/stubs\/HeaderLogin.html","templateExtensions":[],"attrs":{"goal":"milestoneHeaderLoginSeen"},"widgetUrl":"https:\/\/www.researchgate.net\/application.HeaderLogin.html","viewClass":"views.application.HeaderLoginView","yuiModules":["rg.views.application.HeaderLoginView"],"stylesheets":[],"_isYUI":true},"cookieConsent":null,"logoSvgSrc":"https:\/\/c5.rgstatic.net\/m\/235107188705592\/images\/template\/brand-header-logo.svg","logoFallbackSrc":"https:\/\/c5.rgstatic.net\/m\/238113351022438\/images\/template\/brand-header-logo.png","widgetId":"rgw52_56ab1fd7ae687"},"id":"rgw52_56ab1fd7ae687","partials":{"schemaSocialProfiles":"application\/stubs\/partials\/schemaSocialProfiles.html"},"templateName":"application\/stubs\/HeaderLoggedOut.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.HeaderLoggedOut.html","viewClass":null,"yuiModules":[],"stylesheets":[],"_isYUI":true,"initState":[]});})();
(function(){Y.rg.createInitialWidget({"data":{"logEvent":"su_banner","specialCopy":false,"widgetId":"rgw54_56ab1fd7ae687"},"id":"rgw54_56ab1fd7ae687","partials":[],"templateName":"application\/stubs\/LoggedOutBanner.html","templateExtensions":[],"attrs":[],"widgetUrl":"https:\/\/www.researchgate.net\/application.LoggedOutBanner.html","viewClass":"views.application.LoggedOutBannerView","yuiModules":["rg.views.application.LoggedOutBannerView"],"stylesheets":[],"_isYUI":true,"initState":[]});})();
(function(){Y.rg.core.util.ParameterFilter.filter(["ev","cp","ch","ref","dbw","pli","loginT","uid","claimChannel","enrichId","enrichSource","utm_source","utm_medium","utm_campaign","el","ci"]);})();
});}); } else { throw 'YRG was not loaded when attaching widgets'; }</script><script> dataLayer = [{"pageCategory":"publication full-text","publicationType":"Article","eventCategory":"Publication page"}]; </script> <noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-MKVKH7" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-MKVKH7');</script><script>(function(e){function n(n,t,o,a){"use strict";var i=e.document.createElement("link"),r=t||e.document.getElementsByTagName("script")[0],d=e.document.styleSheets;return i.rel="stylesheet",i.href=n,i.media="only x",a&&(i.onload=a),r.parentNode.insertBefore(i,r),i.onloadcssdefined=function(e){for(var t,o=0;d.length>o;o++)d[o].href&&d[o].href.indexOf(n)>-1&&(t=!0);t?e():setTimeout(function(){i.onloadcssdefined(e)})},i.onloadcssdefined(function(){i.media=o||"all"}),i}function t(e,n){e.onload=function(){e.onload=null,n&&n.call(e)},"isApplicationInstalled"in navigator&&"onloadcssdefined"in e&&e.onloadcssdefined(n)}var o=function(a,i){"use strict";if(a&&3===a.length){var r=e.Image,d=!(!document.createElementNS||!document.createElementNS("http://www.w3.org/2000/svg","svg").createSVGRect||!document.implementation.hasFeature("http://www.w3.org/TR/SVG11/feature#Image","1.1")||e.opera&&-1===navigator.userAgent.indexOf("Chrome")||-1!==navigator.userAgent.indexOf("Series40")),c=new r;c.onerror=function(){o.method="png",o.href=a[2],n(a[2])},c.onload=function(){var e=1===c.width&&1===c.height,r=a[e&&d?0:e?1:2];o.method=e&&d?"svg":e?"datapng":"png",o.href=r,t(n(r),i)},c.src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///ywAAAAAAQABAAACAUwAOw==",document.documentElement.className+=" grunticon"}};o.loadCSS=n,o.onloadCSS=t,e.grunticon=o})(this);grunticon(["https://c5.rgstatic.net/c/o1o9o3/styles/icons/_header-ico.svg.css", "https://c5.rgstatic.net/c/o1o9o3/styles/icons/_header-ico.png.css", "https://c5.rgstatic.net/c/o1o9o3/styles/icons/_header-ico.fallback.scss"]);grunticon(["https://c5.rgstatic.net/c/o1o9o3/styles/icons/_admin-ico.svg.css", "https://c5.rgstatic.net/c/o1o9o3/styles/icons/_admin-ico.png.css", "https://c5.rgstatic.net/c/o1o9o3/styles/icons/_admin-ico.fallback.scss"]);grunticon(["https://c5.rgstatic.net/c/o1o9o3/styles/icons/_ico.svg.css", "https://c5.rgstatic.net/c/o1o9o3/styles/icons/_ico.png.css", "https://c5.rgstatic.net/c/o1o9o3/styles/icons/_ico.fallback.scss"]);</script></body>
</html>
